<!DOCTYPE html>
<html lang="en"><head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1"><!-- Begin Jekyll SEO tag v2.6.1 -->
<title>scRNA-seq and scATAC-seq Data Analysis DREAM Challenge - Dataset 2 Data Generation | ARSN Lab Notebook</title>
<meta name="generator" content="Jekyll v4.1.1" />
<meta property="og:title" content="scRNA-seq and scATAC-seq Data Analysis DREAM Challenge - Dataset 2 Data Generation" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="Program Versions: macs2 v2.1.1.20160309 samtools v1.10 bedtools v2.22.0 python v2.7.9 perl v5.16.3 R v3.5.1" />
<meta property="og:description" content="Program Versions: macs2 v2.1.1.20160309 samtools v1.10 bedtools v2.22.0 python v2.7.9 perl v5.16.3 R v3.5.1" />
<link rel="canonical" href="http://localhost:4000/2022/04/21/post-0081.html" />
<meta property="og:url" content="http://localhost:4000/2022/04/21/post-0081.html" />
<meta property="og:site_name" content="ARSN Lab Notebook" />
<meta property="og:type" content="article" />
<meta property="article:published_time" content="2022-04-21T00:00:00-07:00" />
<script type="application/ld+json">
{"mainEntityOfPage":{"@type":"WebPage","@id":"http://localhost:4000/2022/04/21/post-0081.html"},"url":"http://localhost:4000/2022/04/21/post-0081.html","description":"Program Versions: macs2 v2.1.1.20160309 samtools v1.10 bedtools v2.22.0 python v2.7.9 perl v5.16.3 R v3.5.1","dateModified":"2022-04-21T00:00:00-07:00","datePublished":"2022-04-21T00:00:00-07:00","headline":"scRNA-seq and scATAC-seq Data Analysis DREAM Challenge - Dataset 2 Data Generation","@type":"BlogPosting","@context":"https://schema.org"}</script>
<!-- End Jekyll SEO tag -->
<link rel="stylesheet" href="/assets/main.css"><link type="application/atom+xml" rel="alternate" href="http://localhost:4000/feed.xml" title="ARSN Lab Notebook" /></head>
<body><header class="site-header" role="banner">

  <div class="wrapper"><a class="site-title" rel="author" href="/">ARSN Lab Notebook</a><nav class="site-nav">
        <input type="checkbox" id="nav-trigger" class="nav-trigger" />
        <label for="nav-trigger">
          <span class="menu-icon">
            <svg viewBox="0 0 18 15" width="18px" height="15px">
              <path d="M18,1.484c0,0.82-0.665,1.484-1.484,1.484H1.484C0.665,2.969,0,2.304,0,1.484l0,0C0,0.665,0.665,0,1.484,0 h15.032C17.335,0,18,0.665,18,1.484L18,1.484z M18,7.516C18,8.335,17.335,9,16.516,9H1.484C0.665,9,0,8.335,0,7.516l0,0 c0-0.82,0.665-1.484,1.484-1.484h15.032C17.335,6.031,18,6.696,18,7.516L18,7.516z M18,13.516C18,14.335,17.335,15,16.516,15H1.484 C0.665,15,0,14.335,0,13.516l0,0c0-0.82,0.665-1.483,1.484-1.483h15.032C17.335,12.031,18,12.695,18,13.516L18,13.516z"/>
            </svg>
          </span>
        </label>

        <div class="trigger"><a class="page-link" href="/about/">About</a></div>
      </nav></div>
</header>
<main class="page-content" aria-label="Content">
      <div class="wrapper">
        <article class="post h-entry" itemscope itemtype="http://schema.org/BlogPosting">

  <header class="post-header">
    <h1 class="post-title p-name" itemprop="name headline">scRNA-seq and scATAC-seq Data Analysis DREAM Challenge - Dataset 2 Data Generation</h1>
    <p class="post-meta">
      <time class="dt-published" datetime="2022-04-21T00:00:00-07:00" itemprop="datePublished">Apr 21, 2022
      </time></p>
  </header>

  <div class="post-content e-content" itemprop="articleBody">
    <p>Program Versions:</p>
<div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>macs2 v2.1.1.20160309
samtools v1.10
bedtools v2.22.0
python v2.7.9
perl v5.16.3
R v3.5.1
</code></pre></div></div>

<p>R Packages:</p>
<div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>&gt; sessionInfo()
R version 3.5.1 (2018-07-02)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: CentOS Linux 7 (Core)

Matrix products: default
BLAS: /home/groups/oroaklab/src/R/R-3.5.1/lib/libRblas.so
LAPACK: /home/groups/oroaklab/src/R/R-3.5.1/lib/libRlapack.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] grid      stats     graphics  grDevices utils     datasets  methods  
[8] base     

other attached packages:
[1] ComplexHeatmap_1.20.0 Matrix_1.2-18         cisTopic_0.3.0       
[4] Rphenograph_0.99.1    igraph_1.2.4.2        ggplot2_3.2.1 
</code></pre></div></div>

<p>STEP 1 - CALLING PEAKS
<br />INPUT FILE(s): <code class="language-plaintext highlighter-rouge">full_dataset.bam</code>, <code class="language-plaintext highlighter-rouge">downsampled_40.bam</code>
<br />OUTPUT FILE(s): <code class="language-plaintext highlighter-rouge">full_dataset.peaks.bed</code>, <code class="language-plaintext highlighter-rouge">downsampled_40.peaks.bed</code></p>

<div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code># call peaks, full
$ time macs2 callpeak --keep-dup all -t full_dataset.bam -n test1
real    8m37.638s
user    8m30.649s
sys     0m10.417s

# merge and filter peaks, full
$ perl 01_filter_peaks.pl test1_peaks.narrowPeak test1 500

# call peaks, downsampled
$ time macs2 callpeak --keep-dup all -t downsampled_40.bam -n test2
real    6m16.453s
user    5m54.677s
sys     0m7.381s

# merge and filter peaks, downsampled
$ perl 01.filter_peaks.pl test2_peaks.narrowPeak test2 500

$ md5sum *bed
c18d03f4d22e6938202e9578b2086dad  downsampled_40.500.bed
018e2e40df7f998d19df2f340bbb207c  full_dataset.peaks.bed
018e2e40df7f998d19df2f340bbb207c  test1.500.bed
c18d03f4d22e6938202e9578b2086dad  test2.500.bed
</code></pre></div></div>

<p>STEP 2 - MATRIX GENERATION
<br />INPUT FILE(s): <code class="language-plaintext highlighter-rouge">downsampled_40.bam</code>, <code class="language-plaintext highlighter-rouge">downsampled_40.500.bed</code>
<br />OUTPUT FILE(s): <code class="language-plaintext highlighter-rouge">downsampled_40.500.counts.sparseMatrix.cols.gz</code>, <code class="language-plaintext highlighter-rouge">downsampled_40.500.counts.sparseMatrix.rows.gz</code>, <code class="language-plaintext highlighter-rouge">downsampled_40.500.counts.sparseMatrix.values.gz</code></p>

<div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>time perl 02.make_count_matrix.pl downsampled_40.bam downsampled_40.500.bed test2
real    3m25.788s
user    5m30.598s
sys     0m8.572s

$ zcat downsampled_40.500.counts.sparseMatrix.values.gz &gt; temp1
$ zcat test2.counts.sparseMatrix.values.gz &gt; temp2
$ md5sum temp*
708ad4f6aabbbfed78771e3bc29fd5da  temp1
708ad4f6aabbbfed78771e3bc29fd5da  temp2
</code></pre></div></div>

<p>STEP 3 - DIMENSIONALITY REDUCTION WITH CISTOPICS3
<br />INPUT FILE(s): <code class="language-plaintext highlighter-rouge">downsampled_40.500.counts.sparseMatrix.values.gz</code>
<br />OUTPUT FILE(s): <code class="language-plaintext highlighter-rouge">downsampled_40.500.counts.cistopic.15.matrix</code>, <code class="language-plaintext highlighter-rouge">downsampled_40.500.counts.cistopic.20.matrix</code>, <code class="language-plaintext highlighter-rouge">downsampled_40.500.counts.cistopic.25.matrix</code></p>

<div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>$ time Rscript downsampled_40.500.counts.cistopic.r
real    29m28.503s
user    29m16.418s
sys     0m9.162s

# not similar by md5sum
</code></pre></div></div>

<p>STEP 4 - CISTOPIC3 MATRIX TO UMAP DIMS
<br />INPUT FILE(s): <code class="language-plaintext highlighter-rouge">downsampled_40.500.counts.cistopic.15.matrix</code>, <code class="language-plaintext highlighter-rouge">downsampled_40.500.counts.cistopic.20.matrix</code>, <code class="language-plaintext highlighter-rouge">downsampled_40.500.counts.cistopic.25.matrix</code>
<br />OUTPUT FILE(s): <code class="language-plaintext highlighter-rouge">downsampled_40.500.counts.cistopic.15.UMAP.dims</code>, <code class="language-plaintext highlighter-rouge">downsampled_40.500.counts.cistopic.20.UMAP.dims</code>, <code class="language-plaintext highlighter-rouge">downsampled_40.500.counts.cistopic.25.UMAP.dims</code></p>

<div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>$ time Rscript downsampled_40.500.counts.cistopic.15.UMAP.R
real    1m1.257s
user    0m59.971s
sys     0m1.055s

$ time Rscript downsampled_40.500.counts.cistopic.20.UMAP.R
real    1m0.584s
user    0m59.771s
sys     0m0.673s

$ time Rscript downsampled_40.500.counts.cistopic.25.UMAP.R
real    1m5.051s
user    1m3.943s
sys     0m0.939s

# not similar by md5sum
</code></pre></div></div>

<p>STEP 5 - CLUSTERING WITH PHENOGRAPHS
<br />INPUT FILE(s): <code class="language-plaintext highlighter-rouge">downsampled_40.500.counts.cistopic.15.matrix</code>, <code class="language-plaintext highlighter-rouge">downsampled_40.500.counts.cistopic.20.matrix</code>, <code class="language-plaintext highlighter-rouge">downsampled_40.500.counts.cistopic.25.matrix</code>
<br />OUTPUT FILE(s): <code class="language-plaintext highlighter-rouge">downsampled_40.500.counts.cistopic.15.k50.pg.annot</code>, <code class="language-plaintext highlighter-rouge">downsampled_40.500.counts.cistopic.20.k50.pg.annot</code>, <code class="language-plaintext highlighter-rouge">downsampled_40.500.counts.cistopic.25.k50.pg.annot</code></p>

<div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>$ time Rscript test2.15.k50.phenograph.R
real	0m24.443s
user	0m23.771s
sys	0m0.549s

$ time Rscript test2.20.k50.phenograph.R
real	0m26.302s
user	0m25.649s
sys	0m0.594s

$ time Rscript test2.25.k50.phenograph.R
real	0m25.757s
user	0m25.171s
sys	0m0.504s

$ md5sum *annot
af37ea4bec6586681b1c814c93a256ce  downsampled_40.500.counts.cistopic.15.k50.pg.annot
bedc6d7799fc618654e63450691cc9b7  downsampled_40.500.counts.cistopic.20.k50.pg.annot
4915ad625b38b70ac7d3f969c4159a97  downsampled_40.500.counts.cistopic.25.k50.pg.annot
af37ea4bec6586681b1c814c93a256ce  test2.15.k50.pg.annot
bedc6d7799fc618654e63450691cc9b7  test2.20.k50.pg.annot
4915ad625b38b70ac7d3f969c4159a97  test2.25.k50.pg.annot
</code></pre></div></div>

  </div><a class="u-url" href="/2022/04/21/post-0081.html" hidden></a>
</article>

      </div>
    </main><footer class="site-footer h-card">
  <data class="u-url" href="/"></data>

  <div class="wrapper">

    <h2 class="footer-heading">ARSN Lab Notebook</h2>

    <div class="footer-col-wrapper">
      <div class="footer-col footer-col-1">
        <ul class="contact-list">
          <li class="p-name">ARSN Lab Notebook</li><li><a class="u-email" href="mailto:nishidaa@ohsu.edu">nishidaa@ohsu.edu</a></li></ul>
      </div>

      <div class="footer-col footer-col-2"><ul class="social-media-list"><li><a href="https://github.com/arsnishida"><svg class="svg-icon"><use xlink:href="/assets/minima-social-icons.svg#github"></use></svg> <span class="username">arsnishida</span></a></li></ul>
</div>

      <div class="footer-col footer-col-3">
        <p>ARSN Lab notebook.</p>
      </div>
    </div>

  </div>

</footer>
</body>

</html>
