<!DOCTYPE html>
<html lang="en"><head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1"><!-- Begin Jekyll SEO tag v2.6.1 -->
<title>SSC Final Validation Tables with Annotations | ARSN Lab Notebook</title>
<meta name="generator" content="Jekyll v4.1.1" />
<meta property="og:title" content="SSC Final Validation Tables with Annotations" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="Directory: https://ohsu.app.box.com/folder/81820136345" />
<meta property="og:description" content="Directory: https://ohsu.app.box.com/folder/81820136345" />
<link rel="canonical" href="http://localhost:4000/2020/10/09/post-0002.html" />
<meta property="og:url" content="http://localhost:4000/2020/10/09/post-0002.html" />
<meta property="og:site_name" content="ARSN Lab Notebook" />
<meta property="og:type" content="article" />
<meta property="article:published_time" content="2020-10-09T00:00:00-07:00" />
<script type="application/ld+json">
{"mainEntityOfPage":{"@type":"WebPage","@id":"http://localhost:4000/2020/10/09/post-0002.html"},"url":"http://localhost:4000/2020/10/09/post-0002.html","description":"Directory: https://ohsu.app.box.com/folder/81820136345","dateModified":"2020-10-09T00:00:00-07:00","datePublished":"2020-10-09T00:00:00-07:00","headline":"SSC Final Validation Tables with Annotations","@type":"BlogPosting","@context":"https://schema.org"}</script>
<!-- End Jekyll SEO tag -->
<link rel="stylesheet" href="/assets/main.css"><link type="application/atom+xml" rel="alternate" href="http://localhost:4000/feed.xml" title="ARSN Lab Notebook" /></head>
<body><header class="site-header" role="banner">

  <div class="wrapper"><a class="site-title" rel="author" href="/">ARSN Lab Notebook</a><nav class="site-nav">
        <input type="checkbox" id="nav-trigger" class="nav-trigger" />
        <label for="nav-trigger">
          <span class="menu-icon">
            <svg viewBox="0 0 18 15" width="18px" height="15px">
              <path d="M18,1.484c0,0.82-0.665,1.484-1.484,1.484H1.484C0.665,2.969,0,2.304,0,1.484l0,0C0,0.665,0.665,0,1.484,0 h15.032C17.335,0,18,0.665,18,1.484L18,1.484z M18,7.516C18,8.335,17.335,9,16.516,9H1.484C0.665,9,0,8.335,0,7.516l0,0 c0-0.82,0.665-1.484,1.484-1.484h15.032C17.335,6.031,18,6.696,18,7.516L18,7.516z M18,13.516C18,14.335,17.335,15,16.516,15H1.484 C0.665,15,0,14.335,0,13.516l0,0c0-0.82,0.665-1.483,1.484-1.483h15.032C17.335,12.031,18,12.695,18,13.516L18,13.516z"/>
            </svg>
          </span>
        </label>

        <div class="trigger"><a class="page-link" href="/about/">About</a></div>
      </nav></div>
</header>
<main class="page-content" aria-label="Content">
      <div class="wrapper">
        <article class="post h-entry" itemscope itemtype="http://schema.org/BlogPosting">

  <header class="post-header">
    <h1 class="post-title p-name" itemprop="name headline">SSC Final Validation Tables with Annotations</h1>
    <p class="post-meta">
      <time class="dt-published" datetime="2020-10-09T00:00:00-07:00" itemprop="datePublished">Oct 9, 2020
      </time></p>
  </header>

  <div class="post-content e-content" itemprop="articleBody">
    <p>Directory: <code class="language-plaintext highlighter-rouge">https://ohsu.app.box.com/folder/81820136345</code></p>

<p>Updated the SSC variant tables to include annotations derived from the columns.</p>

<p>The biggest discrepancies come from the borderline cases at AF 0.40-0.60 that are hard to call either mosaic or germline. The auto-annotations use the PHET values but can still differ in calls between LCL and WB. Also note that there are cases where it is identified as mosaic parental trans in LCL and mosaic parental nontrans in WB (or vice versa). Twin variant calls in pool 5 do not have twin information in this sheet.</p>

<p>An issue with these annotations are cases where the mutation shows up in small frequency in a non-main sample. This may be more than error and may be evidence of a trans parental mosaic mutation. We are currently only testing pzero on the main sample with the previously derived allele frequency. We could test for novel mosaics in other samples. For now, I am using a hard cutoff where a mutation must have of an allele frequency &gt; 0.01 (found in 1 of 100 reads).</p>

<p>Three examples taken from pool 1 where the originally identified mutation was in a child but from this validation could be a parental mosaic. The first two cases are germline in the child and could be parental mosaics. <a href="https://ohsu.app.box.com/file/728529690028">figure-01</a></p>


  </div><a class="u-url" href="/2020/10/09/post-0002.html" hidden></a>
</article>

      </div>
    </main><footer class="site-footer h-card">
  <data class="u-url" href="/"></data>

  <div class="wrapper">

    <h2 class="footer-heading">ARSN Lab Notebook</h2>

    <div class="footer-col-wrapper">
      <div class="footer-col footer-col-1">
        <ul class="contact-list">
          <li class="p-name">ARSN Lab Notebook</li><li><a class="u-email" href="mailto:nishidaa@ohsu.edu">nishidaa@ohsu.edu</a></li></ul>
      </div>

      <div class="footer-col footer-col-2"><ul class="social-media-list"><li><a href="https://github.com/arsnishida"><svg class="svg-icon"><use xlink:href="/assets/minima-social-icons.svg#github"></use></svg> <span class="username">arsnishida</span></a></li></ul>
</div>

      <div class="footer-col footer-col-3">
        <p>ARSN Lab notebook.</p>
      </div>
    </div>

  </div>

</footer>
</body>

</html>
