---
layout: post
title:  "scRNA-seq and scATAC-seq Data Analysis DREAM Challenge - Dataset 2 Data Generation"
date:   2022-4-21
parent: to_be_sorted
nav_order: 2
---

Program Versions:
```
macs2 v2.1.1.20160309
samtools v1.10
bedtools v2.22.0
python v2.7.9
perl v5.16.3
R v3.5.1
```

R Packages:
```
> sessionInfo()
R version 3.5.1 (2018-07-02)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: CentOS Linux 7 (Core)

Matrix products: default
BLAS: /home/groups/oroaklab/src/R/R-3.5.1/lib/libRblas.so
LAPACK: /home/groups/oroaklab/src/R/R-3.5.1/lib/libRlapack.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] grid      stats     graphics  grDevices utils     datasets  methods  
[8] base     

other attached packages:
[1] ComplexHeatmap_1.20.0 Matrix_1.2-18         cisTopic_0.3.0       
[4] Rphenograph_0.99.1    igraph_1.2.4.2        ggplot2_3.2.1 
```

STEP 1 - CALLING PEAKS
<br>INPUT FILE(s): `full_dataset.bam`, `downsampled_40.bam`
<br>OUTPUT FILE(s): `full_dataset.peaks.bed`, `downsampled_40.peaks.bed`

```
# call peaks, full
$ time macs2 callpeak --keep-dup all -t full_dataset.bam -n test1
real    8m37.638s
user    8m30.649s
sys     0m10.417s

# merge and filter peaks, full
$ perl 01_filter_peaks.pl test1_peaks.narrowPeak test1 500

# call peaks, downsampled
$ time macs2 callpeak --keep-dup all -t downsampled_40.bam -n test2
real    6m16.453s
user    5m54.677s
sys     0m7.381s

# merge and filter peaks, downsampled
$ perl 01.filter_peaks.pl test2_peaks.narrowPeak test2 500

$ md5sum *bed
c18d03f4d22e6938202e9578b2086dad  downsampled_40.500.bed
018e2e40df7f998d19df2f340bbb207c  full_dataset.peaks.bed
018e2e40df7f998d19df2f340bbb207c  test1.500.bed
c18d03f4d22e6938202e9578b2086dad  test2.500.bed
```

STEP 2 - MATRIX GENERATION
<br>INPUT FILE(s): `downsampled_40.bam`, `downsampled_40.500.bed`
<br>OUTPUT FILE(s): `downsampled_40.500.counts.sparseMatrix.cols.gz`, `downsampled_40.500.counts.sparseMatrix.rows.gz`, `downsampled_40.500.counts.sparseMatrix.values.gz`

```
time perl 02.make_count_matrix.pl downsampled_40.bam downsampled_40.500.bed test2
real    3m25.788s
user    5m30.598s
sys     0m8.572s

$ zcat downsampled_40.500.counts.sparseMatrix.values.gz > temp1
$ zcat test2.counts.sparseMatrix.values.gz > temp2
$ md5sum temp*
708ad4f6aabbbfed78771e3bc29fd5da  temp1
708ad4f6aabbbfed78771e3bc29fd5da  temp2
```

STEP 3 - DIMENSIONALITY REDUCTION WITH CISTOPICS3
<br>INPUT FILE(s): `downsampled_40.500.counts.sparseMatrix.values.gz`
<br>OUTPUT FILE(s): `downsampled_40.500.counts.cistopic.15.matrix`, `downsampled_40.500.counts.cistopic.20.matrix`, `downsampled_40.500.counts.cistopic.25.matrix`

```
$ time Rscript downsampled_40.500.counts.cistopic.r
real    29m28.503s
user    29m16.418s
sys     0m9.162s

# not similar by md5sum
```

STEP 4 - CISTOPIC3 MATRIX TO UMAP DIMS
<br>INPUT FILE(s): `downsampled_40.500.counts.cistopic.15.matrix`, `downsampled_40.500.counts.cistopic.20.matrix`, `downsampled_40.500.counts.cistopic.25.matrix`
<br>OUTPUT FILE(s): `downsampled_40.500.counts.cistopic.15.UMAP.dims`, `downsampled_40.500.counts.cistopic.20.UMAP.dims`, `downsampled_40.500.counts.cistopic.25.UMAP.dims`

```
$ time Rscript downsampled_40.500.counts.cistopic.15.UMAP.R
real    1m1.257s
user    0m59.971s
sys     0m1.055s

$ time Rscript downsampled_40.500.counts.cistopic.20.UMAP.R
real    1m0.584s
user    0m59.771s
sys     0m0.673s

$ time Rscript downsampled_40.500.counts.cistopic.25.UMAP.R
real    1m5.051s
user    1m3.943s
sys     0m0.939s

# not similar by md5sum
```

STEP 5 - CLUSTERING WITH PHENOGRAPHS
<br>INPUT FILE(s): `downsampled_40.500.counts.cistopic.15.matrix`, `downsampled_40.500.counts.cistopic.20.matrix`, `downsampled_40.500.counts.cistopic.25.matrix`
<br>OUTPUT FILE(s): `downsampled_40.500.counts.cistopic.15.k50.pg.annot`, `downsampled_40.500.counts.cistopic.20.k50.pg.annot`, `downsampled_40.500.counts.cistopic.25.k50.pg.annot`

```
$ time Rscript test2.15.k50.phenograph.R
real	0m24.443s
user	0m23.771s
sys	0m0.549s

$ time Rscript test2.20.k50.phenograph.R
real	0m26.302s
user	0m25.649s
sys	0m0.594s

$ time Rscript test2.25.k50.phenograph.R
real	0m25.757s
user	0m25.171s
sys	0m0.504s

$ md5sum *annot
af37ea4bec6586681b1c814c93a256ce  downsampled_40.500.counts.cistopic.15.k50.pg.annot
bedc6d7799fc618654e63450691cc9b7  downsampled_40.500.counts.cistopic.20.k50.pg.annot
4915ad625b38b70ac7d3f969c4159a97  downsampled_40.500.counts.cistopic.25.k50.pg.annot
af37ea4bec6586681b1c814c93a256ce  test2.15.k50.pg.annot
bedc6d7799fc618654e63450691cc9b7  test2.20.k50.pg.annot
4915ad625b38b70ac7d3f969c4159a97  test2.25.k50.pg.annot
```
