<!DOCTYPE html>
<html lang="en"><head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1"><!-- Begin Jekyll SEO tag v2.6.1 -->
<title>Scitools scATAC Metrics | ARSN Lab Notebook</title>
<meta name="generator" content="Jekyll v4.1.1" />
<meta property="og:title" content="Scitools scATAC Metrics" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="TSS enrichment at a single cell level: Calculated as the ratio of signal to noise. Signal is defined as the region around TSS +/- 100bp. Noise is defined as the 100bp regions +/- 2000bp from TSS regions that do no overlap with signal. This leads signal and noise being slightly different in size but reads are uniquely counted into a single bin versus other methods where a read can fall into multiple TSS and noise regions. Output is a two-column value file. scitools bam_tssenrich [bam] [hg38 or mm10 or dm6]" />
<meta property="og:description" content="TSS enrichment at a single cell level: Calculated as the ratio of signal to noise. Signal is defined as the region around TSS +/- 100bp. Noise is defined as the 100bp regions +/- 2000bp from TSS regions that do no overlap with signal. This leads signal and noise being slightly different in size but reads are uniquely counted into a single bin versus other methods where a read can fall into multiple TSS and noise regions. Output is a two-column value file. scitools bam_tssenrich [bam] [hg38 or mm10 or dm6]" />
<link rel="canonical" href="http://localhost:4000/2020/12/17/post-0005.html" />
<meta property="og:url" content="http://localhost:4000/2020/12/17/post-0005.html" />
<meta property="og:site_name" content="ARSN Lab Notebook" />
<meta property="og:type" content="article" />
<meta property="article:published_time" content="2020-12-17T00:00:00-08:00" />
<script type="application/ld+json">
{"headline":"Scitools scATAC Metrics","dateModified":"2020-12-17T00:00:00-08:00","datePublished":"2020-12-17T00:00:00-08:00","url":"http://localhost:4000/2020/12/17/post-0005.html","mainEntityOfPage":{"@type":"WebPage","@id":"http://localhost:4000/2020/12/17/post-0005.html"},"description":"TSS enrichment at a single cell level: Calculated as the ratio of signal to noise. Signal is defined as the region around TSS +/- 100bp. Noise is defined as the 100bp regions +/- 2000bp from TSS regions that do no overlap with signal. This leads signal and noise being slightly different in size but reads are uniquely counted into a single bin versus other methods where a read can fall into multiple TSS and noise regions. Output is a two-column value file. scitools bam_tssenrich [bam] [hg38 or mm10 or dm6]","@type":"BlogPosting","@context":"https://schema.org"}</script>
<!-- End Jekyll SEO tag -->
<link rel="stylesheet" href="/assets/main.css"><link type="application/atom+xml" rel="alternate" href="http://localhost:4000/feed.xml" title="ARSN Lab Notebook" /></head>
<body><header class="site-header" role="banner">

  <div class="wrapper"><a class="site-title" rel="author" href="/">ARSN Lab Notebook</a><nav class="site-nav">
        <input type="checkbox" id="nav-trigger" class="nav-trigger" />
        <label for="nav-trigger">
          <span class="menu-icon">
            <svg viewBox="0 0 18 15" width="18px" height="15px">
              <path d="M18,1.484c0,0.82-0.665,1.484-1.484,1.484H1.484C0.665,2.969,0,2.304,0,1.484l0,0C0,0.665,0.665,0,1.484,0 h15.032C17.335,0,18,0.665,18,1.484L18,1.484z M18,7.516C18,8.335,17.335,9,16.516,9H1.484C0.665,9,0,8.335,0,7.516l0,0 c0-0.82,0.665-1.484,1.484-1.484h15.032C17.335,6.031,18,6.696,18,7.516L18,7.516z M18,13.516C18,14.335,17.335,15,16.516,15H1.484 C0.665,15,0,14.335,0,13.516l0,0c0-0.82,0.665-1.483,1.484-1.483h15.032C17.335,12.031,18,12.695,18,13.516L18,13.516z"/>
            </svg>
          </span>
        </label>

        <div class="trigger"><a class="page-link" href="/about/">About</a></div>
      </nav></div>
</header>
<main class="page-content" aria-label="Content">
      <div class="wrapper">
        <article class="post h-entry" itemscope itemtype="http://schema.org/BlogPosting">

  <header class="post-header">
    <h1 class="post-title p-name" itemprop="name headline">Scitools scATAC Metrics</h1>
    <p class="post-meta">
      <time class="dt-published" datetime="2020-12-17T00:00:00-08:00" itemprop="datePublished">Dec 17, 2020
      </time></p>
  </header>

  <div class="post-content e-content" itemprop="articleBody">
    <p><strong>TSS enrichment at a single cell level:</strong>
<br />Calculated as the ratio of signal to noise. Signal is defined as the region around TSS +/- 100bp. Noise is defined as the 100bp regions +/- 2000bp from TSS regions that do no overlap with signal. This leads signal and noise being slightly different in size but reads are uniquely counted into a single bin versus other methods where a read can fall into multiple TSS and noise regions. Output is a two-column value file.
<br /><code class="language-plaintext highlighter-rouge">scitools bam_tssenrich [bam] [hg38 or mm10 or dm6]</code></p>

<p><strong>TSS enrichment at bulk level, ENCODE:</strong>
<br />Calculated via ENCODE standards, <a href="https://www.encodeproject.org/data-standards/terms/#enrichment">ENCODE ENRICHMENT</a>
<br /><code class="language-plaintext highlighter-rouge">scitools bam_tssenrich -E [bam] [hg38 or mm10 or dm6]</code>
<br />Example enrichment output figure: <a href="https://ohsu.app.box.com/file/754659307104">figure-01</a></p>

<p>TSS bulk calculation starts with a bam2bed conversion and is potentially intensive depending on the size of the BAM. A parallelizable version of the bulk TSS calculation can be performed on exacloud.</p>

<p><strong>TSS enrichment at bulk level, inflated:</strong>
<br />This uses a larger distance from TSS sites, larger bins combining more reads, and chooses highest positions via, <a href="https://www.biorxiv.org/content/10.1101/2020.05.10.087585v1">this article</a>
<br /><code class="language-plaintext highlighter-rouge">scitools bam_tssenrich -E -r 2000 -n 11 -s MAX [bam] [hg38 or mm10 or dm6]</code></p>

<p><strong>TSS sites are gathered from ENSEMBL and are curated as such:</strong>
<code class="language-plaintext highlighter-rouge">/home/groups/oroaklab/refs/hg38/ensembl_tss/readme.txt</code>
<code class="language-plaintext highlighter-rouge">/home/groups/oroaklab/refs/mm10/ensembl_tss/readme.txt</code>
<code class="language-plaintext highlighter-rouge">/home/groups/oroaklab/refs/dm6/ensembl_tss/readme.txt</code></p>

<p><strong>Independent FRIP caller:</strong>
<br />This FRIP calculation is divorced from scitools atac_count and is slightly more accurate. The existing FRIP calculation that occurs during matrix generation double-counts reads that overlap multiple peaks.
<br /><code class="language-plaintext highlighter-rouge">scitools bam_frip [bam] [bed]</code></p>


  </div><a class="u-url" href="/2020/12/17/post-0005.html" hidden></a>
</article>

      </div>
    </main><footer class="site-footer h-card">
  <data class="u-url" href="/"></data>

  <div class="wrapper">

    <h2 class="footer-heading">ARSN Lab Notebook</h2>

    <div class="footer-col-wrapper">
      <div class="footer-col footer-col-1">
        <ul class="contact-list">
          <li class="p-name">ARSN Lab Notebook</li><li><a class="u-email" href="mailto:nishidaa@ohsu.edu">nishidaa@ohsu.edu</a></li></ul>
      </div>

      <div class="footer-col footer-col-2"><ul class="social-media-list"><li><a href="https://github.com/arsnishida"><svg class="svg-icon"><use xlink:href="/assets/minima-social-icons.svg#github"></use></svg> <span class="username">arsnishida</span></a></li></ul>
</div>

      <div class="footer-col footer-col-3">
        <p>ARSN Lab notebook.</p>
      </div>
    </div>

  </div>

</footer>
</body>

</html>
