<!DOCTYPE html> <html lang="en-US"> <head> <meta charset="UTF-8"> <meta http-equiv="X-UA-Compatible" content="IE=Edge"> <title>Local Installs - Initial - ARSN Work Blog</title> <link rel="shortcut icon" href="/favicon.ico" type="image/x-icon"> <link rel="stylesheet" href="/assets/css/just-the-docs-default.css"> <script src="/assets/js/vendor/lunr.min.js"></script> <script src="/assets/js/just-the-docs.js"></script> <meta name="viewport" content="width=device-width, initial-scale=1"> <!-- Begin Jekyll SEO tag v2.8.0 --> <title>Local Installs - Initial | ARSN Work Blog</title> <meta name="generator" content="Jekyll v4.3.0" /> <meta property="og:title" content="Local Installs - Initial" /> <meta property="og:locale" content="en_US" /> <meta name="description" content="This is a blog for work projects Andrew Nishida, nishidaa@ohsu.edu, in CEDAR at OHSU." /> <meta property="og:description" content="This is a blog for work projects Andrew Nishida, nishidaa@ohsu.edu, in CEDAR at OHSU." /> <link rel="canonical" href="http://localhost:4000/docs/ravnica/2022-12-05-ravnica-local_install_initial.html" /> <meta property="og:url" content="http://localhost:4000/docs/ravnica/2022-12-05-ravnica-local_install_initial.html" /> <meta property="og:site_name" content="ARSN Work Blog" /> <meta property="og:type" content="article" /> <meta property="article:published_time" content="2022-12-05T00:00:00-08:00" /> <meta name="twitter:card" content="summary" /> <meta property="twitter:title" content="Local Installs - Initial" /> <script type="application/ld+json"> {"@context":"https://schema.org","@type":"BlogPosting","dateModified":"2022-12-05T00:00:00-08:00","datePublished":"2022-12-05T00:00:00-08:00","description":"This is a blog for work projects Andrew Nishida, nishidaa@ohsu.edu, in CEDAR at OHSU.","headline":"Local Installs - Initial","mainEntityOfPage":{"@type":"WebPage","@id":"http://localhost:4000/docs/ravnica/2022-12-05-ravnica-local_install_initial.html"},"url":"http://localhost:4000/docs/ravnica/2022-12-05-ravnica-local_install_initial.html"}</script> <!-- End Jekyll SEO tag --> </head> <body> <a class="skip-to-main" href="#main-content">Skip to main content</a> <svg xmlns="http://www.w3.org/2000/svg" style="display: none;"> <symbol id="svg-link" viewBox="0 0 24 24"> <title>Link</title> <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-link"> <path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path> </svg> </symbol> <symbol id="svg-search" viewBox="0 0 24 24"> <title>Search</title> <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-search"> <circle cx="11" cy="11" r="8"></circle><line x1="21" y1="21" x2="16.65" y2="16.65"></line> </svg> </symbol> <symbol id="svg-menu" viewBox="0 0 24 24"> <title>Menu</title> <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-menu"> <line x1="3" y1="12" x2="21" y2="12"></line><line x1="3" y1="6" x2="21" y2="6"></line><line x1="3" y1="18" x2="21" y2="18"></line> </svg> </symbol> <symbol id="svg-arrow-right" viewBox="0 0 24 24"> <title>Expand</title> <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-chevron-right"> <polyline points="9 18 15 12 9 6"></polyline> </svg> </symbol> <symbol id="svg-doc" viewBox="0 0 24 24"> <title>Document</title> <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-file"> <path d="M13 2H6a2 2 0 0 0-2 2v16a2 2 0 0 0 2 2h12a2 2 0 0 0 2-2V9z"></path><polyline points="13 2 13 9 20 9"></polyline> </svg> </symbol> <!-- Feather. MIT License: https://github.com/feathericons/feather/blob/master/LICENSE --> <symbol id="svg-external-link" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-external-link"> <title id="svg-external-link-title">(external link)</title> <path d="M18 13v6a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V8a2 2 0 0 1 2-2h6"></path><polyline points="15 3 21 3 21 9"></polyline><line x1="10" y1="14" x2="21" y2="3"></line> </symbol> </svg> <div class="side-bar"> <div class="site-header"> <a href="/" class="site-title lh-tight"> ARSN Work Blog </a> <a href="#" id="menu-button" class="site-button"> <svg viewBox="0 0 24 24" class="icon"><use xlink:href="#svg-menu"></use></svg> </a> </div> <nav aria-label="Main" id="site-nav" class="site-nav"> <ul class="nav-list"><li class="nav-list-item"><a href="/" class="nav-list-link">Home</a></li><li class="nav-list-item active"><a href="#" class="nav-list-expander" aria-label="toggle links in ravnica category"> <svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg> </a><a href="/docs/ravnica/" class="nav-list-link">ravnica</a><ul class="nav-list "><li class="nav-list-item "><a href="/docs/ravnica/2022-12-05-ravnica-global_install_initial.html" class="nav-list-link">Global Installs - Initial</a></li><li class="nav-list-item active"><a href="/docs/ravnica/2022-12-05-ravnica-local_install_initial.html" class="nav-list-link active">Local Installs - Initial</a></li><li class="nav-list-item "><a href="/docs/ravnica/2022-12-05-ravnica-mamba_install_initial.html" class="nav-list-link">Mamba Install - Initial</a></li><li class="nav-list-item "><a href="/docs/ravnica/2022-12-05-ravnica-mamba_setup.html" class="nav-list-link">Mamba Setup</a></li><li class="nav-list-item "><a href="/docs/ravnica/2022-12-05-ravnica-r_4.2.2_library_info.html" class="nav-list-link">R 4.2.2 Library Info</a></li></ul></li><li class="nav-list-item"><a href="#" class="nav-list-expander" aria-label="toggle links in to_be_sorted category"> <svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg> </a><a href="/docs/to_be_sorted/" class="nav-list-link">to_be_sorted</a><ul class="nav-list "><li class="nav-list-item "><a href="/docs/to_be_sorted/2020-10-09-post-0002.html" class="nav-list-link">SSC Final Validation Tables with Annotations</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2020-10-09-post-0003.html" class="nav-list-link">Two Week Update: 2020-10-09</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2020-11-09-post-0004.html" class="nav-list-link">Two Week Update: 2020-11-09</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2020-12-17-post-0005.html" class="nav-list-link">Scitools scATAC Metrics</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-02-18-post-0006.html" class="nav-list-link">Scitools Scrublet Function</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-05-05-post-0007.html" class="nav-list-link">NextSeq Fidelity Evaluation</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-05-07-post-0008.html" class="nav-list-link">Single Cell TBR1 Preliminary Evaluations</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-05-08-post-0009.html" class="nav-list-link">Single Cell Alignments with no Alt Chromosomes</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-05-09-post-0010.html" class="nav-list-link">Single Cell TBR1 Coembedding Tests</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-05-11-post-0011.html" class="nav-list-link">Single Cell TBR1 ENCODE DHS Index Tests</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-05-11-post-0012.html" class="nav-list-link">SPARK Mosaic List, Summary Counts of Annotations</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-05-27-post-0013.html" class="nav-list-link">Single Cell TBR1 ENCODE DHS Index Final UMAP and Clusters</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-05-28-post-0014.html" class="nav-list-link">Single Cell TBR1 Cluster DA</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-05-28-post-0015.html" class="nav-list-link">SPARK RefSeq CDS and Splice Site Reference Generation</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-05-30-post-0016.html" class="nav-list-link">SPARK Joint Coverage Counter Tool Fidelity</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-05-30-post-0017.html" class="nav-list-link">SPARK Joint Coverage Plots</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-06-02-post-0018.html" class="nav-list-link">SPARK Joint Coverage Table Fix</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-06-03-post-0019.html" class="nav-list-link">SPARK Mosaic List FRACTION_MORE Annotations</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-06-03-post-0020.html" class="nav-list-link">SPARK Mosaic Index</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-06-10-post-0021.html" class="nav-list-link">SPARK Gene Set Joint Coverage</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-06-10-post-0022.html" class="nav-list-link">Single Cell TBR1 Cicero/Monocle3 with TSS > 2.7</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-06-22-post-0023.html" class="nav-list-link">SPARK Preliminary Pass 01, Filtering and Classifications</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-06-22-post-0024.html" class="nav-list-link">SPARK Preliminary Pass 02, Loading for Coverage and SNV Count Plots</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-06-26-post-0025.html" class="nav-list-link">SPARK Preliminary Pass 02b, Coverage and SNV Count Plots as Hists</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-06-28-post-0026.html" class="nav-list-link">SPARK Preliminary Pass 03, Allele Fraction Histograms</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-07-06-post-0027.html" class="nav-list-link">SPARK Preliminary Pass 04, Coverages and SNV Counts with Joint Family Coverages</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-07-09-post-0028.html" class="nav-list-link">Single Cell TBR1 ChromVAR with TSS > 2.7</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-07-12-post-0029.html" class="nav-list-link">Single Cell TBR1 ChromVAR with TSS > 1.5</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-07-14-post-0030.html" class="nav-list-link">SPARK Preliminary Pass 5, Final Filtering</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-07-17-post-0031.html" class="nav-list-link">SPARK Preliminary Pass 6a and 6b, Simplex Trio Coverages, SNV Counts, and AFs</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-07-17-post-0032.html" class="nav-list-link">SPARK Preliminary Pass 7a and 7b, Multiplex Quad Coverages, SNV Counts, and AFs</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-07-18-post-0033.html" class="nav-list-link">Single Cell TBR1 BioCluster/Genotype/Line/Time DA with TSS > 2.7</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-07-19-post-0034.html" class="nav-list-link">Single Cell TBR1 Misc Figures - ChromVAR Compare, FDR Hists, 3D</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-07-26-post-0035.html" class="nav-list-link">Single Cell TBR1 BioCluster/Genotype/Line/Time DA with TSS > 2.7, Filtering Ambiguous Groups</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-07-26-post-0036.html" class="nav-list-link">SPARK First Pass, Synonymous SNV Splice Distances</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-07-26-post-0037.html" class="nav-list-link">SPARK Preliminary Outlier Detection, Secondary Pass</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-07-26-post-0038.html" class="nav-list-link">SPARK Final Table Creation</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-07-27-post-0039.html" class="nav-list-link">SPARK Mosaic Index, July 27</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-07-29-post-0040.html" class="nav-list-link">Single Cell TBR1 Bigwig Tracks</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-08-03-post-0041.html" class="nav-list-link">SPARK High Confidence Allele Frequencies 01</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-08-03-post-0042.html" class="nav-list-link">SPARK High Confidence Synonymous Splice Distances 01</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-08-03-post-0043.html" class="nav-list-link">SPARK Burden Tables Autosomal 01</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-08-09-post-0044.html" class="nav-list-link">SPARK Burden Tables Gene Sets 01</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-08-12-post-0045.html" class="nav-list-link">Single Cell TBR1 Cistopic Boxplots and UMAPs</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-08-12-post-0046.html" class="nav-list-link">Single Cell TBR1 ChIP-Seq Peak DA</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-08-26-post-0047.html" class="nav-list-link">SPARK IGV Shots of High Confident Calls</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-09-08-post-0048.html" class="nav-list-link">SPARK Burden Tables Gene Sets 02</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-09-14-post-0049.html" class="nav-list-link">SPARK Burden Tables Gene Sets 03</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-09-15-post-0050.html" class="nav-list-link">SPARK Concatenated High Confidence Table Checks 01</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-10-01-post-0051.html" class="nav-list-link">SPARK Final Sample List</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-10-01-post-0052.html" class="nav-list-link">SPARK Final IGV and Cohort Count Filter</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-10-06-post-0053.html" class="nav-list-link">SPARK LGD and NS List</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-10-06-post-0054.html" class="nav-list-link">SPARK Burden Tables Gene Sets 04</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-10-07-post-0055.html" class="nav-list-link">Drosophilia Tau Preliminary DA</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-10-10-post-0056.html" class="nav-list-link">SPARK Summary of Final Lists</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-10-10-post-0057.html" class="nav-list-link">SPARK Final List Allele Frequency Distributions</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-10-11-post-0058.html" class="nav-list-link">SPARK Monozygotic Twins in Quad Families</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-10-11-post-0059.html" class="nav-list-link">SPARK Burden Tables Gene Sets, Final 01</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-10-13-post-0060.html" class="nav-list-link">SPARK Burden w/ REVEL Scores Gene Sets, 01</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-10-13-post-0061.html" class="nav-list-link">SPARK Parental Trans Mosaics Breakdown</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-10-13-post-0062.html" class="nav-list-link">SPARK HC Tables Against SPARK 167</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-10-14-post-0063.html" class="nav-list-link">SPARK Concat Variant Lists to DNV List Comparison</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-11-04-post-0064.html" class="nav-list-link">Contamination References</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-11-28-post-0065.html" class="nav-list-link">Single Cell TBR1 Pseudo-Bulk Depth Tests</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-11-30-post-0066.html" class="nav-list-link">SPARK SSC List Formatting to Current SPARK</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-11-30-post-0067.html" class="nav-list-link">SPARK Burden Tables Gene Sets, SPARK+SSC</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2021-12-03-post-0068.html" class="nav-list-link">SPARK VEP Annotations</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2022-01-08-post-0069.html" class="nav-list-link">Drosophilia Tau Final DA</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2022-01-18-post-0070.html" class="nav-list-link">SPARK SSC hg38 Transformed Burden, 01</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2022-01-19-post-0071.html" class="nav-list-link">SPARK Pre Best Practices Filter, Model Scores</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2022-01-21-post-0072.html" class="nav-list-link">Drosophilia Tau DA Results Test Analysis</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2020-10-07-post-0001.html" class="nav-list-link">Example</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2022-03-01-post-0074.html" class="nav-list-link">OneDrive Upload and Download with Rclone</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2022-03-07-post-0075.html" class="nav-list-link">Shotgun Sequencing of Amplicon Sequence, 220305</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2022-03-19-post-0076.html" class="nav-list-link">SPARK WES2 and WES3 List Inconsistencies</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2022-04-03-post-0077.html" class="nav-list-link">scRNA-seq First Pass Pipeline - 220324_HB_Coassay_RNA</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2022-04-06-post-0078.html" class="nav-list-link">Shotgun Sequencing of Amplicon Sequence, 220406</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2022-04-08-post-0079.html" class="nav-list-link">SPARK WES2 and WES3 Outliers by Counts and Coverage</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2022-04-11-post-0080.html" class="nav-list-link">scRNA-seq Second Pass Pipeline - 220324_HB_Coassay_RNA</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2022-04-21-post-0081.html" class="nav-list-link">scRNA-seq and scATAC-seq Data Analysis DREAM Challenge - Dataset 2 Data Generation</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2022-04-28-post-0082.html" class="nav-list-link">SPARK Mosaic Index, 22-04-28</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2022-05-03-post-0083.html" class="nav-list-link">SPARK IGV HC List Screenshots, 2022</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2022-05-18-post-0084.html" class="nav-list-link">Celsee and Docker Info</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2022-05-18-post-0085.html" class="nav-list-link">Example of HDR Processing via BWA Alignment</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2022-05-18-post-0086.html" class="nav-list-link">Cron Info for Automatically Running bcl2fastq</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2022-05-18-post-0087.html" class="nav-list-link">SRA Download Info for 'TBR1 Regulates Autism Risk Genes in the Developing Neocortex'</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2022-05-20-post-0088.html" class="nav-list-link">SPARK Post-Hoc Cohort, IGV, and iWES Filter Lists</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2022-05-20-post-0089.html" class="nav-list-link">Final O'Roak Lab Data Locations</a></li><li class="nav-list-item "><a href="/docs/to_be_sorted/2022-01-25-post-0073.html" class="nav-list-link">Dropbox Upload and Download with Curl and API Key</a></li></ul></li></ul> </nav> <footer class="site-footer"> This site uses <a href="https://github.com/just-the-docs/just-the-docs">Just the Docs</a>, a documentation theme for Jekyll. </footer> </div> <div class="main" id="top"> <div id="main-header" class="main-header"> <div class="search"> <div class="search-input-wrap"> <input type="text" id="search-input" class="search-input" tabindex="0" placeholder="Search ARSN Work Blog" aria-label="Search ARSN Work Blog" autocomplete="off"> <label for="search-input" class="search-label"><svg viewBox="0 0 24 24" class="search-icon"><use xlink:href="#svg-search"></use></svg></label> </div> <div id="search-results" class="search-results"></div> </div> <nav aria-label="Auxiliary" class="aux-nav"> <ul class="aux-nav-list"> <li class="aux-nav-list-item"> <a href="//github.com/just-the-docs/just-the-docs" class="site-button" > Just the Docs on GitHub </a> </li> </ul> </nav> </div> <div id="main-content-wrap" class="main-content-wrap"> <nav aria-label="Breadcrumb" class="breadcrumb-nav"> <ol class="breadcrumb-nav-list"> <li class="breadcrumb-nav-list-item"><a href="/docs/ravnica/">ravnica</a></li> <li class="breadcrumb-nav-list-item"><span>Local Installs - Initial</span></li> </ol> </nav> <div id="main-content" class="main-content" role="main"> <p>Install cmake</p> <div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>tar xfz cmake-3.24.3.tar.gz
cd cmake-3.24.3/
./bootstrap --prefix=/home/groups/ravnica/src/cmake/cmake-3.24.3
make
make install
</code></pre></div></div> <p>Install boost</p> <div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>tar xfz boost_1_80_0.tar.gz
cd boost_1_80_0
./bootstrap.sh --prefix=/home/groups/ravnica/src/boost/boost_1_80_0
./b2 install
</code></pre></div></div> <p>Install bedtools</p> <div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>wget https://github.com/arq5x/bedtools2/releases/download/v2.29.1/bedtools-2.29.1.tar.gz
tar -zxvf bedtools-2.29.1.tar.gz
mv bedtools2/ bedtools2.29.1
cd bedtools2.29.1/
make
</code></pre></div></div> <p>Install bedops</p> <div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>bzip2 -dk bedops_linux_x86_64-v2.4.41.tar.bz2 
tar xf bedops_linux_x86_64-v2.4.41.tar
rm bedops_linux_x86_64-v2.4.41.tar
mkdir 2.4.41
mv bin 2.4.41
</code></pre></div></div> <p>Install samtools</p> <div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>bzip2 -dk samtools-1.16.1.tar.bz2
tar xf samtools-1.16.1.tar
rm samtools-1.16.1.tar
cd samtools-1.16.1
./configure --prefix=/home/groups/ravnica/src/samtools/samtools-1.16.1
make
make install
</code></pre></div></div> <p>Install htslib</p> <div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>bzip2 -dk htslib-1.16.tar.bz2 
tar xf htslib-1.16.tar
rm htslib-1.16.tar
cd htslib-1.16
./configure --prefix=/home/groups/ravnica/src/htslib/htslib-1.16/
make
make install
</code></pre></div></div> <p>Install bcftools</p> <div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>bzip2 -dk bcftools-1.16.tar.bz2
tar xf bcftools-1.16.tar
rm bcftools-1.16.tar
cd bcftools-1.16
./configure --prefix=/home/groups/ravnica/src/bcftools/bcftools-1.16
make
make install
</code></pre></div></div> <p>Install install sratoolkit</p> <div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>tar xfz sratoolkit.3.0.0-ubuntu64.tar.gz
</code></pre></div></div> <p>Install subread</p> <div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>tar xfz subread-2.0.3-source.tar.gz
cd subread-2.0.3-source/src
make -f Makefile.Linux
</code></pre></div></div> <p>Install STAR</p> <div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>wget https://github.com/alexdobin/STAR/archive/2.7.10b.tar.gz
tar -xzf 2.7.10b.tar.gz
cd STAR-2.7.10b/source
make STAR
</code></pre></div></div> <p>Install bwa, pulled nov14 from master branch 139f68f on Sep 22. There’s a bug with gcc versions that the tagged release cannot handle.</p> <div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>git clone https://github.com/lh3/bwa.git
mv bwa bwa-139f68f-0.7.17
cd bwa-139f68f-0.7.17
make
</code></pre></div></div> <p>Install BLAST</p> <div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>tar xvf ncbi-blast-2.13.0+-src.tar.gz
cd ncbi-blast-2.13.0+-src/c++
./configure --prefix=/home/groups/ravnica/src/blast/ncbi-blast-2.13.0+-src
make
make install
</code></pre></div></div> <p>Install kraken2</p> <div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>tar xvf kraken2-2.1.2.tar.gz
module load blast
./install_kraken2.sh /home/groups/ravnica/src/kraken/kraken2-2.1.2
</code></pre></div></div> <p>Install GMP</p> <div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>tar xf gmp-6.1.0.tar
make
make install
</code></pre></div></div> <p>Install MPFR</p> <div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>./configure --prefix=/home/groups/ravnica/src/mpfr/mpfr-3.1.4 --with-gmp-build=/home/groups/ravnica/src/gmp/gmp-6.1.0
make
make install
</code></pre></div></div> <p>Install MPC</p> <div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>tar xf mpc-1.0.3.tar
./configure --prefix=/home/groups/ravnica/src/mpc/mpc-1.0.3 --with-gmp=/home/groups/ravnica/src/gmp/gmp-6.1.0 --with-mpfr=/home/groups/ravnica/src/mpfr/mpfr-3.1.4
make
make install
</code></pre></div></div> <p>Install this older version of gcc for bcl2fastq.</p> <div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>tar xfz gcc-8.1.0.tar.gz
./configure --prefix=/home/groups/ravnica/src/gcc/gcc-8.1.0 --with-gmp=/home/groups/ravnica/src/gmp/gmp-6.1.0 --with-mpfr=/home/groups/ravnica/src/mpfr/mpfr-3.1.4 --with-mpc=/home/groups/ravnica/src/mpc/mpc-1.0.3 --disable-multilib --disable-libsanitizer
make
make install
</code></pre></div></div> <p>Separetely install older cmake also for bcl2fastq. There’s a minor issue with the order in which the module should be loaded. I think it has to be done after bootstrapping, not before like on my notes.</p> <div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>tar xfz cmake-2.8.9.tar.gz
cd cmake-2.8.9/
module load gcc/8.1.0
export LD_LIBRARY_PATH=/usr/lib/x86_64-linux-gnu:/home/groups/ravnica/src/gmp/gmp-6.1.0/lib:/home/groups/ravnica/src/mpfr/mpfr-3.1.4/lib:/home/groups/ravnica/src/mpc/mpc-1.0.3/lib:/home/groups/ravnica/src/gcc/gcc-8.1.0/lib64
./bootstrap
make
make install
</code></pre></div></div> <p>Install bcl2fastq. bcl2fastq will try to install its specific versions of cmake and boost. cmake-2.8.9 needs an older version of gcc so install separately. But let it install its own version of boost. Add an explicit path to C libraries since cmake-2.8.9 is needed but all its C libraries won’t work.</p> <div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>tar xfz bcl2fastq2-v2.20.0.422-Source.tar.gz
mv bcl2fastq bcl2fastq-v2.20.0.422
module load cmake/2.8.9
export C_INCLUDE_PATH=/usr/include/x86_64-linux-gnu
./src/configure --prefix=/home/groups/ravnica/src/bcl2fastq2/bcl2fastq2-v2.20.0.422
make
make install
</code></pre></div></div> <p>Install older autoconf</p> <div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>curl -O http://ftp.gnu.org/gnu/autoconf/autoconf-2.69.tar.gz
tar zxvf autoconf-2.69.tar.gz
cd autoconf-2.69
./configure &amp;&amp; make &amp;&amp; sudo make install
</code></pre></div></div> <p>Install pcre2</p> <div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>tar xfz pcre2-10.40.tar.gz 
./configure --prefix=""
make
make install
</code></pre></div></div> <p>Install Java</p> <div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>tar xfz openjdk-11.0.2_linux-x64_bin.tar.gz 
# that’s it
</code></pre></div></div> <p>Install R, 4.2.2</p> <div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>tar xfz R-4.2.2.tar.gz
cd R-4.2.2/
module load pcre2/10.40
./configure --prefix="/home/groups/ravnica/src/R/R-4.2.2" --with-cairo --with-libpng --with-libtiff --with-jpeglib
make
make install
</code></pre></div></div> <p>Install R, 4.2.1</p> <div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>tar xfz R-4.2.1.tar.gz
cd R-4.2.1/
module load pcre2/10.40
./configure --prefix="/home/groups/ravnica/src/R/R-4.2.1" --with-cairo --with-libpng --with-libtiff --with-jpeglib
make
make install
</code></pre></div></div> <p>Install JAGS</p> <div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>tar xfz JAGS-4.3.1.tar.gz
cd JAGS-4.3.1
./configure --prefix="/home/groups/ravnica/src/jags/JAGS-4.3.1"
make
make install
</code></pre></div></div> </div> </div> <div class="search-overlay"></div> </div> </body> </html>
