<!DOCTYPE html> <html lang="en-US"> <head> <meta charset="UTF-8"> <meta http-equiv="X-UA-Compatible" content="IE=Edge"> <title>23-02-07, Feature Sets - ARSN Work Blog</title> <link rel="shortcut icon" href="/favicon.ico" type="image/x-icon"> <link rel="stylesheet" href="/assets/css/just-the-docs-default.css"> <script src="/assets/js/vendor/lunr.min.js"></script> <script src="/assets/js/just-the-docs.js"></script> <meta name="viewport" content="width=device-width, initial-scale=1"> <!-- Begin Jekyll SEO tag v2.8.0 --> <title>23-02-07, Feature Sets | ARSN Work Blog</title> <meta name="generator" content="Jekyll v4.3.0" /> <meta property="og:title" content="23-02-07, Feature Sets" /> <meta property="og:locale" content="en_US" /> <meta name="description" content="This is a blog for work projects Andrew Nishida, nishidaa@ohsu.edu, in CEDAR at OHSU." /> <meta property="og:description" content="This is a blog for work projects Andrew Nishida, nishidaa@ohsu.edu, in CEDAR at OHSU." /> <link rel="canonical" href="http://localhost:4000/docs/sc_klf4bc/2023-02-07-sc_klf4bc-feature_sets.html" /> <meta property="og:url" content="http://localhost:4000/docs/sc_klf4bc/2023-02-07-sc_klf4bc-feature_sets.html" /> <meta property="og:site_name" content="ARSN Work Blog" /> <meta property="og:type" content="article" /> <meta property="article:published_time" content="2023-02-07T00:00:00-08:00" /> <meta name="twitter:card" content="summary" /> <meta property="twitter:title" content="23-02-07, Feature Sets" /> <script type="application/ld+json"> {"@context":"https://schema.org","@type":"BlogPosting","dateModified":"2023-02-07T00:00:00-08:00","datePublished":"2023-02-07T00:00:00-08:00","description":"This is a blog for work projects Andrew Nishida, nishidaa@ohsu.edu, in CEDAR at OHSU.","headline":"23-02-07, Feature Sets","mainEntityOfPage":{"@type":"WebPage","@id":"http://localhost:4000/docs/sc_klf4bc/2023-02-07-sc_klf4bc-feature_sets.html"},"url":"http://localhost:4000/docs/sc_klf4bc/2023-02-07-sc_klf4bc-feature_sets.html"}</script> <!-- End Jekyll SEO tag --> </head> <body> <a class="skip-to-main" href="#main-content">Skip to main content</a> <svg xmlns="http://www.w3.org/2000/svg" style="display: none;"> <symbol id="svg-link" viewBox="0 0 24 24"> <title>Link</title> <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-link"> <path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path> </svg> </symbol> <symbol id="svg-search" viewBox="0 0 24 24"> <title>Search</title> <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-search"> <circle cx="11" cy="11" r="8"></circle><line x1="21" y1="21" x2="16.65" y2="16.65"></line> </svg> </symbol> <symbol id="svg-menu" viewBox="0 0 24 24"> <title>Menu</title> <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-menu"> <line x1="3" y1="12" x2="21" y2="12"></line><line x1="3" y1="6" x2="21" y2="6"></line><line x1="3" y1="18" x2="21" y2="18"></line> </svg> </symbol> <symbol id="svg-arrow-right" viewBox="0 0 24 24"> <title>Expand</title> <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-chevron-right"> <polyline points="9 18 15 12 9 6"></polyline> </svg> </symbol> <symbol id="svg-doc" viewBox="0 0 24 24"> <title>Document</title> <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-file"> <path d="M13 2H6a2 2 0 0 0-2 2v16a2 2 0 0 0 2 2h12a2 2 0 0 0 2-2V9z"></path><polyline points="13 2 13 9 20 9"></polyline> </svg> </symbol> <!-- Feather. MIT License: https://github.com/feathericons/feather/blob/master/LICENSE --> <symbol id="svg-external-link" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-external-link"> <title id="svg-external-link-title">(external link)</title> <path d="M18 13v6a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V8a2 2 0 0 1 2-2h6"></path><polyline points="15 3 21 3 21 9"></polyline><line x1="10" y1="14" x2="21" y2="3"></line> </symbol> </svg> <div class="side-bar"> <div class="site-header"> <a href="/" class="site-title lh-tight"> ARSN Work Blog </a> <a href="#" id="menu-button" class="site-button"> <svg viewBox="0 0 24 24" class="icon"><use xlink:href="#svg-menu"></use></svg> </a> </div> <nav aria-label="Main" id="site-nav" class="site-nav"> <ul class="nav-list"><li class="nav-list-item"><a href="/" class="nav-list-link">Home</a></li><li class="nav-list-item active"><a href="#" class="nav-list-expander" aria-label="toggle links in sc_klf4bc category"> <svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg> </a><a href="/docs/sc_klf4bc/" class="nav-list-link">sc_klf4bc</a><ul class="nav-list "><li class="nav-list-item "><a href="/docs/sc_klf4bc/2022-12-12-sc_klf4bc-intial_processing.html" class="nav-list-link">22-12-13, KLF4 BC Initial Processing</a></li><li class="nav-list-item active"><a href="/docs/sc_klf4bc/2023-02-07-sc_klf4bc-feature_sets.html" class="nav-list-link active">23-02-07, Feature Sets</a></li></ul></li><li class="nav-list-item"><a href="#" class="nav-list-expander" aria-label="toggle links in sc_dm6tau category"> <svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg> </a><a href="/docs/sc_dm6tau/" class="nav-list-link">sc_dm6tau</a><ul class="nav-list "><li class="nav-list-item "><a href="/docs/sc_dm6tau/2021-10-07-post-0055.html" class="nav-list-link">Drosophilia Tau Preliminary DA</a></li><li class="nav-list-item "><a href="/docs/sc_dm6tau/2022-01-08-post-0069.html" class="nav-list-link">Drosophilia Tau Final DA</a></li><li class="nav-list-item "><a href="/docs/sc_dm6tau/2022-01-21-post-0072.html" class="nav-list-link">Drosophilia Tau DA Results Test Analysis</a></li></ul></li><li class="nav-list-item"><a href="#" class="nav-list-expander" aria-label="toggle links in sc_breastmultiome category"> <svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg> </a><a href="/docs/sc_breastmultiome/" class="nav-list-link">sc_breastmultiome</a><ul class="nav-list "><li class="nav-list-item "><a href="/docs/sc_breastmultiome/2022-12-12-sc_breastmultiome-epiconfig.html" class="nav-list-link">22-12-12, Epiconfig Runs on Full and Subset Data</a></li><li class="nav-list-item "><a href="/docs/sc_breastmultiome/2022-12-12-sc_breastmultiome-project_info.html" class="nav-list-link">22-12-12, Project Info</a></li><li class="nav-list-item "><a href="/docs/sc_breastmultiome/2023-01-12-sc_breastmultiome-epiconfig-downsample.html" class="nav-list-link">23-01-12, Epiconfig Run on Downampled Epithelial</a></li><li class="nav-list-item "><a href="/docs/sc_breastmultiome/2023-01-17-sc_breastmultiome-epiconfig-pam50umaps.html" class="nav-list-link">23-01-17, PAM50 UMAPs</a></li><li class="nav-list-item "><a href="/docs/sc_breastmultiome/2023-01-24-sc_breastmultiome-epiconfig-norm_luma.html" class="nav-list-link">23-01-24, Normal and LumA Subsets</a></li><li class="nav-list-item "><a href="/docs/sc_breastmultiome/2023-01-30-sc_breastmultiome-epiconfig-cnvnormal.html" class="nav-list-link">23-01-30, CNV Normal Calls</a></li><li class="nav-list-item "><a href="/docs/sc_breastmultiome/2023-02-03-sc_breastmultiome-epiconfig-callednormalCNV.html" class="nav-list-link">23-02-03, Normal Samples with Normal CNV Calls</a></li><li class="nav-list-item "><a href="/docs/sc_breastmultiome/2023-02-13-sc_breastmultiome-epiconfig-cancersea-clusters.html" class="nav-list-link">23-02-13, CancerSEA Plots w/ Epiconfig Clustesr</a></li></ul></li><li class="nav-list-item"><a href="#" class="nav-list-expander" aria-label="toggle links in ravnica category"> <svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg> </a><a href="/docs/ravnica/" class="nav-list-link">ravnica</a><ul class="nav-list "><li class="nav-list-item "><a href="/docs/ravnica/2022-01-25-post-0073.html" class="nav-list-link">Dropbox Upload and Download with Curl and API Key</a></li><li class="nav-list-item "><a href="/docs/ravnica/2022-05-18-post-0086.html" class="nav-list-link">Cron Info for Automatically Running bcl2fastq</a></li><li class="nav-list-item "><a href="/docs/ravnica/2022-12-05-ravnica-global_install_initial.html" class="nav-list-link">22-12-05, Global Installs - Initial</a></li><li class="nav-list-item "><a href="/docs/ravnica/2022-12-05-ravnica-local_install_initial.html" class="nav-list-link">22-12-05, Local Installs - Initial</a></li><li class="nav-list-item "><a href="/docs/ravnica/2022-12-05-ravnica-mamba_install_initial.html" class="nav-list-link">22-12-05, Mamba Install - Initial</a></li><li class="nav-list-item "><a href="/docs/ravnica/2022-12-05-ravnica-mamba_setup.html" class="nav-list-link">22-12-05, Mamba Setup</a></li><li class="nav-list-item "><a href="/docs/ravnica/2022-12-05-ravnica-r_4.2.2_library_info.html" class="nav-list-link">22-12-05, R 4.2.2 Library Info</a></li><li class="nav-list-item "><a href="/docs/ravnica/2022-12-06-ravnica-example_bashprofile.html" class="nav-list-link">22-12-06, Example Bash Profile</a></li><li class="nav-list-item "><a href="/docs/ravnica/2022-12-06-ravnica-venv_macs3.html" class="nav-list-link">22-12-06, VENV for Macs3</a></li></ul></li><li class="nav-list-item"><a href="#" class="nav-list-expander" aria-label="toggle links in scitools category"> <svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg> </a><a href="/docs/scitools/" class="nav-list-link">scitools</a><ul class="nav-list "><li class="nav-list-item "><a href="/docs/scitools/2020-12-17-post-0005.html" class="nav-list-link">Scitools scATAC Metrics</a></li><li class="nav-list-item "><a href="/docs/scitools/2021-02-18-post-0006.html" class="nav-list-link">Scitools Scrublet Function</a></li><li class="nav-list-item "><a href="/docs/scitools/2021-05-08-post-0009.html" class="nav-list-link">Single Cell Alignments with no Alt Chromosomes</a></li><li class="nav-list-item "><a href="/docs/scitools/2022-04-03-post-0077.html" class="nav-list-link">scRNA-seq First Pass Pipeline - 220324_HB_Coassay_RNA</a></li><li class="nav-list-item "><a href="/docs/scitools/2022-04-11-post-0080.html" class="nav-list-link">scRNA-seq Second Pass Pipeline - 220324_HB_Coassay_RNA</a></li></ul></li><li class="nav-list-item"><a href="#" class="nav-list-expander" aria-label="toggle links in archive_misc category"> <svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg> </a><a href="/docs/archive_misc/" class="nav-list-link">archive_misc</a><ul class="nav-list "><li class="nav-list-item "><a href="/docs/archive_misc/2020-10-09-post-0003.html" class="nav-list-link">Two Week Update: 2020-10-09</a></li><li class="nav-list-item "><a href="/docs/archive_misc/2020-11-09-post-0004.html" class="nav-list-link">Two Week Update: 2020-11-09</a></li><li class="nav-list-item "><a href="/docs/archive_misc/2021-05-05-post-0007.html" class="nav-list-link">NextSeq Fidelity Evaluation</a></li><li class="nav-list-item "><a href="/docs/archive_misc/2021-11-04-post-0064.html" class="nav-list-link">Contamination References</a></li><li class="nav-list-item "><a href="/docs/archive_misc/2022-03-01-post-0074.html" class="nav-list-link">OneDrive Upload and Download with Rclone</a></li><li class="nav-list-item "><a href="/docs/archive_misc/2022-04-21-post-0081.html" class="nav-list-link">scRNA-seq and scATAC-seq Data Analysis DREAM Challenge - Dataset 2 Data Generation</a></li><li class="nav-list-item "><a href="/docs/archive_misc/2022-05-18-post-0084.html" class="nav-list-link">Celsee and Docker Info</a></li><li class="nav-list-item "><a href="/docs/archive_misc/2020-10-07-post-0001.html" class="nav-list-link">Example</a></li><li class="nav-list-item "><a href="/docs/archive_misc/2022-05-20-post-0089.html" class="nav-list-link">Final O'Roak Lab Data Locations</a></li></ul></li><li class="nav-list-item"><a href="#" class="nav-list-expander" aria-label="toggle links in archive_amplicon category"> <svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg> </a><a href="/docs/archive_amplicon/" class="nav-list-link">archive_amplicon</a><ul class="nav-list "><li class="nav-list-item "><a href="/docs/archive_amplicon/2022-03-07-post-0075.html" class="nav-list-link">Shotgun Sequencing of Amplicon Sequence, 220305</a></li><li class="nav-list-item "><a href="/docs/archive_amplicon/2022-04-06-post-0078.html" class="nav-list-link">Shotgun Sequencing of Amplicon Sequence, 220406</a></li><li class="nav-list-item "><a href="/docs/archive_amplicon/2022-05-18-post-0085.html" class="nav-list-link">Example of HDR Processing via BWA Alignment</a></li></ul></li><li class="nav-list-item"><a href="#" class="nav-list-expander" aria-label="toggle links in archive_spark category"> <svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg> </a><a href="/docs/archive_spark/" class="nav-list-link">archive_spark</a><ul class="nav-list "><li class="nav-list-item "><a href="/docs/archive_spark/2021-05-11-post-0012.html" class="nav-list-link">21-05-11, SPARK Mosaic List, Summary Counts of Annotations</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-05-28-post-0015.html" class="nav-list-link">21-05-28, SPARK RefSeq CDS and Splice Site Reference Generation</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-05-30-post-0016.html" class="nav-list-link">21-05-30, SPARK Joint Coverage Counter Tool Fidelity</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-05-30-post-0017.html" class="nav-list-link">21-05-30, SPARK Joint Coverage Plots</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-06-02-post-0018.html" class="nav-list-link">21-06-02, SPARK Joint Coverage Table Fix</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-06-03-post-0019.html" class="nav-list-link">21-06-03, SPARK Mosaic List FRACTION_MORE Annotations</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-06-03-post-0020.html" class="nav-list-link">21-06-03, SPARK Mosaic Index</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-06-10-post-0021.html" class="nav-list-link">21-06-10, SPARK Gene Set Joint Coverage</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-06-22-post-0023.html" class="nav-list-link">21-06-22, SPARK Preliminary Pass 01, Filtering and Classifications</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-06-22-post-0024.html" class="nav-list-link">21-06-22, SPARK Preliminary Pass 02, Loading for Coverage and SNV Count Plots</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-06-26-post-0025.html" class="nav-list-link">21-06-26, SPARK Preliminary Pass 02b, Coverage and SNV Count Plots as Hists</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-06-28-post-0026.html" class="nav-list-link">21-06-28, SPARK Preliminary Pass 03, Allele Fraction Histograms</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-07-06-post-0027.html" class="nav-list-link">21-07-06, SPARK Preliminary Pass 04, Coverages and SNV Counts with Joint Family Coverages</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-07-14-post-0030.html" class="nav-list-link">21-07-14, SPARK Preliminary Pass 5, Final Filtering</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-07-17-post-0031.html" class="nav-list-link">21-07-17, SPARK Preliminary Pass 6a and 6b, Simplex Trio Coverages, SNV Counts, and AFs</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-07-17-post-0032.html" class="nav-list-link">21-07-17, SPARK Preliminary Pass 7a and 7b, Multiplex Quad Coverages, SNV Counts, and AFs</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-07-26-post-0036.html" class="nav-list-link">21-07-26, SPARK First Pass, Synonymous SNV Splice Distances</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-07-26-post-0037.html" class="nav-list-link">21-07-26, SPARK Preliminary Outlier Detection, Secondary Pass</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-07-26-post-0038.html" class="nav-list-link">21-07-26, SPARK Final Table Creation</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-07-27-post-0039.html" class="nav-list-link">21-07-27, SPARK Mosaic Index, July 27</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-08-03-post-0041.html" class="nav-list-link">21-08-03, SPARK High Confidence Allele Frequencies 01</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-08-03-post-0042.html" class="nav-list-link">21-08-03, SPARK High Confidence Synonymous Splice Distances 01</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-08-03-post-0043.html" class="nav-list-link">21-08-03, SPARK Burden Tables Autosomal 01</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-08-09-post-0044.html" class="nav-list-link">21-08-09, SPARK Burden Tables Gene Sets 01</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-08-26-post-0047.html" class="nav-list-link">21-08-26, SPARK IGV Shots of High Confident Calls</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-09-08-post-0048.html" class="nav-list-link">21-09-08, SPARK Burden Tables Gene Sets 02</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-09-14-post-0049.html" class="nav-list-link">21-09-14, SPARK Burden Tables Gene Sets 03</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-09-15-post-0050.html" class="nav-list-link">21-09-15, SPARK Concatenated High Confidence Table Checks 01</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-01-post-0051.html" class="nav-list-link">21-10-1, SPARK Final Sample List</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-01-post-0052.html" class="nav-list-link">21-10-01, SPARK Final IGV and Cohort Count Filter</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-06-post-0053.html" class="nav-list-link">21-10-06, SPARK LGD and NS List</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-06-post-0054.html" class="nav-list-link">21-10-06, SPARK Burden Tables Gene Sets 04</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-10-post-0056.html" class="nav-list-link">21-10-10, SPARK Summary of Final Lists</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-10-post-0057.html" class="nav-list-link">21-10-10, SPARK Final List Allele Frequency Distributions</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-11-post-0058.html" class="nav-list-link">21-10-11, SPARK Monozygotic Twins in Quad Families</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-11-post-0059.html" class="nav-list-link">21-10-11, SPARK Burden Tables Gene Sets, Final 01</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-13-post-0060.html" class="nav-list-link">21-10-13, SPARK Burden w/ REVEL Scores Gene Sets, 01</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-13-post-0061.html" class="nav-list-link">21-10-13, SPARK Parental Trans Mosaics Breakdown</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-13-post-0062.html" class="nav-list-link">21-10-13, SPARK HC Tables Against SPARK 167</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-14-post-0063.html" class="nav-list-link">21-10-14, SPARK Concat Variant Lists to DNV List Comparison</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-11-30-post-0066.html" class="nav-list-link">21-11-30, SPARK SSC List Formatting to Current SPARK</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-11-30-post-0067.html" class="nav-list-link">21-11-30, SPARK Burden Tables Gene Sets, SPARK+SSC</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-12-03-post-0068.html" class="nav-list-link">21-12-03, SPARK VEP Annotations</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2022-01-18-post-0070.html" class="nav-list-link">22-01-18, SPARK SSC hg38 Transformed Burden, 01</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2022-01-19-post-0071.html" class="nav-list-link">22-01-19, SPARK Pre Best Practices Filter, Model Scores</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2022-03-19-post-0076.html" class="nav-list-link">22-03-19, SPARK WES2 and WES3 List Inconsistencies</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2022-04-08-post-0079.html" class="nav-list-link">22-04-08, SPARK WES2 and WES3 Outliers by Counts and Coverage</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2022-04-28-post-0082.html" class="nav-list-link">22-04-28, SPARK Mosaic Index</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2022-05-03-post-0083.html" class="nav-list-link">22-05-03, SPARK IGV HC List Screenshots, 2022</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2022-05-20-post-0088.html" class="nav-list-link">22-05-20, SPARK Post-Hoc Cohort, IGV, and iWES Filter Lists</a></li></ul></li><li class="nav-list-item"><a href="#" class="nav-list-expander" aria-label="toggle links in archive_sc_tbr1 category"> <svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg> </a><a href="/docs/archive_sc_tbr1/" class="nav-list-link">archive_sc_tbr1</a><ul class="nav-list "><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-05-07-post-0008.html" class="nav-list-link">Single Cell TBR1 Preliminary Evaluations</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-05-09-post-0010.html" class="nav-list-link">Single Cell TBR1 Coembedding Tests</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-05-11-post-0011.html" class="nav-list-link">Single Cell TBR1 ENCODE DHS Index Tests</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-05-27-post-0013.html" class="nav-list-link">Single Cell TBR1 ENCODE DHS Index Final UMAP and Clusters</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-05-28-post-0014.html" class="nav-list-link">Single Cell TBR1 Cluster DA</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-06-10-post-0022.html" class="nav-list-link">Single Cell TBR1 Cicero/Monocle3 with TSS > 2.7</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-07-09-post-0028.html" class="nav-list-link">Single Cell TBR1 ChromVAR with TSS > 2.7</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-07-12-post-0029.html" class="nav-list-link">Single Cell TBR1 ChromVAR with TSS > 1.5</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-07-18-post-0033.html" class="nav-list-link">Single Cell TBR1 BioCluster/Genotype/Line/Time DA with TSS > 2.7</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-07-19-post-0034.html" class="nav-list-link">Single Cell TBR1 Misc Figures - ChromVAR Compare, FDR Hists, 3D</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-07-26-post-0035.html" class="nav-list-link">Single Cell TBR1 BioCluster/Genotype/Line/Time DA with TSS > 2.7, Filtering Ambiguous Groups</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-07-29-post-0040.html" class="nav-list-link">Single Cell TBR1 Bigwig Tracks</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-08-12-post-0045.html" class="nav-list-link">Single Cell TBR1 Cistopic Boxplots and UMAPs</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-08-12-post-0046.html" class="nav-list-link">Single Cell TBR1 ChIP-Seq Peak DA</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-11-28-post-0065.html" class="nav-list-link">Single Cell TBR1 Pseudo-Bulk Depth Tests</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2022-05-18-post-0087.html" class="nav-list-link">SRA Download Info for 'TBR1 Regulates Autism Risk Genes in the Developing Neocortex'</a></li></ul></li><li class="nav-list-item"><a href="#" class="nav-list-expander" aria-label="toggle links in archive_ssc category"> <svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg> </a><a href="/docs/archive_ssc/" class="nav-list-link">archive_ssc</a><ul class="nav-list "><li class="nav-list-item "><a href="/docs/archive_ssc/2020-10-09-post-0002.html" class="nav-list-link">SSC Final Validation Tables with Annotations</a></li></ul></li></ul> </nav> <footer class="site-footer"> This site uses <a href="https://github.com/just-the-docs/just-the-docs">Just the Docs</a>, a documentation theme for Jekyll. </footer> </div> <div class="main" id="top"> <div id="main-header" class="main-header"> <div class="search"> <div class="search-input-wrap"> <input type="text" id="search-input" class="search-input" tabindex="0" placeholder="Search ARSN Work Blog" aria-label="Search ARSN Work Blog" autocomplete="off"> <label for="search-input" class="search-label"><svg viewBox="0 0 24 24" class="search-icon"><use xlink:href="#svg-search"></use></svg></label> </div> <div id="search-results" class="search-results"></div> </div> <nav aria-label="Auxiliary" class="aux-nav"> <ul class="aux-nav-list"> <li class="aux-nav-list-item"> <a href="//github.com/just-the-docs/just-the-docs" class="site-button" > Just the Docs on GitHub </a> </li> </ul> </nav> </div> <div id="main-content-wrap" class="main-content-wrap"> <nav aria-label="Breadcrumb" class="breadcrumb-nav"> <ol class="breadcrumb-nav-list"> <li class="breadcrumb-nav-list-item"><a href="/docs/sc_klf4bc/">sc_klf4bc</a></li> <li class="breadcrumb-nav-list-item"><span>23-02-07, Feature Sets</span></li> </ol> </nav> <div id="main-content" class="main-content" role="main"> <h2 class="no_toc text-delta" id="table-of-contents"> <a href="#table-of-contents" class="anchor-heading" aria-labelledby="table-of-contents"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> Table of contents </h2> <ol id="markdown-toc"> <li><a href="#0-what-is-feature-sets" id="markdown-toc-0-what-is-feature-sets">0. What is Feature Sets?</a></li> <li><a href="#1-set-environment" id="markdown-toc-1-set-environment">1. Set Environment</a> <ol> <li><a href="#1a-software-modules" id="markdown-toc-1a-software-modules">1a. Software Modules</a></li> <li><a href="#1b-libraries" id="markdown-toc-1b-libraries">1b. Libraries</a></li> <li><a href="#1c-populate-working-directory" id="markdown-toc-1c-populate-working-directory">1c. Populate Working Directory</a></li> </ol> </li> <li><a href="#2-populate-data-references-and-scripts" id="markdown-toc-2-populate-data-references-and-scripts">2. Populate Data, References, and Scripts</a> <ol> <li><a href="#2a-populate-data" id="markdown-toc-2a-populate-data">2a. Populate Data</a></li> <li><a href="#2b-populate-references" id="markdown-toc-2b-populate-references">2b. Populate References</a></li> </ol> </li> <li><a href="#3-create-feature-lists" id="markdown-toc-3-create-feature-lists">3. Create Feature Lists</a> <ol> <li><a href="#3a-features-from-all" id="markdown-toc-3a-features-from-all">3a. Features from All</a></li> <li><a href="#3b-features-from-vocab" id="markdown-toc-3b-features-from-vocab">3b. Features from Vocab</a></li> <li><a href="#3c-features-from-topics" id="markdown-toc-3c-features-from-topics">3c. Features from Topics</a></li> </ol> </li> <li><a href="#4-distance-analysis" id="markdown-toc-4-distance-analysis">4. Distance Analysis</a> <ol> <li><a href="#4a-hybrid-matrix" id="markdown-toc-4a-hybrid-matrix">4a. Hybrid Matrix</a></li> <li><a href="#4b-distance-distributions-global-tss" id="markdown-toc-4b-distance-distributions-global-tss">4b. Distance Distributions, Global TSS</a></li> <li><a href="#4c-distance-distributions-within-topic-tss" id="markdown-toc-4c-distance-distributions-within-topic-tss">4c. Distance Distributions, Within Topic TSS</a></li> </ol> </li> <li><a href="#5-gene-info" id="markdown-toc-5-gene-info">5. Gene Info</a> <ol> <li><a href="#5a-prep-gene-info" id="markdown-toc-5a-prep-gene-info">5a. Prep Gene Info</a></li> </ol> </li> <li><a href="#6-correlations" id="markdown-toc-6-correlations">6. Correlations</a> <ol> <li><a href="#6a-pearson--spearman" id="markdown-toc-6a-pearson--spearman">6a. Pearson / Spearman</a></li> <li><a href="#6b-ccans" id="markdown-toc-6b-ccans">6b. CCANâ€™s</a></li> </ol> </li> <li><a href="#7-enrichment---subset" id="markdown-toc-7-enrichment---subset">7. Enrichment - Subset</a></li> <li><a href="#8-enrichment---similar-set" id="markdown-toc-8-enrichment---similar-set">8. Enrichment - Similar Set</a></li> <li><a href="#9-enrichment---superset" id="markdown-toc-9-enrichment---superset">9. Enrichment - Superset</a></li> </ol> <h1 id="0-what-is-feature-sets"> <a href="#0-what-is-feature-sets" class="anchor-heading" aria-labelledby="0-what-is-feature-sets"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> 0. What is Feature Sets? </h1> <p>Identification of chromatin features in a dataset can easily generate 100K-1M features. To understand the biological relevancy of features and for practical purposes, features will almost always be treated into subset groups. Generally in the single-cell context, subsets of the full feature list attempt to represent variablity within cells and describing features within and between a collection of feature sets is meant to capture biological differences in data. Internally, the only available information of a feature is the region itself usually represented in a BED format of chromosome, start, and stop. Biologically relevancy of the features must be derived from the overlap of these features to outside references.</p> <h1 id="1-set-environment"> <a href="#1-set-environment" class="anchor-heading" aria-labelledby="1-set-environment"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> 1. Set Environment </h1> <h2 id="1a-software-modules"> <a href="#1a-software-modules" class="anchor-heading" aria-labelledby="1a-software-modules"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> 1a. Software Modules </h2> <p>Any versions should be work. PLACEHOLDER - add more downstream software.</p> <div class="language-sh highlighter-rouge"><div class="highlight"><pre class="highlight"><code><span class="c"># software modules</span>
module load bedops/2.4.41
module load R/4.2.2
</code></pre></div></div> <h2 id="1b-libraries"> <a href="#1b-libraries" class="anchor-heading" aria-labelledby="1b-libraries"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> 1b. Libraries </h2> <p>PLACEHOLDER - print a list and move to exacloud.</p> <div class="language-sh highlighter-rouge"><div class="highlight"><pre class="highlight"><code><span class="nb">export </span><span class="nv">R_LIBS_USER</span><span class="o">=</span><span class="s1">'/home/groups/ravnica/env/R_library_backups/R-4.2.2_copy'</span>	
</code></pre></div></div> <h2 id="1c-populate-working-directory"> <a href="#1c-populate-working-directory" class="anchor-heading" aria-labelledby="1c-populate-working-directory"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> 1c. Populate Working Directory </h2> <div class="language-sh highlighter-rouge"><div class="highlight"><pre class="highlight"><code><span class="nb">cd</span> /home/groups/ravnica/users/nishida/temp_features
<span class="nb">mkdir </span>figures<span class="p">;</span> <span class="nb">mkdir </span>matrices<span class="p">;</span> <span class="nb">mkdir </span>refs<span class="p">;</span> <span class="nb">mkdir </span>scripts<span class="p">;</span> <span class="nb">mkdir </span>tmp
<span class="nb">cd </span>matrices/
<span class="nb">mkdir </span>1_distance<span class="p">;</span> <span class="nb">mkdir </span>2_geneinfo<span class="p">;</span> <span class="nb">mkdir </span>3_correlations<span class="p">;</span> <span class="nb">mkdir </span>4_enrichment_subset<span class="p">;</span> <span class="nb">mkdir </span>5_enrichment_set<span class="p">;</span> <span class="nb">mkdir </span>6_enrichment_superset
<span class="nb">cd</span> ..
</code></pre></div></div><hr /> <h1 id="2-populate-data-references-and-scripts"> <a href="#2-populate-data-references-and-scripts" class="anchor-heading" aria-labelledby="2-populate-data-references-and-scripts"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> 2. Populate Data, References, and Scripts </h1> <h2 id="2a-populate-data"> <a href="#2a-populate-data" class="anchor-heading" aria-labelledby="2a-populate-data"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> 2a. Populate Data </h2> <p>Data includes the full R object of the complete Seurat multiomics dataset split and fed into Epiconfig, the vocab matrix, and the LDA model with desired number of topics.</p> <div class="language-sh highlighter-rouge"><div class="highlight"><pre class="highlight"><code><span class="nb">cp</span> /home/groups/CEDAR/nishida/epiconfig/MCF7_KLF4_wTopics_SO.rds <span class="nb">.</span>
<span class="nb">cp</span> /home/groups/CEDAR/nishida/epiconfig/klf4/CLRNorm_100kb_2000Feats_TFIDFvar75_Model/vocab_matrix.rds <span class="nb">.</span>
<span class="nb">cp</span> /home/groups/CEDAR/nishida/epiconfig/MCF7/CLRNorm_100kb_2000Feats_TFIDFvar75_Model/default_lda_out/lda_65topics.rds <span class="nb">.</span>
</code></pre></div></div> <h2 id="2b-populate-references"> <a href="#2b-populate-references" class="anchor-heading" aria-labelledby="2b-populate-references"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> 2b. Populate References </h2> <p>These will always be genome and data dependent. In the future we will start curating these in a centralized location but for now pull them from whenever. Here I am copying across drives on purpose (exacloud to ravnica) but normally these should be symbolic links <code class="language-plaintext highlighter-rouge">ln -s</code>.</p> <div class="language-sh highlighter-rouge"><div class="highlight"><pre class="highlight"><code><span class="nb">cd </span>refs/
<span class="c">#################</span>
<span class="c"># Genome Specific</span>
<span class="c">#################</span>
<span class="c"># GTF, this is an old HAVANA/ENSEMBL GTF default packaged with cellranger missing annotations (like 5' and 3' UTR)</span>
<span class="nb">cp</span> /home/groups/CEDAR/anno/CellRanger/refdata-cellranger-arc-GRCh38-2020-A-2.0.0/genes/genes.gtf.gz <span class="nb">.</span>
<span class="c"># GTF, more recent GTF with more discrete annotation</span>
<span class="nb">cp</span> /home/groups/CEDAR/nishida/refs/hg38/hg38_ucsc/hg38.ensGene.gtf <span class="nb">.</span>
<span class="c"># ENSEMBL TSS</span>
<span class="nb">cp</span> /home/groups/CEDAR/nishida/refs/hg38/hg38_ensembl_tss/ensembl.hg38.tss.chr.bed <span class="nb">.</span> 
<span class="c"># DHS Index</span>
<span class="nb">cp</span> /home/groups/CEDAR/nishida/refs/hg38/hg38_DHS_index/DHS_Index_and_Vocabulary_hg38_WM20190703.bed <span class="nb">.</span>
<span class="c"># Motifs</span>
<span class="nb">cp</span> /home/groups/CEDAR/nishida/refs/fimo_scans/cat_hg38.bed <span class="nb">.</span>
<span class="c">#################################</span>
<span class="c"># Cell-Type/Tissue/Data Specific:</span>
<span class="c">#################################</span>
<span class="c"># active enhancer mark, H3K27ac</span>
<span class="nb">cp</span> /home/groups/CEDAR/nishida/SC_KLF4BC/refs/H3K27Ac_intersect_all3.bed <span class="nb">.</span>
<span class="c"># superenhancers</span>
<span class="nb">cp</span> /home/groups/CEDAR/nishida/SC_KLF4BC/refs/Homer_SE_MCF7_sg_NT_1.bed <span class="nb">.</span>
<span class="c"># CCANs</span>
<span class="nb">cp</span> /home/groups/CEDAR/nishida/SC_KLF4BC/refs/MCF7.CCANs.txt <span class="nb">.</span>
<span class="c"># relevant cistrome data</span>
<span class="c"># here I'm pulling out relevant cistrome subsets by the "Tissue_type" column matching "Breast"</span>
<span class="c"># and then collapsing into new names by "Factor", "Cell_line", and "Cell_type" columns</span>
<span class="c"># change these based on your desired subset and query</span>
<span class="c"># I'm removing spaces and backslashes but other annotations might need more character changes</span>
<span class="nb">mkdir </span>cistrome
<span class="nb">cat</span> /home/groups/CEDAR/mulqueen/ref/cistrome/human_factor_full_QC.txt | <span class="nb">cut</span> <span class="nt">-f</span> 1,4-7 | <span class="nb">awk</span> <span class="s1">'$5 == "Breast"'</span> | <span class="nb">cut</span> <span class="nt">-f</span> 1-4 | <span class="nb">sed</span> <span class="nt">-e</span> <span class="s1">'s/\t/_/2g'</span> | <span class="nb">sed</span> <span class="nt">-e</span> <span class="s1">'s/\//-/g'</span> | <span class="nb">sed</span> <span class="nt">-e</span> <span class="s1">'s/ //g'</span> <span class="o">&gt;</span> cistrome/cistrome_subset.txt
<span class="k">while </span><span class="nv">IFS</span><span class="o">=</span> <span class="nb">read</span> <span class="nt">-r</span> line
<span class="k">do
	</span><span class="nv">a</span><span class="o">=</span><span class="si">$(</span><span class="nb">echo</span> <span class="s2">"</span><span class="nv">$line</span><span class="s2">"</span> | <span class="nb">cut</span> <span class="nt">-f</span> 1<span class="si">)</span>
	<span class="nv">b</span><span class="o">=</span><span class="si">$(</span><span class="nb">echo</span> <span class="s2">"</span><span class="nv">$line</span><span class="s2">"</span> | <span class="nb">cut</span> <span class="nt">-f</span> 2<span class="si">)</span>
	<span class="nb">cp</span> /home/groups/CEDAR/mulqueen/ref/cistrome/human_factor/<span class="k">${</span><span class="nv">a</span><span class="k">}</span>_sort_peaks.narrowPeak.bed cistrome/<span class="k">${</span><span class="nv">a</span><span class="k">}</span>_<span class="k">${</span><span class="nv">b</span><span class="k">}</span>.bed
<span class="k">done</span> &lt; cistrome/cistrome_subset.txt
<span class="nb">cd</span> ..
</code></pre></div></div><hr /> <h1 id="3-create-feature-lists"> <a href="#3-create-feature-lists" class="anchor-heading" aria-labelledby="3-create-feature-lists"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> 3. Create Feature Lists </h1> <h2 id="3a-features-from-all"> <a href="#3a-features-from-all" class="anchor-heading" aria-labelledby="3a-features-from-all"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> 3a. Features from All </h2> <p>Pass the input path and output prefix. Seurat object assay names may not be standardized, adjust accordingly.</p> <div class="language-sh highlighter-rouge"><div class="highlight"><pre class="highlight"><code>Rscript scripts/pull_features.all.Rscript MCF7_KLF4_wTopics_SO.rds features.all
</code></pre></div></div> <p>Content of <code class="language-plaintext highlighter-rouge">scripts/pull_features.vocab.Rscript</code>:</p> <div class="language-R highlighter-rouge"><div class="highlight"><pre class="highlight"><code><span class="c1"># !/usr/bin/R</span><span class="w">

</span><span class="n">library</span><span class="p">(</span><span class="n">Signac</span><span class="p">)</span><span class="w">

</span><span class="n">args</span><span class="w"> </span><span class="o">&lt;-</span><span class="w"> </span><span class="n">commandArgs</span><span class="p">(</span><span class="n">trailingOnly</span><span class="w"> </span><span class="o">=</span><span class="w"> </span><span class="nb">T</span><span class="p">)</span><span class="w">
</span><span class="n">input_path</span><span class="w"> </span><span class="o">&lt;-</span><span class="w"> </span><span class="n">args</span><span class="p">[</span><span class="m">1</span><span class="p">]</span><span class="w">
</span><span class="n">output_prefix</span><span class="w"> </span><span class="o">&lt;-</span><span class="w"> </span><span class="n">args</span><span class="p">[</span><span class="m">2</span><span class="p">]</span><span class="w">

</span><span class="n">input_model</span><span class="w"> </span><span class="o">&lt;-</span><span class="w"> </span><span class="n">readRDS</span><span class="p">(</span><span class="n">input_path</span><span class="p">)</span><span class="w">
</span><span class="n">rna</span><span class="w"> </span><span class="o">&lt;-</span><span class="w"> </span><span class="n">rownames</span><span class="p">(</span><span class="n">input_model</span><span class="o">@</span><span class="n">assays</span><span class="o">$</span><span class="n">RNA</span><span class="p">)</span><span class="w">
</span><span class="n">atac</span><span class="w"> </span><span class="o">&lt;-</span><span class="w"> </span><span class="n">rownames</span><span class="p">(</span><span class="n">input_model</span><span class="o">@</span><span class="n">assays</span><span class="o">$</span><span class="n">ATAC</span><span class="p">)</span><span class="w">

</span><span class="n">write.table</span><span class="p">(</span><span class="n">rna</span><span class="p">,</span><span class="n">paste</span><span class="p">(</span><span class="n">output_prefix</span><span class="p">,</span><span class="s2">".rna.txt"</span><span class="p">,</span><span class="n">sep</span><span class="o">=</span><span class="s2">""</span><span class="p">),</span><span class="n">sep</span><span class="o">=</span><span class="s2">"\t"</span><span class="p">,</span><span class="n">quote</span><span class="o">=</span><span class="nb">F</span><span class="p">,</span><span class="n">row.names</span><span class="o">=</span><span class="nb">F</span><span class="p">,</span><span class="n">col.names</span><span class="o">=</span><span class="nb">F</span><span class="p">)</span><span class="w">
</span><span class="n">write.table</span><span class="p">(</span><span class="n">atac</span><span class="p">,</span><span class="n">paste</span><span class="p">(</span><span class="n">output_prefix</span><span class="p">,</span><span class="s2">".atac.txt"</span><span class="p">,</span><span class="n">sep</span><span class="o">=</span><span class="s2">""</span><span class="p">),</span><span class="n">sep</span><span class="o">=</span><span class="s2">"\t"</span><span class="p">,</span><span class="n">quote</span><span class="o">=</span><span class="nb">F</span><span class="p">,</span><span class="n">row.names</span><span class="o">=</span><span class="nb">F</span><span class="p">,</span><span class="n">col.names</span><span class="o">=</span><span class="nb">F</span><span class="p">)</span><span class="w">
</span></code></pre></div></div> <h2 id="3b-features-from-vocab"> <a href="#3b-features-from-vocab" class="anchor-heading" aria-labelledby="3b-features-from-vocab"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> 3b. Features from Vocab </h2> <p>Pass the vocab matrix input path and the output prefix.</p> <div class="language-sh highlighter-rouge"><div class="highlight"><pre class="highlight"><code>Rscript scripts/pull_features.vocab.Rscript vocab_matrix.rds features.vocab
</code></pre></div></div> <p>Content of <code class="language-plaintext highlighter-rouge">scripts/pull_features.vocab.Rscript</code>:</p> <div class="language-R highlighter-rouge"><div class="highlight"><pre class="highlight"><code><span class="c1"># !/usr/bin/R</span><span class="w">

</span><span class="n">library</span><span class="p">(</span><span class="n">Matrix</span><span class="p">)</span><span class="w">

</span><span class="n">args</span><span class="w"> </span><span class="o">&lt;-</span><span class="w"> </span><span class="n">commandArgs</span><span class="p">(</span><span class="n">trailingOnly</span><span class="w"> </span><span class="o">=</span><span class="w"> </span><span class="nb">T</span><span class="p">)</span><span class="w">
</span><span class="n">input_path</span><span class="w"> </span><span class="o">&lt;-</span><span class="w"> </span><span class="n">args</span><span class="p">[</span><span class="m">1</span><span class="p">]</span><span class="w">
</span><span class="n">output_prefix</span><span class="w"> </span><span class="o">&lt;-</span><span class="w"> </span><span class="n">args</span><span class="p">[</span><span class="m">2</span><span class="p">]</span><span class="w">

</span><span class="n">input_model</span><span class="w"> </span><span class="o">&lt;-</span><span class="w"> </span><span class="n">readRDS</span><span class="p">(</span><span class="n">input_path</span><span class="p">)</span><span class="w">
</span><span class="n">features</span><span class="w"> </span><span class="o">&lt;-</span><span class="w"> </span><span class="n">row.names</span><span class="p">(</span><span class="n">input_model</span><span class="p">)</span><span class="w">

</span><span class="c1"># split features</span><span class="w">
</span><span class="n">atac_features</span><span class="w"> </span><span class="o">&lt;-</span><span class="w"> </span><span class="n">features</span><span class="p">[</span><span class="n">which</span><span class="p">(</span><span class="n">grepl</span><span class="p">(</span><span class="s2">"^chr"</span><span class="p">,</span><span class="n">features</span><span class="p">))]</span><span class="w">
</span><span class="n">hybrid_features</span><span class="w"> </span><span class="o">&lt;-</span><span class="w"> </span><span class="n">features</span><span class="p">[</span><span class="n">which</span><span class="p">(</span><span class="n">grepl</span><span class="p">(</span><span class="s2">"_ON$"</span><span class="p">,</span><span class="n">features</span><span class="p">))]</span><span class="w">
</span><span class="n">rna_features</span><span class="w"> </span><span class="o">&lt;-</span><span class="w"> </span><span class="n">features</span><span class="p">[</span><span class="o">!</span><span class="w"> </span><span class="n">features</span><span class="w"> </span><span class="o">%in%</span><span class="w"> </span><span class="nf">c</span><span class="p">(</span><span class="n">atac_features</span><span class="p">,</span><span class="n">hybrid_features</span><span class="p">)]</span><span class="w">

</span><span class="c1"># check feature counts</span><span class="w">
</span><span class="n">sum_features</span><span class="w"> </span><span class="o">&lt;-</span><span class="w"> </span><span class="nf">length</span><span class="p">(</span><span class="n">atac_features</span><span class="p">)</span><span class="w"> </span><span class="o">+</span><span class="w"> </span><span class="nf">length</span><span class="p">(</span><span class="n">hybrid_features</span><span class="p">)</span><span class="w"> </span><span class="o">+</span><span class="w"> </span><span class="nf">length</span><span class="p">(</span><span class="n">rna_features</span><span class="p">)</span><span class="w">
</span><span class="k">if</span><span class="w"> </span><span class="p">(</span><span class="n">sum_features</span><span class="w"> </span><span class="o">!=</span><span class="w"> </span><span class="nf">length</span><span class="p">(</span><span class="n">features</span><span class="p">))</span><span class="w"> </span><span class="p">{</span><span class="w">
	</span><span class="n">print</span><span class="p">(</span><span class="s2">"Feature splitting by feature names does not add to the total number of features. Check feature names."</span><span class="p">)</span><span class="w">
	</span><span class="n">quit</span><span class="p">()</span><span class="w">
</span><span class="p">}</span><span class="w">

</span><span class="n">write.table</span><span class="p">(</span><span class="n">atac_features</span><span class="p">,</span><span class="n">paste</span><span class="p">(</span><span class="n">output_prefix</span><span class="p">,</span><span class="s2">".atac.txt"</span><span class="p">,</span><span class="n">sep</span><span class="o">=</span><span class="s2">""</span><span class="p">),</span><span class="n">row.names</span><span class="o">=</span><span class="nb">F</span><span class="p">,</span><span class="n">col.names</span><span class="o">=</span><span class="nb">F</span><span class="p">,</span><span class="n">quote</span><span class="o">=</span><span class="nb">F</span><span class="p">,</span><span class="n">sep</span><span class="o">=</span><span class="s2">"\t"</span><span class="p">)</span><span class="w">
</span><span class="n">write.table</span><span class="p">(</span><span class="n">rna_features</span><span class="p">,</span><span class="n">paste</span><span class="p">(</span><span class="n">output_prefix</span><span class="p">,</span><span class="s2">".rna.txt"</span><span class="p">,</span><span class="n">sep</span><span class="o">=</span><span class="s2">""</span><span class="p">),</span><span class="n">row.names</span><span class="o">=</span><span class="nb">F</span><span class="p">,</span><span class="n">col.names</span><span class="o">=</span><span class="nb">F</span><span class="p">,</span><span class="n">quote</span><span class="o">=</span><span class="nb">F</span><span class="p">,</span><span class="n">sep</span><span class="o">=</span><span class="s2">"\t"</span><span class="p">)</span><span class="w">
</span><span class="n">write.table</span><span class="p">(</span><span class="n">hybrid_features</span><span class="p">,</span><span class="n">paste</span><span class="p">(</span><span class="n">output_prefix</span><span class="p">,</span><span class="s2">".hybrid.txt"</span><span class="p">,</span><span class="n">sep</span><span class="o">=</span><span class="s2">""</span><span class="p">),</span><span class="n">row.names</span><span class="o">=</span><span class="nb">F</span><span class="p">,</span><span class="n">col.names</span><span class="o">=</span><span class="nb">F</span><span class="p">,</span><span class="n">quote</span><span class="o">=</span><span class="nb">F</span><span class="p">,</span><span class="n">sep</span><span class="o">=</span><span class="s2">"\t"</span><span class="p">)</span><span class="w">
</span></code></pre></div></div> <h2 id="3c-features-from-topics"> <a href="#3c-features-from-topics" class="anchor-heading" aria-labelledby="3c-features-from-topics"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> 3c. Features from Topics </h2> <p>For every lambda in the range of 0 to 1 in increments of 0.05, pull top words/features. Pass the lda model, lambda, max words, and output path. Number of max words here is 2000 but adjust accordingly to length of full vocab.</p> <div class="language-sh highlighter-rouge"><div class="highlight"><pre class="highlight"><code><span class="nv">array</span><span class="o">=(</span>0.05 0.10 0.15 0.20 0.25 0.30 0.35 0.40 0.45 0.50 0.55 0.60 0.65 0.70 0.75 0.80 0.85 0.90 0.95 1.00<span class="o">)</span>
<span class="k">for </span>lambda <span class="k">in</span> <span class="s2">"</span><span class="k">${</span><span class="nv">array</span><span class="p">[@]</span><span class="k">}</span><span class="s2">"</span>
<span class="k">do
	</span><span class="nb">mkdir </span>topwords_<span class="k">${</span><span class="nv">lambda</span><span class="k">}</span>
	Rscript scripts/lambda_top_words.Rscript lda_65topics.rds <span class="k">${</span><span class="nv">lambda</span><span class="k">}</span> 2000 topwords_<span class="k">${</span><span class="nv">lambda</span><span class="k">}</span>
<span class="k">done</span>
</code></pre></div></div> <p>Content of <code class="language-plaintext highlighter-rouge">scripts/lambda_top_words.Rscript</code>:</p> <div class="language-R highlighter-rouge"><div class="highlight"><pre class="highlight"><code><span class="c1"># !/usr/bin/R</span><span class="w">

</span><span class="n">args</span><span class="w"> </span><span class="o">&lt;-</span><span class="w"> </span><span class="n">commandArgs</span><span class="p">(</span><span class="n">trailingOnly</span><span class="w"> </span><span class="o">=</span><span class="w"> </span><span class="nb">T</span><span class="p">)</span><span class="w">
</span><span class="n">input_path</span><span class="w"> </span><span class="o">&lt;-</span><span class="w"> </span><span class="n">args</span><span class="p">[</span><span class="m">1</span><span class="p">]</span><span class="w">
</span><span class="n">lambda</span><span class="w"> </span><span class="o">&lt;-</span><span class="w"> </span><span class="n">args</span><span class="p">[</span><span class="m">2</span><span class="p">]</span><span class="w">
</span><span class="n">maxwords</span><span class="w"> </span><span class="o">&lt;-</span><span class="w"> </span><span class="n">args</span><span class="p">[</span><span class="m">3</span><span class="p">]</span><span class="w">
</span><span class="n">output_path</span><span class="w"> </span><span class="o">&lt;-</span><span class="w"> </span><span class="n">args</span><span class="p">[</span><span class="m">4</span><span class="p">]</span><span class="w">

</span><span class="n">selected_model</span><span class="w"> </span><span class="o">&lt;-</span><span class="w"> </span><span class="n">readRDS</span><span class="p">(</span><span class="n">input_path</span><span class="p">)</span><span class="w">
</span><span class="n">num_topics</span><span class="w"> </span><span class="o">&lt;-</span><span class="w"> </span><span class="n">nrow</span><span class="p">(</span><span class="n">selected_model</span><span class="o">$</span><span class="n">topic_word_distribution</span><span class="p">)</span><span class="w">

</span><span class="c1"># print topwords per topic</span><span class="w">
</span><span class="k">for</span><span class="w"> </span><span class="p">(</span><span class="n">i</span><span class="w"> </span><span class="k">in</span><span class="w"> </span><span class="m">1</span><span class="o">:</span><span class="n">num_topics</span><span class="p">)</span><span class="w"> </span><span class="p">{</span><span class="w">
	</span><span class="n">top_words_ldavis</span><span class="w"> </span><span class="o">&lt;-</span><span class="w"> </span><span class="n">selected_model</span><span class="o">$</span><span class="n">get_top_words</span><span class="p">(</span><span class="n">n</span><span class="w"> </span><span class="o">=</span><span class="w"> </span><span class="nf">length</span><span class="p">(</span><span class="n">selected_model</span><span class="o">$</span><span class="n">.__enclos_env__</span><span class="o">$</span><span class="n">private</span><span class="o">$</span><span class="n">vocabulary</span><span class="p">),</span><span class="w"> </span><span class="c1">#61299,</span><span class="w">
		</span><span class="n">topic_number</span><span class="w"> </span><span class="o">=</span><span class="w"> </span><span class="nf">as.numeric</span><span class="p">(</span><span class="n">i</span><span class="p">),</span><span class="w"> </span><span class="c1"># 1L:selected_model$.__enclos_env__$private$n_topics,</span><span class="w">
		</span><span class="n">lambda</span><span class="w"> </span><span class="o">=</span><span class="w"> </span><span class="nf">as.numeric</span><span class="p">(</span><span class="n">lambda</span><span class="p">))[</span><span class="m">1</span><span class="o">:</span><span class="n">maxwords</span><span class="p">]</span><span class="w">
	</span><span class="n">filename</span><span class="w"> </span><span class="o">&lt;-</span><span class="w"> </span><span class="n">paste</span><span class="p">(</span><span class="n">output_path</span><span class="p">,</span><span class="w"> </span><span class="s2">"/topic_"</span><span class="p">,</span><span class="n">i</span><span class="p">,</span><span class="s2">".txt"</span><span class="p">,</span><span class="n">sep</span><span class="o">=</span><span class="s2">""</span><span class="p">)</span><span class="w">
	</span><span class="n">write.table</span><span class="p">(</span><span class="n">top_words_ldavis</span><span class="p">,</span><span class="n">filename</span><span class="p">,</span><span class="n">sep</span><span class="o">=</span><span class="s2">"\t"</span><span class="p">,</span><span class="n">quote</span><span class="o">=</span><span class="nb">F</span><span class="p">,</span><span class="n">row.names</span><span class="o">=</span><span class="nb">F</span><span class="p">,</span><span class="n">col.names</span><span class="o">=</span><span class="nb">F</span><span class="p">)</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre></div></div><hr /> <h1 id="4-distance-analysis"> <a href="#4-distance-analysis" class="anchor-heading" aria-labelledby="4-distance-analysis"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> 4. Distance Analysis </h1> <h2 id="4a-hybrid-matrix"> <a href="#4a-hybrid-matrix" class="anchor-heading" aria-labelledby="4a-hybrid-matrix"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> 4a. Hybrid Matrix </h2> <p>Hybrid words are based on distance. Due to their lower number, just count them up per topic/lambda. Makes a matrix, <code class="language-plaintext highlighter-rouge">matrix.hybwords.txt</code>, with columns 1. number topics, 2. lambda, 3. number of hybrid words.</p> <div class="language-sh highlighter-rouge"><div class="highlight"><pre class="highlight"><code><span class="c"># variables</span>
<span class="nv">OUT_PATH</span><span class="o">=</span><span class="s2">"/home/groups/ravnica/users/nishida/temp_features/matrices/1_distance"</span>
<span class="nv">VOCAB_PATH</span><span class="o">=</span><span class="s2">"features.vocab.hybrid.txt"</span>

<span class="c"># create the header</span>
<span class="nb">echo</span> <span class="nt">-e</span> <span class="s2">"TOPIC</span><span class="se">\t</span><span class="s2">LAMBDA</span><span class="se">\t</span><span class="s2">COUNT"</span> <span class="o">&gt;</span> <span class="nv">$OUT_PATH</span>/matrix.hybwords.txt

<span class="c"># number of hybrid words matrix</span>
<span class="nv">array</span><span class="o">=(</span>0.05 0.10 0.15 0.20 0.25 0.30 0.35 0.40 0.45 0.50 0.55 0.60 0.65 0.70 0.75 0.80 0.85 0.90 0.95 1.00<span class="o">)</span>
<span class="k">for </span>n_topic <span class="k">in</span> <span class="si">$(</span><span class="nb">seq </span>1 65<span class="si">)</span>
<span class="k">do
	for </span>lambda <span class="k">in</span> <span class="s2">"</span><span class="k">${</span><span class="nv">array</span><span class="p">[@]</span><span class="k">}</span><span class="s2">"</span>
	<span class="k">do
		</span><span class="nv">numhyb</span><span class="o">=</span><span class="si">$(</span><span class="nb">cat </span>topwords_<span class="k">${</span><span class="nv">lambda</span><span class="k">}</span>/topic_<span class="k">${</span><span class="nv">n_topic</span><span class="k">}</span>.txt | <span class="nb">grep</span> <span class="s2">"ON$"</span> | <span class="nb">wc</span> <span class="nt">-l</span><span class="si">)</span>
		<span class="nb">echo</span> <span class="nt">-e</span> <span class="s2">"</span><span class="k">${</span><span class="nv">n_topic</span><span class="k">}</span><span class="se">\t</span><span class="k">${</span><span class="nv">lambda</span><span class="k">}</span><span class="se">\t</span><span class="nv">$numhyb</span><span class="s2">"</span>
	<span class="k">done
done</span> <span class="o">&gt;&gt;</span> <span class="nv">$OUT_PATH</span>/matrix.hybwords.txt

<span class="c"># add the vocab total at the end</span>
<span class="nv">numhyb</span><span class="o">=</span><span class="si">$(</span><span class="nb">cat</span> <span class="nv">$VOCAB_PATH</span> | <span class="nb">wc</span> <span class="nt">-l</span><span class="si">)</span>
<span class="nb">echo</span> <span class="nt">-e</span> <span class="s2">"VOCAB_TOTAL</span><span class="se">\t</span><span class="s2">VOCAB_TOTAL</span><span class="se">\t</span><span class="nv">$numhyb</span><span class="s2">"</span> <span class="o">&gt;&gt;</span> <span class="nv">$OUT_PATH</span>/matrix.hybwords.txt
</code></pre></div></div> <h2 id="4b-distance-distributions-global-tss"> <a href="#4b-distance-distributions-global-tss" class="anchor-heading" aria-labelledby="4b-distance-distributions-global-tss"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> 4b. Distance Distributions, Global TSS </h2> <p>Calculates a matrix with the distribution of distances per topic with every lambda. The global distances can be calculated on the full set of features and subset from that. Point TMP to wherever and clean up at your leisure. Output is a matrix for global, vocab, and each topic with distances and counts per lambda.</p> <div class="language-sh highlighter-rouge"><div class="highlight"><pre class="highlight"><code><span class="c"># output path</span>
<span class="nv">OUT_PATH</span><span class="o">=</span><span class="s2">"/home/groups/ravnica/users/nishida/temp_features/matrices/1_distance"</span>
<span class="nv">TMP</span><span class="o">=</span><span class="s2">"tmp/"</span>

<span class="c"># calculate all distances</span>
<span class="c"># reformat to bed, sort bed, find distances relative to TSS, save relevant columns</span>
<span class="nb">cat </span>features.all.atac.txt  | <span class="nb">tr</span> <span class="s1">'-'</span> <span class="s1">'\t'</span> | sort-bed - | closest-features <span class="nt">--closest</span> <span class="nt">--delim</span> <span class="s2">"</span><span class="se">\t</span><span class="s2">"</span> <span class="nt">--dist</span> - refs/ensembl.hg38.tss.chr.bed | <span class="nb">cut</span> <span class="nt">-f</span> 1,2,3,7,8,9 <span class="o">&gt;</span> features.all.atac.sorted_distances.txt

<span class="c"># calculate vocab distance by overlap to all features</span>
<span class="nb">cat </span>features.vocab.atac.txt | <span class="nb">tr</span> <span class="s1">'-'</span> <span class="s1">'\t'</span> | sort-bed - | bedops <span class="nt">-e</span> features.all.atac.sorted_distances.txt - <span class="o">&gt;</span> features.vocab.atac.sorted_distances.txt

<span class="c"># summarize all feature distances into count distributions</span>
<span class="nv">INPUT</span><span class="o">=</span><span class="s2">"features.all.atac.sorted_distances.txt"</span>
<span class="nv">OUTPUT</span><span class="o">=</span><span class="s2">"matrix.distance_all.txt"</span>
<span class="c"># header</span>
<span class="nb">echo</span> <span class="nt">-e</span> <span class="s2">"DISTANCE</span><span class="se">\t</span><span class="s2">COUNT_ALL"</span> <span class="o">&gt;</span> <span class="nv">$OUT_PATH</span>/<span class="nv">$OUTPUT</span>
<span class="c"># grab distances, sort numerically, collapse to counts, reformat white space, swap columns, reformat to tabs</span>
<span class="nb">cat</span> <span class="nv">$INPUT</span> | <span class="nb">cut</span> <span class="nt">-f</span> 6 | <span class="nb">sort</span> <span class="nt">-n</span> | <span class="nb">uniq</span> <span class="nt">-c</span> | <span class="nb">sed</span> <span class="nt">-e</span> <span class="s1">'s/^[ ]*//g'</span> | <span class="nb">awk</span> <span class="s1">'{print $2,$1}'</span> | <span class="nb">tr</span> <span class="s1">' '</span> <span class="s1">'\t'</span> <span class="o">&gt;&gt;</span> <span class="nv">$OUT_PATH</span>/<span class="nv">$OUTPUT</span>

<span class="c"># summarize all vocab distances into count distributions</span>
<span class="nv">INPUT</span><span class="o">=</span><span class="s2">"features.vocab.atac.sorted_distances.txt"</span>
<span class="nv">OUTPUT</span><span class="o">=</span><span class="s2">"matrix.distance_vocab.txt"</span>
<span class="c"># header</span>
<span class="nb">echo</span> <span class="nt">-e</span> <span class="s2">"DISTANCE</span><span class="se">\t</span><span class="s2">COUNT_VOCAB"</span> <span class="o">&gt;</span> <span class="nv">$OUT_PATH</span>/<span class="nv">$OUTPUT</span>
<span class="c"># grab distances, sort numerically, collapse to counts, reformat white space, swap columns, reformat to tabs</span>
<span class="nb">cat</span> <span class="nv">$INPUT</span> | <span class="nb">cut</span> <span class="nt">-f</span> 6 | <span class="nb">sort</span> <span class="nt">-n</span> | <span class="nb">uniq</span> <span class="nt">-c</span> | <span class="nb">sed</span> <span class="nt">-e</span> <span class="s1">'s/^[ ]*//g'</span> | <span class="nb">awk</span> <span class="s1">'{print $2,$1}'</span> | <span class="nb">tr</span> <span class="s1">' '</span> <span class="s1">'\t'</span> <span class="o">&gt;&gt;</span> <span class="nv">$OUT_PATH</span>/<span class="nv">$OUTPUT</span>
<span class="c"># prep topic matrixes seeding with vocab since topics are a subset of vocab, save to TMP</span>
<span class="nb">cat</span> <span class="nv">$OUT_PATH</span>/<span class="nv">$OUTPUT</span> | <span class="nb">cut</span> <span class="nt">-f</span> 1,2 | <span class="nb">sed </span>1d <span class="o">&gt;</span> <span class="k">${</span><span class="nv">TMP</span><span class="k">}</span>/tmp.vocabdist.txt

<span class="c"># get padded distance counts per topic and lambda, save to TMP</span>
<span class="nv">REF</span><span class="o">=</span><span class="s2">"features.vocab.atac.sorted_distances.txt"</span>
<span class="nv">array</span><span class="o">=(</span>0.05 0.10 0.15 0.20 0.25 0.30 0.35 0.40 0.45 0.50 0.55 0.60 0.65 0.70 0.75 0.80 0.85 0.90 0.95 1.00<span class="o">)</span>
<span class="k">for </span>n_topic <span class="k">in</span> <span class="si">$(</span><span class="nb">seq </span>1 65<span class="si">)</span>
<span class="k">do
	for </span>lambda <span class="k">in</span> <span class="s2">"</span><span class="k">${</span><span class="nv">array</span><span class="p">[@]</span><span class="k">}</span><span class="s2">"</span>
	<span class="k">do</span>
		<span class="c"># get atac features, format to bed, sort, compare overlap, save distance, sort, count, clean whitespace, swap columns, reformat, pad with 0's via key, reformat, save relevant column</span>
		<span class="nb">cat </span>topwords_<span class="k">${</span><span class="nv">lambda</span><span class="k">}</span>/topic_<span class="k">${</span><span class="nv">n_topic</span><span class="k">}</span>.txt | <span class="nb">grep</span> <span class="s2">"^chr"</span> | <span class="nb">tr</span> <span class="s1">'-'</span> <span class="s1">'\t'</span> | sort-bed - | bedops <span class="nt">-e</span> <span class="nv">$REF</span> - | <span class="nb">cut</span> <span class="nt">-f</span> 6 | <span class="nb">sort</span> <span class="nt">-n</span> | <span class="nb">uniq</span> <span class="nt">-c</span> | <span class="nb">sed</span> <span class="nt">-e</span> <span class="s1">'s/^[ ]*//g'</span> | <span class="nb">awk</span> <span class="s1">'{print $2,$1}'</span> | <span class="nb">tr</span> <span class="s1">' '</span> <span class="s1">'\t'</span> | <span class="nb">awk</span> <span class="s1">'NR == FNR { key[$1] = $2; next } { $2 = ($1 in key) ? key[$1] : 0 }; 1'</span> - <span class="k">${</span><span class="nv">TMP</span><span class="k">}</span>/tmp.vocabdist.txt | <span class="nb">tr</span> <span class="s1">' '</span> <span class="s1">'\t'</span> | <span class="nb">cut</span> <span class="nt">-f</span> 2 <span class="o">&gt;</span> <span class="k">${</span><span class="nv">TMP</span><span class="k">}</span>/<span class="k">${</span><span class="nv">n_topic</span><span class="k">}</span>_<span class="k">${</span><span class="nv">lambda</span><span class="k">}</span>.txt
		
		<span class="c"># if the file is completely empty (no ATAC features) pad it entirely with 0's</span>
		<span class="k">if</span> <span class="o">[</span> <span class="o">!</span> <span class="nt">-s</span> <span class="k">${</span><span class="nv">TMP</span><span class="k">}</span>/<span class="k">${</span><span class="nv">n_topic</span><span class="k">}</span>_<span class="k">${</span><span class="nv">lambda</span><span class="k">}</span>.txt <span class="o">]</span><span class="p">;</span> <span class="k">then
			</span><span class="nb">cat</span> <span class="k">${</span><span class="nv">TMP</span><span class="k">}</span>/tmp.vocabdist.txt | <span class="nb">awk</span> <span class="s1">'{print 0}'</span> <span class="o">&gt;</span> <span class="k">${</span><span class="nv">TMP</span><span class="k">}</span>/<span class="k">${</span><span class="nv">n_topic</span><span class="k">}</span>_<span class="k">${</span><span class="nv">lambda</span><span class="k">}</span>.txt
		<span class="k">fi
	done
done</span>

<span class="c"># PLACEHOLDER - turn this into a better loop rather than explicity calling each lambda</span>
<span class="k">for </span>num <span class="k">in</span> <span class="si">$(</span><span class="nb">seq </span>1 65<span class="si">)</span>
<span class="k">do</span>
	<span class="c"># header</span>
	<span class="nb">echo</span> <span class="nt">-e</span> <span class="s2">"DISTANCE</span><span class="se">\t</span><span class="s2">COUNT_VOCAB</span><span class="se">\t</span><span class="s2">COUNT_L0.05</span><span class="se">\t</span><span class="s2">COUNT_L0.10</span><span class="se">\t</span><span class="s2">COUNT_L0.15</span><span class="se">\t</span><span class="s2">COUNT_L0.20</span><span class="se">\t</span><span class="s2">COUNT_L0.25</span><span class="se">\t</span><span class="s2">COUNT_L0.30</span><span class="se">\t</span><span class="s2">COUNT_L0.35</span><span class="se">\t</span><span class="s2">COUNT_L0.40</span><span class="se">\t</span><span class="s2">COUNT_L0.45</span><span class="se">\t</span><span class="s2">COUNT_L0.50</span><span class="se">\t</span><span class="s2">COUNT_L0.55</span><span class="se">\t</span><span class="s2">COUNT_L0.60</span><span class="se">\t</span><span class="s2">COUNT_L0.65</span><span class="se">\t</span><span class="s2">COUNT_L0.70</span><span class="se">\t</span><span class="s2">COUNT_L0.75</span><span class="se">\t</span><span class="s2">COUNT_L0.80</span><span class="se">\t</span><span class="s2">COUNT_L0.85</span><span class="se">\t</span><span class="s2">COUNT_L0.90</span><span class="se">\t</span><span class="s2">COUNT_L0.95</span><span class="se">\t</span><span class="s2">COUNT_L1.00"</span> <span class="o">&gt;</span> <span class="nv">$OUT_PATH</span>/matrix.distance.topic_<span class="k">${</span><span class="nv">num</span><span class="k">}</span>.txt
	<span class="nb">paste</span> <span class="k">${</span><span class="nv">TMP</span><span class="k">}</span>/tmp.vocabdist.txt <span class="k">${</span><span class="nv">TMP</span><span class="k">}</span>/<span class="k">${</span><span class="nv">num</span><span class="k">}</span>_0.05.txt <span class="k">${</span><span class="nv">TMP</span><span class="k">}</span>/<span class="k">${</span><span class="nv">num</span><span class="k">}</span>_0.10.txt <span class="k">${</span><span class="nv">TMP</span><span class="k">}</span>/<span class="k">${</span><span class="nv">num</span><span class="k">}</span>_0.15.txt <span class="k">${</span><span class="nv">TMP</span><span class="k">}</span>/<span class="k">${</span><span class="nv">num</span><span class="k">}</span>_0.20.txt <span class="k">${</span><span class="nv">TMP</span><span class="k">}</span>/<span class="k">${</span><span class="nv">num</span><span class="k">}</span>_0.25.txt <span class="k">${</span><span class="nv">TMP</span><span class="k">}</span>/<span class="k">${</span><span class="nv">num</span><span class="k">}</span>_0.30.txt <span class="k">${</span><span class="nv">TMP</span><span class="k">}</span>/<span class="k">${</span><span class="nv">num</span><span class="k">}</span>_0.35.txt <span class="k">${</span><span class="nv">TMP</span><span class="k">}</span>/<span class="k">${</span><span class="nv">num</span><span class="k">}</span>_0.40.txt <span class="k">${</span><span class="nv">TMP</span><span class="k">}</span>/<span class="k">${</span><span class="nv">num</span><span class="k">}</span>_0.45.txt <span class="k">${</span><span class="nv">TMP</span><span class="k">}</span>/<span class="k">${</span><span class="nv">num</span><span class="k">}</span>_0.50.txt <span class="k">${</span><span class="nv">TMP</span><span class="k">}</span>/<span class="k">${</span><span class="nv">num</span><span class="k">}</span>_0.55.txt <span class="k">${</span><span class="nv">TMP</span><span class="k">}</span>/<span class="k">${</span><span class="nv">num</span><span class="k">}</span>_0.60.txt <span class="k">${</span><span class="nv">TMP</span><span class="k">}</span>/<span class="k">${</span><span class="nv">num</span><span class="k">}</span>_0.65.txt <span class="k">${</span><span class="nv">TMP</span><span class="k">}</span>/<span class="k">${</span><span class="nv">num</span><span class="k">}</span>_0.70.txt <span class="k">${</span><span class="nv">TMP</span><span class="k">}</span>/<span class="k">${</span><span class="nv">num</span><span class="k">}</span>_0.75.txt <span class="k">${</span><span class="nv">TMP</span><span class="k">}</span>/<span class="k">${</span><span class="nv">num</span><span class="k">}</span>_0.80.txt <span class="k">${</span><span class="nv">TMP</span><span class="k">}</span>/<span class="k">${</span><span class="nv">num</span><span class="k">}</span>_0.85.txt <span class="k">${</span><span class="nv">TMP</span><span class="k">}</span>/<span class="k">${</span><span class="nv">num</span><span class="k">}</span>_0.90.txt <span class="k">${</span><span class="nv">TMP</span><span class="k">}</span>/<span class="k">${</span><span class="nv">num</span><span class="k">}</span>_0.95.txt <span class="k">${</span><span class="nv">TMP</span><span class="k">}</span>/<span class="k">${</span><span class="nv">num</span><span class="k">}</span>_1.00.txt <span class="o">&gt;&gt;</span> <span class="nv">$OUT_PATH</span>/matrix.distance.topic_<span class="k">${</span><span class="nv">num</span><span class="k">}</span>.txt
<span class="k">done</span>
</code></pre></div></div> <h2 id="4c-distance-distributions-within-topic-tss"> <a href="#4c-distance-distributions-within-topic-tss" class="anchor-heading" aria-labelledby="4c-distance-distributions-within-topic-tss"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> 4c. Distance Distributions, Within Topic TSS </h2> <p>Calculates a matrix with the distrubution of distances within the realm of a topic. Here I am using the original GTF from Cell Ranger to specifically match names. In the future, we should probably switch to accession IDâ€™s. This GTF is a combination of ENSEMBL and HAVANA, is a bit older, and lacks some discrete categories (like splitting 5â€™ and 3â€™ UTR annotations). The output is a bit wide for my liking but with max words of 2000 it isnâ€™t obscene. Columns are 1. Topic, 2. Lambda, 3. Number of ATAC features, 4. Number of RNA features, 5. Number of RNA features found, 6. Number of ATAC features with no match (not on the chromosome) and 7. comma separated distances.</p> <div class="language-sh highlighter-rouge"><div class="highlight"><pre class="highlight"><code><span class="nv">OUT_PATH</span><span class="o">=</span><span class="s2">"/home/groups/ravnica/users/nishida/temp_features/matrices/1_distance"</span>
<span class="nv">REF</span><span class="o">=</span><span class="s2">"refs/genes.gtf.gz"</span>
<span class="nv">TMP</span><span class="o">=</span><span class="s2">"tmp/"</span>

<span class="c"># pull out a reference of start codons and common gene names</span>
zcat <span class="nv">$REF</span> | <span class="nb">awk</span> <span class="s1">'$3 == "start_codon"'</span> | <span class="nb">cut</span> <span class="nt">-f</span> 1,4,5 <span class="o">&gt;</span> <span class="k">${</span><span class="nv">TMP</span><span class="k">}</span>/tmp.gtf.coords.bed
zcat <span class="nv">$REF</span> | <span class="nb">awk</span> <span class="s1">'$3 == "start_codon"'</span> | <span class="nb">cut</span> <span class="nt">-f</span> 9 | <span class="nb">sed</span> <span class="nt">-e</span> <span class="s1">'s/.*gene_name \"//g'</span> | <span class="nb">sed</span> <span class="nt">-e</span> <span class="s1">'s/\".*//g'</span> <span class="o">&gt;</span> <span class="k">${</span><span class="nv">TMP</span><span class="k">}</span>/tmp.gtf.gnames.bed

<span class="c"># strangely some of the 'start_codons' here are not 3bp, pad these with by one base to the stop</span>
<span class="nb">paste</span> <span class="k">${</span><span class="nv">TMP</span><span class="k">}</span>/tmp.gtf.coords.bed <span class="k">${</span><span class="nv">TMP</span><span class="k">}</span>/tmp.gtf.gnames.bed | <span class="nb">awk</span> <span class="s1">'{if ($2 == $3) {printf ("%s\t%s\t%s\t%s\n",$1,$2,$3+1,$4)} else {print}}'</span> | sort-bed - <span class="o">&gt;</span> <span class="k">${</span><span class="nv">TMP</span><span class="k">}</span>/tmp.gtf_starts.sorted.bed

<span class="nb">echo</span> <span class="s2">"TOPIC</span><span class="se">\t</span><span class="s2">LAMBDA</span><span class="se">\t</span><span class="s2">NUM_ATAC</span><span class="se">\t</span><span class="s2">NUM_RNA</span><span class="se">\t</span><span class="s2">NUM_RNA_FOUND</span><span class="se">\t</span><span class="s2">NUM_ATAC_NA</span><span class="se">\t</span><span class="s2">DISTANCE_STRING"</span> <span class="o">&gt;</span> <span class="k">${</span><span class="nv">OUT_PATH</span><span class="k">}</span>/matrix.distance.within.txt
<span class="nv">array</span><span class="o">=(</span>0.05 0.10 0.15 0.20 0.25 0.30 0.35 0.40 0.45 0.50 0.55 0.60 0.65 0.70 0.75 0.80 0.85 0.90 0.95 1.00<span class="o">)</span>
<span class="k">for </span>n_topic <span class="k">in</span> <span class="si">$(</span><span class="nb">seq </span>1 65<span class="si">)</span>
<span class="k">do
	for </span>lambda <span class="k">in</span> <span class="s2">"</span><span class="k">${</span><span class="nv">array</span><span class="p">[@]</span><span class="k">}</span><span class="s2">"</span>
	<span class="k">do</span>
		
		<span class="c"># per topic and per lambda, write a list of comma-separated sorted distances</span>
		<span class="nb">cat </span>topwords_<span class="k">${</span><span class="nv">lambda</span><span class="k">}</span>/topic_<span class="k">${</span><span class="nv">n_topic</span><span class="k">}</span>.txt | <span class="nb">grep</span> <span class="nt">-v</span> <span class="s2">"_ON$"</span> | <span class="nb">grep</span> <span class="s2">"^chr"</span> <span class="o">&gt;</span> tmp/tmp.atac.txt
		<span class="nb">cat </span>topwords_<span class="k">${</span><span class="nv">lambda</span><span class="k">}</span>/topic_<span class="k">${</span><span class="nv">n_topic</span><span class="k">}</span>.txt | <span class="nb">grep</span> <span class="nt">-v</span> <span class="s2">"_ON$"</span> | <span class="nb">grep</span> <span class="nt">-v</span> <span class="s2">"^chr"</span> <span class="o">&gt;</span> tmp/tmp.rna.txt
		
		<span class="c"># subset rna bed</span>
		<span class="k">while </span><span class="nv">IFS</span><span class="o">=</span> <span class="nb">read</span> <span class="nt">-r</span> line
		<span class="k">do
			</span><span class="nb">cat </span>tmp/tmp.gtf_starts.sorted.bed | <span class="nb">awk</span> <span class="nt">-v</span> <span class="nv">name</span><span class="o">=</span><span class="s2">"</span><span class="nv">$line</span><span class="s2">"</span> <span class="s1">'$4 == name'</span>
		<span class="k">done</span> &lt; tmp/tmp.rna.txt | sort-bed - <span class="o">&gt;</span> tmp/tmp.rna.gtf_starts.sorted.bed
		
		<span class="c"># calculate values</span>
		<span class="nv">num_atac</span><span class="o">=</span><span class="si">$(</span><span class="nb">cat </span>tmp/tmp.atac.txt | <span class="nb">wc</span> <span class="nt">-l</span><span class="si">)</span>
		<span class="nv">num_rna</span><span class="o">=</span><span class="si">$(</span><span class="nb">cat </span>tmp/tmp.rna.txt | <span class="nb">wc</span> <span class="nt">-l</span><span class="si">)</span>
		<span class="nv">num_rna_found</span><span class="o">=</span><span class="si">$(</span><span class="nb">cat </span>tmp/tmp.rna.gtf_starts.sorted.bed | <span class="nb">cut</span> <span class="nt">-f</span> 4 | <span class="nb">sort</span> | <span class="nb">uniq</span> | <span class="nb">wc</span> <span class="nt">-l</span><span class="si">)</span>
		<span class="nv">num_atac_na</span><span class="o">=</span><span class="si">$(</span><span class="nb">cat </span>tmp/tmp.atac.txt | <span class="nb">tr</span> <span class="s1">'-'</span> <span class="s1">'\t'</span> | sort-bed - | closest-features <span class="nt">--dist</span> <span class="nt">--closest</span> <span class="nt">--no-query</span> <span class="nt">--delim</span> <span class="s1">'\t'</span> - tmp/tmp.rna.gtf_starts.sorted.bed | <span class="nb">cut</span> <span class="nt">-f</span> 5 | <span class="nb">grep</span> <span class="nt">-c</span> <span class="s2">"NA"</span><span class="si">)</span>
		<span class="nv">dist_str</span><span class="o">=</span><span class="si">$(</span><span class="nb">cat </span>tmp/tmp.atac.txt | <span class="nb">tr</span> <span class="s1">'-'</span> <span class="s1">'\t'</span> | sort-bed - | closest-features <span class="nt">--dist</span> <span class="nt">--closest</span> <span class="nt">--no-query</span> <span class="nt">--delim</span> <span class="s1">'\t'</span> - tmp/tmp.rna.gtf_starts.sorted.bed | <span class="nb">cut</span> <span class="nt">-f</span> 4 | <span class="nb">grep</span> <span class="nt">-v</span> <span class="s2">"NA"</span> | <span class="nb">tr</span> <span class="s1">'\n'</span> <span class="s1">','</span> | <span class="nb">sed</span> <span class="s1">'s/.$//'</span><span class="si">)</span>
		
		<span class="nb">echo</span> <span class="nt">-e</span> <span class="s2">"</span><span class="nv">$n_topic</span><span class="se">\t</span><span class="nv">$lambda</span><span class="se">\t</span><span class="nv">$num_atac</span><span class="se">\t</span><span class="nv">$num_rna</span><span class="se">\t</span><span class="nv">$num_rna_found</span><span class="se">\t</span><span class="nv">$num_atac_na</span><span class="se">\t</span><span class="nv">$dist_str</span><span class="s2">"</span> <span class="o">&gt;&gt;</span> <span class="k">${</span><span class="nv">OUT_PATH</span><span class="k">}</span>/matrix.distance.within.txt
		
	<span class="k">done
done</span>
</code></pre></div></div><hr /> <h1 id="5-gene-info"> <a href="#5-gene-info" class="anchor-heading" aria-labelledby="5-gene-info"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> 5. Gene Info </h1> <h2 id="5a-prep-gene-info"> <a href="#5a-prep-gene-info" class="anchor-heading" aria-labelledby="5a-prep-gene-info"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> 5a. Prep Gene Info </h2> <p>Here Iâ€™m using a GTF which is more recent and has more annotations (UTR is broken into 5â€™ and 3â€™).</p> <div class="language-sh highlighter-rouge"><div class="highlight"><pre class="highlight"><code><span class="c">#what did ryan ryan do?</span>
<span class="c">#GTF2 - hg38.ensGene.gtf</span>
</code></pre></div></div><hr /> <h1 id="6-correlations"> <a href="#6-correlations" class="anchor-heading" aria-labelledby="6-correlations"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> 6. Correlations </h1> <h2 id="6a-pearson--spearman"> <a href="#6a-pearson--spearman" class="anchor-heading" aria-labelledby="6a-pearson--spearman"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> 6a. Pearson / Spearman </h2> <div class="language-sh highlighter-rouge"><div class="highlight"><pre class="highlight"><code>
</code></pre></div></div> <h2 id="6b-ccans"> <a href="#6b-ccans" class="anchor-heading" aria-labelledby="6b-ccans"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> 6b. CCANâ€™s </h2> <div class="language-sh highlighter-rouge"><div class="highlight"><pre class="highlight"><code><span class="c"># CCAN - MCF7.CCANs.txt</span>
</code></pre></div></div><hr /> <h1 id="7-enrichment---subset"> <a href="#7-enrichment---subset" class="anchor-heading" aria-labelledby="7-enrichment---subset"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> 7. Enrichment - Subset </h1> <div class="language-sh highlighter-rouge"><div class="highlight"><pre class="highlight"><code><span class="c"># FIMO - cat_hg38.bed</span>
<span class="c"># hypergeo</span>
<span class="nb">cd</span> /home/groups/oroaklab/adey_lab/projects/maga/00_DataFreeze_Cortex_and_Hippocampus/Regions/cortex/cortex.filt.temp40_topic.Rds.topics

<span class="c"># tally backgrounds</span>
<span class="nb">cat</span> <span class="k">*</span>.bed | sort-bed - <span class="o">&gt;</span> all.bed
<span class="nb">cat </span>all.bed | bedops <span class="nt">-e</span> 1 /home/groups/oroaklab/nishida/fimo_scans/cat_hg38.bed - <span class="o">&gt;</span> background.bed
<span class="nb">cat </span>background.bed | <span class="nb">cut</span> <span class="nt">-f</span> 5,6 | <span class="nb">tr</span> <span class="s1">'\t'</span> <span class="s1">'_'</span> | <span class="nb">sort</span> | <span class="nb">uniq</span> <span class="nt">-c</span> | <span class="nb">sed</span> <span class="nt">-e</span> <span class="s1">'s/^[ ]*//g'</span> | <span class="nb">tr</span> <span class="s1">' '</span> <span class="s1">'\t'</span> | <span class="nb">awk</span> <span class="s1">'{print $2,$1}'</span> | <span class="nb">tr</span> <span class="s1">' '</span> <span class="s1">'\t'</span> <span class="o">&gt;</span> background_counts.txt

<span class="c"># tally topics and compare to bg with hypergeo</span>
<span class="nv">array</span><span class="o">=(</span>Topic_10 Topic_11 Topic_12 Topic_13 Topic_14 Topic_15 Topic_16 Topic_17 Topic_18 Topic_19 Topic_1 Topic_20 Topic_21 Topic_22 Topic_23 Topic_24 Topic_25 Topic_26 Topic_27 Topic_28 Topic_29 Topic_2 Topic_30 Topic_31 Topic_32 Topic_33 Topic_34 Topic_35 Topic_36 Topic_37 Topic_38 Topic_39 Topic_3 Topic_40 Topic_4 Topic_5 Topic_6 Topic_7 Topic_8 Topic_9<span class="o">)</span>
<span class="k">for </span>i <span class="k">in</span> <span class="s2">"</span><span class="k">${</span><span class="nv">array</span><span class="p">[@]</span><span class="k">}</span><span class="s2">"</span>
<span class="k">do
</span><span class="nb">cat</span> <span class="nv">$i</span>.bed | sort-bed - | bedops <span class="nt">-e</span> 1 /home/groups/oroaklab/nishida/fimo_scans/cat_hg38.bed - <span class="o">&gt;</span> <span class="nv">$i</span>.motifs.bed
<span class="nb">cat</span> <span class="nv">$i</span>.motifs.bed | <span class="nb">cut</span> <span class="nt">-f</span> 5,6 | <span class="nb">tr</span> <span class="s1">'\t'</span> <span class="s1">'_'</span> | <span class="nb">sort</span> | <span class="nb">uniq</span> <span class="nt">-c</span> | <span class="nb">sed</span> <span class="nt">-e</span> <span class="s1">'s/^[ ]*//g'</span> | <span class="nb">tr</span> <span class="s1">' '</span> <span class="s1">'\t'</span> | <span class="nb">awk</span> <span class="s1">'{print $2,$1}'</span> | <span class="nb">tr</span> <span class="s1">' '</span> <span class="s1">'\t'</span> <span class="o">&gt;</span> <span class="nv">$i</span>.motifcounts.txt
Rscript /home/groups/oroaklab/nishida/scripts/sc/motifs_hypergeo.Rscript <span class="nv">$i</span>.motifcounts.txt background_counts.txt <span class="nv">$i</span>.hypergeo.txt
<span class="k">done</span>
</code></pre></div></div><hr /> <h1 id="8-enrichment---similar-set"> <a href="#8-enrichment---similar-set" class="anchor-heading" aria-labelledby="8-enrichment---similar-set"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> 8. Enrichment - Similar Set </h1> <div class="language-sh highlighter-rouge"><div class="highlight"><pre class="highlight"><code><span class="c">#DHSi - DHS_Index_and_Vocabulary_hg38_WM20190703.bed</span>
<span class="c">#CISTROMES - /cistrome/etc</span>
</code></pre></div></div><hr /> <h1 id="9-enrichment---superset"> <a href="#9-enrichment---superset" class="anchor-heading" aria-labelledby="9-enrichment---superset"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> 9. Enrichment - Superset </h1> <div class="language-sh highlighter-rouge"><div class="highlight"><pre class="highlight"><code><span class="c">#SUPERENH - Homer_SE_MCF7_sg_NT_1.bed # SE computed by Aaron in MCF7 WT cells.</span>
<span class="c">#ACTIVEENHMARK - H3K27Ac_intersect_all3.bed # This BED is the "active enhancers" mark in WT_E2 treated MCF7.</span>
</code></pre></div></div> </div> </div> <div class="search-overlay"></div> </div> </body> </html>
