<!DOCTYPE html> <html lang="en-US"> <head> <meta charset="UTF-8"> <meta http-equiv="X-UA-Compatible" content="IE=Edge"> <title>Shotgun Sequencing of Amplicon Sequence, 220305 - ARSN Work Blog</title> <link rel="shortcut icon" href="/favicon.ico" type="image/x-icon"> <link rel="stylesheet" href="/assets/css/just-the-docs-default.css"> <script src="/assets/js/vendor/lunr.min.js"></script> <script src="/assets/js/just-the-docs.js"></script> <meta name="viewport" content="width=device-width, initial-scale=1"> <!-- Begin Jekyll SEO tag v2.8.0 --> <title>Shotgun Sequencing of Amplicon Sequence, 220305 | ARSN Work Blog</title> <meta name="generator" content="Jekyll v4.3.0" /> <meta property="og:title" content="Shotgun Sequencing of Amplicon Sequence, 220305" /> <meta property="og:locale" content="en_US" /> <meta name="description" content="This is a blog for work projects Andrew Nishida, nishidaa@ohsu.edu, in CEDAR at OHSU." /> <meta property="og:description" content="This is a blog for work projects Andrew Nishida, nishidaa@ohsu.edu, in CEDAR at OHSU." /> <link rel="canonical" href="http://localhost:4000/docs/archive_amplicon/2022-03-07-post-0075.html" /> <meta property="og:url" content="http://localhost:4000/docs/archive_amplicon/2022-03-07-post-0075.html" /> <meta property="og:site_name" content="ARSN Work Blog" /> <meta property="og:type" content="article" /> <meta property="article:published_time" content="2022-03-07T00:00:00-08:00" /> <meta name="twitter:card" content="summary" /> <meta property="twitter:title" content="Shotgun Sequencing of Amplicon Sequence, 220305" /> <script type="application/ld+json"> {"@context":"https://schema.org","@type":"BlogPosting","dateModified":"2022-03-07T00:00:00-08:00","datePublished":"2022-03-07T00:00:00-08:00","description":"This is a blog for work projects Andrew Nishida, nishidaa@ohsu.edu, in CEDAR at OHSU.","headline":"Shotgun Sequencing of Amplicon Sequence, 220305","mainEntityOfPage":{"@type":"WebPage","@id":"http://localhost:4000/docs/archive_amplicon/2022-03-07-post-0075.html"},"url":"http://localhost:4000/docs/archive_amplicon/2022-03-07-post-0075.html"}</script> <!-- End Jekyll SEO tag --> </head> <body> <a class="skip-to-main" href="#main-content">Skip to main content</a> <svg xmlns="http://www.w3.org/2000/svg" style="display: none;"> <symbol id="svg-link" viewBox="0 0 24 24"> <title>Link</title> <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-link"> <path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path> </svg> </symbol> <symbol id="svg-search" viewBox="0 0 24 24"> <title>Search</title> <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-search"> <circle cx="11" cy="11" r="8"></circle><line x1="21" y1="21" x2="16.65" y2="16.65"></line> </svg> </symbol> <symbol id="svg-menu" viewBox="0 0 24 24"> <title>Menu</title> <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-menu"> <line x1="3" y1="12" x2="21" y2="12"></line><line x1="3" y1="6" x2="21" y2="6"></line><line x1="3" y1="18" x2="21" y2="18"></line> </svg> </symbol> <symbol id="svg-arrow-right" viewBox="0 0 24 24"> <title>Expand</title> <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-chevron-right"> <polyline points="9 18 15 12 9 6"></polyline> </svg> </symbol> <symbol id="svg-doc" viewBox="0 0 24 24"> <title>Document</title> <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-file"> <path d="M13 2H6a2 2 0 0 0-2 2v16a2 2 0 0 0 2 2h12a2 2 0 0 0 2-2V9z"></path><polyline points="13 2 13 9 20 9"></polyline> </svg> </symbol> <!-- Feather. MIT License: https://github.com/feathericons/feather/blob/master/LICENSE --> <symbol id="svg-external-link" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-external-link"> <title id="svg-external-link-title">(external link)</title> <path d="M18 13v6a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V8a2 2 0 0 1 2-2h6"></path><polyline points="15 3 21 3 21 9"></polyline><line x1="10" y1="14" x2="21" y2="3"></line> </symbol> </svg> <div class="side-bar"> <div class="site-header"> <a href="/" class="site-title lh-tight"> ARSN Work Blog </a> <a href="#" id="menu-button" class="site-button"> <svg viewBox="0 0 24 24" class="icon"><use xlink:href="#svg-menu"></use></svg> </a> </div> <nav aria-label="Main" id="site-nav" class="site-nav"> <ul class="nav-list"><li class="nav-list-item"><a href="/" class="nav-list-link">Home</a></li><li class="nav-list-item"><a href="#" class="nav-list-expander" aria-label="toggle links in sc_klf4bc category"> <svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg> </a><a href="/docs/sc_klf4bc/" class="nav-list-link">sc_klf4bc</a><ul class="nav-list "><li class="nav-list-item "><a href="/docs/sc_klf4bc/2022-12-12-sc_breastmultiome-epiconfig_filterset.html" class="nav-list-link">22-12-13, KLF4 BC Initial Processing</a></li></ul></li><li class="nav-list-item"><a href="#" class="nav-list-expander" aria-label="toggle links in ravnica category"> <svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg> </a><a href="/docs/ravnica/" class="nav-list-link">ravnica</a><ul class="nav-list "><li class="nav-list-item "><a href="/docs/ravnica/2022-01-25-post-0073.html" class="nav-list-link">Dropbox Upload and Download with Curl and API Key</a></li><li class="nav-list-item "><a href="/docs/ravnica/2022-05-18-post-0086.html" class="nav-list-link">Cron Info for Automatically Running bcl2fastq</a></li><li class="nav-list-item "><a href="/docs/ravnica/2022-12-05-ravnica-global_install_initial.html" class="nav-list-link">22-12-05, Global Installs - Initial</a></li><li class="nav-list-item "><a href="/docs/ravnica/2022-12-05-ravnica-local_install_initial.html" class="nav-list-link">22-12-05, Local Installs - Initial</a></li><li class="nav-list-item "><a href="/docs/ravnica/2022-12-05-ravnica-mamba_install_initial.html" class="nav-list-link">22-12-05, Mamba Install - Initial</a></li><li class="nav-list-item "><a href="/docs/ravnica/2022-12-05-ravnica-mamba_setup.html" class="nav-list-link">22-12-05, Mamba Setup</a></li><li class="nav-list-item "><a href="/docs/ravnica/2022-12-05-ravnica-r_4.2.2_library_info.html" class="nav-list-link">22-12-05, R 4.2.2 Library Info</a></li><li class="nav-list-item "><a href="/docs/ravnica/2022-12-06-ravnica-example_bashprofile.html" class="nav-list-link">22-12-06, Example Bash Profile</a></li><li class="nav-list-item "><a href="/docs/ravnica/2022-12-06-ravnica-venv_macs3.html" class="nav-list-link">22-12-06, VENV for Macs3</a></li></ul></li><li class="nav-list-item"><a href="#" class="nav-list-expander" aria-label="toggle links in sc_breastmultiome category"> <svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg> </a><a href="/docs/sc_breastmultiome/" class="nav-list-link">sc_breastmultiome</a><ul class="nav-list "><li class="nav-list-item "><a href="/docs/sc_breastmultiome/2022-12-12-sc_breastmultiome-epiconfig.html" class="nav-list-link">22-12-12, Epiconfig Runs on Full and Subset Data</a></li><li class="nav-list-item "><a href="/docs/sc_breastmultiome/2022-12-12-sc_breastmultiome-project_info.html" class="nav-list-link">22-12-12, Project Info</a></li><li class="nav-list-item "><a href="/docs/sc_breastmultiome/2023-01-12-sc_breastmultiome-epiconfig-downsample.html" class="nav-list-link">23-01-12, Epiconfig Run on Downampled Epithelial</a></li><li class="nav-list-item "><a href="/docs/sc_breastmultiome/2023-01-24-sc_breastmultiome-epiconfig-norm_luma.html" class="nav-list-link">23-01-24, Normal and LumA Subsets</a></li><li class="nav-list-item "><a href="/docs/sc_breastmultiome/2023-01-30-sc_breastmultiome-epiconfig-cnvnormal.html" class="nav-list-link">23-01-30, CNV Normal Calls</a></li><li class="nav-list-item "><a href="/docs/sc_breastmultiome/2023-02-03-sc_breastmultiome-epiconfig-callednormalCNV.html" class="nav-list-link">23-02-03, Normal Samples with Normal CNV Calls</a></li><li class="nav-list-item "><a href="/docs/sc_breastmultiome/2023-01-17-sc_breastmultiome-epiconfig-pam50umaps.html" class="nav-list-link">23-01-17, PAM50 UMAPs</a></li></ul></li><li class="nav-list-item"><a href="#" class="nav-list-expander" aria-label="toggle links in sc_dm6tau category"> <svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg> </a><a href="/docs/sc_dm6tau/" class="nav-list-link">sc_dm6tau</a><ul class="nav-list "><li class="nav-list-item "><a href="/docs/sc_dm6tau/2021-10-07-post-0055.html" class="nav-list-link">Drosophilia Tau Preliminary DA</a></li><li class="nav-list-item "><a href="/docs/sc_dm6tau/2022-01-08-post-0069.html" class="nav-list-link">Drosophilia Tau Final DA</a></li><li class="nav-list-item "><a href="/docs/sc_dm6tau/2022-01-21-post-0072.html" class="nav-list-link">Drosophilia Tau DA Results Test Analysis</a></li></ul></li><li class="nav-list-item"><a href="#" class="nav-list-expander" aria-label="toggle links in scitools category"> <svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg> </a><a href="/docs/scitools/" class="nav-list-link">scitools</a><ul class="nav-list "><li class="nav-list-item "><a href="/docs/scitools/2020-12-17-post-0005.html" class="nav-list-link">Scitools scATAC Metrics</a></li><li class="nav-list-item "><a href="/docs/scitools/2021-02-18-post-0006.html" class="nav-list-link">Scitools Scrublet Function</a></li><li class="nav-list-item "><a href="/docs/scitools/2021-05-08-post-0009.html" class="nav-list-link">Single Cell Alignments with no Alt Chromosomes</a></li><li class="nav-list-item "><a href="/docs/scitools/2022-04-03-post-0077.html" class="nav-list-link">scRNA-seq First Pass Pipeline - 220324_HB_Coassay_RNA</a></li><li class="nav-list-item "><a href="/docs/scitools/2022-04-11-post-0080.html" class="nav-list-link">scRNA-seq Second Pass Pipeline - 220324_HB_Coassay_RNA</a></li></ul></li><li class="nav-list-item"><a href="#" class="nav-list-expander" aria-label="toggle links in archive_misc category"> <svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg> </a><a href="/docs/archive_misc/" class="nav-list-link">archive_misc</a><ul class="nav-list "><li class="nav-list-item "><a href="/docs/archive_misc/2020-10-09-post-0003.html" class="nav-list-link">Two Week Update: 2020-10-09</a></li><li class="nav-list-item "><a href="/docs/archive_misc/2020-11-09-post-0004.html" class="nav-list-link">Two Week Update: 2020-11-09</a></li><li class="nav-list-item "><a href="/docs/archive_misc/2021-05-05-post-0007.html" class="nav-list-link">NextSeq Fidelity Evaluation</a></li><li class="nav-list-item "><a href="/docs/archive_misc/2021-11-04-post-0064.html" class="nav-list-link">Contamination References</a></li><li class="nav-list-item "><a href="/docs/archive_misc/2022-03-01-post-0074.html" class="nav-list-link">OneDrive Upload and Download with Rclone</a></li><li class="nav-list-item "><a href="/docs/archive_misc/2022-04-21-post-0081.html" class="nav-list-link">scRNA-seq and scATAC-seq Data Analysis DREAM Challenge - Dataset 2 Data Generation</a></li><li class="nav-list-item "><a href="/docs/archive_misc/2022-05-18-post-0084.html" class="nav-list-link">Celsee and Docker Info</a></li><li class="nav-list-item "><a href="/docs/archive_misc/2022-05-20-post-0089.html" class="nav-list-link">Final O'Roak Lab Data Locations</a></li><li class="nav-list-item "><a href="/docs/archive_misc/2020-10-07-post-0001.html" class="nav-list-link">Example</a></li></ul></li><li class="nav-list-item"><a href="#" class="nav-list-expander" aria-label="toggle links in archive_spark category"> <svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg> </a><a href="/docs/archive_spark/" class="nav-list-link">archive_spark</a><ul class="nav-list "><li class="nav-list-item "><a href="/docs/archive_spark/2021-05-11-post-0012.html" class="nav-list-link">21-05-11, SPARK Mosaic List, Summary Counts of Annotations</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-05-28-post-0015.html" class="nav-list-link">21-05-28, SPARK RefSeq CDS and Splice Site Reference Generation</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-05-30-post-0016.html" class="nav-list-link">21-05-30, SPARK Joint Coverage Counter Tool Fidelity</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-05-30-post-0017.html" class="nav-list-link">21-05-30, SPARK Joint Coverage Plots</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-06-02-post-0018.html" class="nav-list-link">21-06-02, SPARK Joint Coverage Table Fix</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-06-03-post-0019.html" class="nav-list-link">21-06-03, SPARK Mosaic List FRACTION_MORE Annotations</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-06-03-post-0020.html" class="nav-list-link">21-06-03, SPARK Mosaic Index</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-06-10-post-0021.html" class="nav-list-link">21-06-10, SPARK Gene Set Joint Coverage</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-06-22-post-0023.html" class="nav-list-link">21-06-22, SPARK Preliminary Pass 01, Filtering and Classifications</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-06-22-post-0024.html" class="nav-list-link">21-06-22, SPARK Preliminary Pass 02, Loading for Coverage and SNV Count Plots</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-06-26-post-0025.html" class="nav-list-link">21-06-26, SPARK Preliminary Pass 02b, Coverage and SNV Count Plots as Hists</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-06-28-post-0026.html" class="nav-list-link">21-06-28, SPARK Preliminary Pass 03, Allele Fraction Histograms</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-07-06-post-0027.html" class="nav-list-link">21-07-06, SPARK Preliminary Pass 04, Coverages and SNV Counts with Joint Family Coverages</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-07-14-post-0030.html" class="nav-list-link">21-07-14, SPARK Preliminary Pass 5, Final Filtering</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-07-17-post-0031.html" class="nav-list-link">21-07-17, SPARK Preliminary Pass 6a and 6b, Simplex Trio Coverages, SNV Counts, and AFs</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-07-17-post-0032.html" class="nav-list-link">21-07-17, SPARK Preliminary Pass 7a and 7b, Multiplex Quad Coverages, SNV Counts, and AFs</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-07-26-post-0036.html" class="nav-list-link">21-07-26, SPARK First Pass, Synonymous SNV Splice Distances</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-07-26-post-0037.html" class="nav-list-link">21-07-26, SPARK Preliminary Outlier Detection, Secondary Pass</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-07-26-post-0038.html" class="nav-list-link">21-07-26, SPARK Final Table Creation</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-07-27-post-0039.html" class="nav-list-link">21-07-27, SPARK Mosaic Index, July 27</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-08-03-post-0041.html" class="nav-list-link">21-08-03, SPARK High Confidence Allele Frequencies 01</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-08-03-post-0042.html" class="nav-list-link">21-08-03, SPARK High Confidence Synonymous Splice Distances 01</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-08-03-post-0043.html" class="nav-list-link">21-08-03, SPARK Burden Tables Autosomal 01</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-08-09-post-0044.html" class="nav-list-link">21-08-09, SPARK Burden Tables Gene Sets 01</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-08-26-post-0047.html" class="nav-list-link">21-08-26, SPARK IGV Shots of High Confident Calls</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-09-08-post-0048.html" class="nav-list-link">21-09-08, SPARK Burden Tables Gene Sets 02</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-09-14-post-0049.html" class="nav-list-link">21-09-14, SPARK Burden Tables Gene Sets 03</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-09-15-post-0050.html" class="nav-list-link">21-09-15, SPARK Concatenated High Confidence Table Checks 01</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-01-post-0051.html" class="nav-list-link">21-10-1, SPARK Final Sample List</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-01-post-0052.html" class="nav-list-link">21-10-01, SPARK Final IGV and Cohort Count Filter</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-06-post-0053.html" class="nav-list-link">21-10-06, SPARK LGD and NS List</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-06-post-0054.html" class="nav-list-link">21-10-06, SPARK Burden Tables Gene Sets 04</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-10-post-0056.html" class="nav-list-link">21-10-10, SPARK Summary of Final Lists</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-10-post-0057.html" class="nav-list-link">21-10-10, SPARK Final List Allele Frequency Distributions</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-11-post-0058.html" class="nav-list-link">21-10-11, SPARK Monozygotic Twins in Quad Families</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-11-post-0059.html" class="nav-list-link">21-10-11, SPARK Burden Tables Gene Sets, Final 01</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-13-post-0060.html" class="nav-list-link">21-10-13, SPARK Burden w/ REVEL Scores Gene Sets, 01</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-13-post-0061.html" class="nav-list-link">21-10-13, SPARK Parental Trans Mosaics Breakdown</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-13-post-0062.html" class="nav-list-link">21-10-13, SPARK HC Tables Against SPARK 167</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-14-post-0063.html" class="nav-list-link">21-10-14, SPARK Concat Variant Lists to DNV List Comparison</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-11-30-post-0066.html" class="nav-list-link">21-11-30, SPARK SSC List Formatting to Current SPARK</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-11-30-post-0067.html" class="nav-list-link">21-11-30, SPARK Burden Tables Gene Sets, SPARK+SSC</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-12-03-post-0068.html" class="nav-list-link">21-12-03, SPARK VEP Annotations</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2022-01-18-post-0070.html" class="nav-list-link">22-01-18, SPARK SSC hg38 Transformed Burden, 01</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2022-01-19-post-0071.html" class="nav-list-link">22-01-19, SPARK Pre Best Practices Filter, Model Scores</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2022-03-19-post-0076.html" class="nav-list-link">22-03-19, SPARK WES2 and WES3 List Inconsistencies</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2022-04-08-post-0079.html" class="nav-list-link">22-04-08, SPARK WES2 and WES3 Outliers by Counts and Coverage</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2022-04-28-post-0082.html" class="nav-list-link">22-04-28, SPARK Mosaic Index</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2022-05-03-post-0083.html" class="nav-list-link">22-05-03, SPARK IGV HC List Screenshots, 2022</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2022-05-20-post-0088.html" class="nav-list-link">22-05-20, SPARK Post-Hoc Cohort, IGV, and iWES Filter Lists</a></li></ul></li><li class="nav-list-item"><a href="#" class="nav-list-expander" aria-label="toggle links in archive_ssc category"> <svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg> </a><a href="/docs/archive_ssc/" class="nav-list-link">archive_ssc</a><ul class="nav-list "><li class="nav-list-item "><a href="/docs/archive_ssc/2020-10-09-post-0002.html" class="nav-list-link">SSC Final Validation Tables with Annotations</a></li></ul></li><li class="nav-list-item"><a href="#" class="nav-list-expander" aria-label="toggle links in archive_sc_tbr1 category"> <svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg> </a><a href="/docs/archive_sc_tbr1/" class="nav-list-link">archive_sc_tbr1</a><ul class="nav-list "><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-05-07-post-0008.html" class="nav-list-link">Single Cell TBR1 Preliminary Evaluations</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-05-09-post-0010.html" class="nav-list-link">Single Cell TBR1 Coembedding Tests</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-05-11-post-0011.html" class="nav-list-link">Single Cell TBR1 ENCODE DHS Index Tests</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-05-27-post-0013.html" class="nav-list-link">Single Cell TBR1 ENCODE DHS Index Final UMAP and Clusters</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-05-28-post-0014.html" class="nav-list-link">Single Cell TBR1 Cluster DA</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-06-10-post-0022.html" class="nav-list-link">Single Cell TBR1 Cicero/Monocle3 with TSS > 2.7</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-07-09-post-0028.html" class="nav-list-link">Single Cell TBR1 ChromVAR with TSS > 2.7</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-07-12-post-0029.html" class="nav-list-link">Single Cell TBR1 ChromVAR with TSS > 1.5</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-07-18-post-0033.html" class="nav-list-link">Single Cell TBR1 BioCluster/Genotype/Line/Time DA with TSS > 2.7</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-07-19-post-0034.html" class="nav-list-link">Single Cell TBR1 Misc Figures - ChromVAR Compare, FDR Hists, 3D</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-07-26-post-0035.html" class="nav-list-link">Single Cell TBR1 BioCluster/Genotype/Line/Time DA with TSS > 2.7, Filtering Ambiguous Groups</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-07-29-post-0040.html" class="nav-list-link">Single Cell TBR1 Bigwig Tracks</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-08-12-post-0045.html" class="nav-list-link">Single Cell TBR1 Cistopic Boxplots and UMAPs</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-08-12-post-0046.html" class="nav-list-link">Single Cell TBR1 ChIP-Seq Peak DA</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-11-28-post-0065.html" class="nav-list-link">Single Cell TBR1 Pseudo-Bulk Depth Tests</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2022-05-18-post-0087.html" class="nav-list-link">SRA Download Info for 'TBR1 Regulates Autism Risk Genes in the Developing Neocortex'</a></li></ul></li><li class="nav-list-item active"><a href="#" class="nav-list-expander" aria-label="toggle links in archive_amplicon category"> <svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg> </a><a href="/docs/archive_amplicon/" class="nav-list-link">archive_amplicon</a><ul class="nav-list "><li class="nav-list-item active"><a href="/docs/archive_amplicon/2022-03-07-post-0075.html" class="nav-list-link active">Shotgun Sequencing of Amplicon Sequence, 220305</a></li><li class="nav-list-item "><a href="/docs/archive_amplicon/2022-04-06-post-0078.html" class="nav-list-link">Shotgun Sequencing of Amplicon Sequence, 220406</a></li><li class="nav-list-item "><a href="/docs/archive_amplicon/2022-05-18-post-0085.html" class="nav-list-link">Example of HDR Processing via BWA Alignment</a></li></ul></li></ul> </nav> <footer class="site-footer"> This site uses <a href="https://github.com/just-the-docs/just-the-docs">Just the Docs</a>, a documentation theme for Jekyll. </footer> </div> <div class="main" id="top"> <div id="main-header" class="main-header"> <div class="search"> <div class="search-input-wrap"> <input type="text" id="search-input" class="search-input" tabindex="0" placeholder="Search ARSN Work Blog" aria-label="Search ARSN Work Blog" autocomplete="off"> <label for="search-input" class="search-label"><svg viewBox="0 0 24 24" class="search-icon"><use xlink:href="#svg-search"></use></svg></label> </div> <div id="search-results" class="search-results"></div> </div> <nav aria-label="Auxiliary" class="aux-nav"> <ul class="aux-nav-list"> <li class="aux-nav-list-item"> <a href="//github.com/just-the-docs/just-the-docs" class="site-button" > Just the Docs on GitHub </a> </li> </ul> </nav> </div> <div id="main-content-wrap" class="main-content-wrap"> <nav aria-label="Breadcrumb" class="breadcrumb-nav"> <ol class="breadcrumb-nav-list"> <li class="breadcrumb-nav-list-item"><a href="/docs/archive_amplicon/">archive_amplicon</a></li> <li class="breadcrumb-nav-list-item"><span>Shotgun Sequencing of Amplicon Sequence, 220305</span></li> </ol> </nav> <div id="main-content" class="main-content" role="main"> <p>Dir: <code class="language-plaintext highlighter-rouge">/home/groups/oroaklab/nishida/amplicon_220305</code> <br />Run: <code class="language-plaintext highlighter-rouge">/home/groups/oroaklab/nishida/amplicon_220305/run.txt</code></p> <p>This describes how to find our desired edit from shotgun sequencing of amplicon sequence. In lieu of standard amplicon input, we are taking the reads, aligning to hg38, and using GATKâ€™s Haplotype Caller to identify the changes. Reads are being treated as paired-end and as a side-note about half the reads can be merged by overlap.</p> <p>Since this is being done relative to hg38 and not a specific amplicon sequence we have to identify what we are looking for relative to the genome. The region of interest in TBR1 is here:</p> <div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code># with the desired editing being designated with a '-'
# and the NHEJ region from G4 defined by '*'
&gt;chr2:161,416,779-161,416,834
ref  - GTTCCCGTACCCCG*GCCAGCACGGACCGG*CGCACCCCGCCTTCTCCATCGGCAGCC
edit - GTTCCCGTACCCCG*GCCAGCACGGACCGG*CGCA-CCCGCCTTCTCCATCGGCAGCC
</code></pre></div></div> <p>This makes 161,416,811 the site where ACCCC becomes ACCC which will show up as AC to A in the VCFs from Haplotype Caller.</p> <p>For each sample, the following output is provided:</p> <ul> <li>SAMPLE - Sample ID.</li> <li>EDIT_T_OR_F - True if the variant 161416811 AC to A is called. False if otherwise.</li> <li>DEPTH - The depth at 161416811 if EDIT_T_OR_F is True. NA if False.</li> <li>ALT_DEPTH - The depth of the edit at 161416811 if EDIT_T_OR_F is True. NA if False.</li> <li>EDIT_FREQ - ALT_DEPTH / DEPTH</li> <li>NUM_VARS_IN_NHEJ_REGION - The number of other variants called in the NHEJ region, 161416793-161416808.</li> <li>NUM_VARS_WITHIN_10_BP - The number of other variants called +/- 10bp from 161416811 (161416801-161416821).</li> <li>NUM_VARS_WITHIN_25_BP - The number of other variants called +/- 25bp from 161416811 (161416786-161416836).</li> <li>NUM_VARS_WITHIN_50_BP - The number of other variants called +/- 50bp from 161416811 (161416761-161416861).</li> <li>READ_DEPTH_AT_161416811 - Number of reads at the site via GATK DepthOfCoverage (not going to be the same as the depths counted via Haplotype Caller due to local alignment)</li> </ul> <p>This output is copy+pasted below here. This indicates the edit was found in 62 of the 156 samples. It was also found in the NTC3 sample which I am assuming is a control so there might be issues.</p> <div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>#SAMPLE	EDIT_T_OR_F	DEPTH	ALT_DEPTH	EDIT_FREQ	NUM_VARS_IN_NHEJ_REGION	NUM_VARS_WITHIN_10_BP	NUM_VARS_WITHIN_25_BP	NUM_VARS_WITHIN_50_BP	READ_DEPTH_AT_161416811
100	FALSE	NA	NA	NA	1	0	1	1	10296
101	FALSE	NA	NA	NA	1	0	1	1	6196
102	TRUE	2782	908	0.326383896477354	0	0	0	0	6618
103	TRUE	2176	877	0.403033088235294	2	0	2	2	3364
104	TRUE	2892	1362	0.470954356846473	0	0	0	1	4995
105	FALSE	NA	NA	NA	2	0	2	2	5074
106	TRUE	2697	1457	0.540229885057471	0	0	0	0	6571
107	TRUE	2793	1057	0.378446115288221	0	0	0	0	7373
108	FALSE	NA	NA	NA	1	0	1	1	5867
109	TRUE	2738	1123	0.410153396639883	0	0	0	0	6667
110	FALSE	NA	NA	NA	1	0	1	1	4203
111	FALSE	NA	NA	NA	1	0	1	1	4760
112	FALSE	NA	NA	NA	1	0	1	1	3917
113	TRUE	2827	1168	0.413158825610187	0	0	0	0	9585
114	FALSE	NA	NA	NA	0	0	0	0	1927
115	TRUE	2579	1374	0.532764637456378	0	0	0	0	5415
116	TRUE	3050	914	0.299672131147541	0	0	0	1	8658
117	TRUE	2793	1314	0.47046186895811	0	0	0	0	7584
118	FALSE	NA	NA	NA	1	0	1	1	8982
119	FALSE	NA	NA	NA	1	0	1	1	9123
120	FALSE	NA	NA	NA	0	0	0	0	0
121	FALSE	NA	NA	NA	1	0	1	1	7319
122	TRUE	2806	1297	0.462223806129722	0	0	0	0	7244
123	TRUE	2802	1235	0.440756602426838	0	0	0	0	7008
124	TRUE	2191	1994	0.910086718393428	0	0	0	0	3191
125	TRUE	2794	1191	0.426270579813887	0	0	0	0	7752
126	TRUE	2648	1064	0.401812688821752	0	0	0	0	5297
127	TRUE	2757	879	0.318824809575626	0	0	0	0	6590
128	TRUE	2674	1037	0.387808526551982	0	0	0	0	5252
129	FALSE	NA	NA	NA	1	0	1	1	6935
130	TRUE	2614	1936	0.740627390971691	0	0	0	0	5507
131	TRUE	2638	1099	0.416603487490523	1	0	1	1	8718
132	FALSE	NA	NA	NA	1	0	1	1	1
133	FALSE	NA	NA	NA	3	1	3	3	7580
134	TRUE	2566	913	0.355806703039751	0	0	0	0	4057
135	FALSE	NA	NA	NA	1	0	1	1	8615
136	FALSE	NA	NA	NA	1	0	1	1	6476
137	TRUE	2772	754	0.272005772005772	0	0	0	0	7440
138	FALSE	NA	NA	NA	1	0	1	1	3651
139	FALSE	NA	NA	NA	1	0	1	1	6580
140	FALSE	NA	NA	NA	2	0	2	2	60
141	FALSE	NA	NA	NA	0	0	0	0	0
142	TRUE	2541	1668	0.656434474616293	2	0	2	2	6482
143	TRUE	3159	834	0.264007597340931	0	0	0	0	7610
144	TRUE	2837	1366	0.4814945364822	0	0	0	1	5891
145	FALSE	NA	NA	NA	1	0	1	1	7478
146	FALSE	NA	NA	NA	1	0	1	1	7069
147	FALSE	NA	NA	NA	3	1	3	3	4833
148	FALSE	NA	NA	NA	1	0	1	1	4689
149	FALSE	NA	NA	NA	1	0	1	1	3455
150	TRUE	2832	1124	0.396892655367232	1	0	1	1	4757
151	FALSE	NA	NA	NA	1	0	1	1	7004
152	TRUE	1700	1522	0.895294117647059	2	0	2	2	2767
153	TRUE	2832	1176	0.415254237288136	0	0	0	0	9232
154	FALSE	NA	NA	NA	1	0	1	1	8615
155	FALSE	NA	NA	NA	1	0	1	1	4943
156	FALSE	NA	NA	NA	1	0	1	1	1483
157	FALSE	NA	NA	NA	1	0	1	1	8740
158	FALSE	NA	NA	NA	1	0	1	1	4472
159	FALSE	NA	NA	NA	0	0	0	0	0
160	TRUE	143	89	0.622377622377622	0	0	0	0	93
161	TRUE	2278	2135	0.937225636523266	0	0	0	0	3544
162	FALSE	NA	NA	NA	1	0	1	1	10034
163	FALSE	NA	NA	NA	1	0	1	1	8007
164	FALSE	NA	NA	NA	1	0	1	1	8149
165	FALSE	NA	NA	NA	1	0	1	1	2059
166	TRUE	2050	1824	0.889756097560976	2	0	2	2	6024
167	FALSE	NA	NA	NA	1	0	1	1	8825
168	FALSE	NA	NA	NA	3	1	3	3	3551
169	TRUE	1531	398	0.259960809928152	0	0	0	0	1461
170	FALSE	NA	NA	NA	2	0	2	2	2909
171	FALSE	NA	NA	NA	1	0	1	1	7104
172	TRUE	1775	555	0.312676056338028	1	0	1	1	2754
173	TRUE	2795	1408	0.503756708407871	2	0	2	2	8365
174	FALSE	NA	NA	NA	1	0	1	1	4731
175	TRUE	2331	686	0.294294294294294	1	0	1	1	8228
176	FALSE	NA	NA	NA	0	0	0	0	7518
177	TRUE	2382	985	0.413518052057095	0	0	0	0	4829
178	FALSE	NA	NA	NA	3	1	3	3	3881
179	TRUE	2827	1336	0.472585779978776	0	0	0	0	8273
180	TRUE	2283	2001	0.876478318002628	2	0	2	2	6998
181	FALSE	NA	NA	NA	1	0	1	1	5274
182	FALSE	NA	NA	NA	0	0	0	0	132
183	FALSE	NA	NA	NA	0	0	0	0	88
184	FALSE	NA	NA	NA	2	0	2	2	6233
185	TRUE	2740	1786	0.651824817518248	0	0	0	0	7204
186	FALSE	NA	NA	NA	3	1	3	3	3485
187	FALSE	NA	NA	NA	3	1	3	3	5168
188	TRUE	2789	1109	0.39763356041592	0	0	0	0	8810
189	FALSE	NA	NA	NA	2	1	3	3	7601
190	TRUE	2770	1361	0.491335740072202	0	0	0	0	7554
191	FALSE	NA	NA	NA	1	0	1	1	5427
192	TRUE	2149	1926	0.896230805025593	0	0	0	0	2857
193	FALSE	NA	NA	NA	1	0	1	1	596
194	FALSE	NA	NA	NA	1	0	1	1	39
195	TRUE	212	92	0.433962264150943	0	0	0	0	153
196	TRUE	7	5	0.714285714285714	0	0	0	0	7
197	FALSE	NA	NA	NA	1	0	1	1	32
198	TRUE	89	44	0.49438202247191	0	0	0	0	75
199	FALSE	NA	NA	NA	1	0	1	1	250
200	FALSE	NA	NA	NA	0	0	0	0	117
201	FALSE	NA	NA	NA	1	0	1	1	39
202	FALSE	NA	NA	NA	3	1	3	3	324
203	FALSE	NA	NA	NA	0	0	0	0	0
204	TRUE	13	9	0.692307692307692	0	0	0	0	11
205	TRUE	477	125	0.262054507337526	0	0	0	0	314
206	FALSE	NA	NA	NA	1	0	1	1	398
207	FALSE	NA	NA	NA	1	0	1	1	258
208	TRUE	212	154	0.726415094339623	0	0	0	0	156
209	FALSE	NA	NA	NA	1	0	1	1	12
210	FALSE	NA	NA	NA	1	0	1	1	383
211	FALSE	NA	NA	NA	0	0	0	0	1
212	FALSE	NA	NA	NA	1	0	1	1	130
213	TRUE	17	10	0.588235294117647	2	1	2	3	16
214	FALSE	NA	NA	NA	1	0	1	1	494
215	FALSE	NA	NA	NA	0	0	0	0	107
216	FALSE	NA	NA	NA	1	0	1	1	294
217	FALSE	NA	NA	NA	1	0	1	1	105
218	TRUE	47	43	0.914893617021277	2	0	2	2	39
219	FALSE	NA	NA	NA	1	0	1	1	150
220	TRUE	583	166	0.284734133790738	0	0	0	0	474
221	FALSE	NA	NA	NA	1	0	1	1	200
222	TRUE	452	230	0.508849557522124	0	0	0	0	400
223	FALSE	NA	NA	NA	0	0	0	0	394
224	FALSE	NA	NA	NA	0	0	0	0	162
225	TRUE	441	398	0.90249433106576	0	0	0	0	347
226	TRUE	675	220	0.325925925925926	0	0	0	0	571
227	TRUE	55	39	0.709090909090909	0	0	0	0	38
228	FALSE	NA	NA	NA	0	0	0	0	0
229	FALSE	NA	NA	NA	3	1	3	3	19
230	FALSE	NA	NA	NA	1	0	1	1	6
231	TRUE	311	142	0.456591639871383	2	0	2	2	226
232	FALSE	NA	NA	NA	1	0	1	1	220
233	FALSE	NA	NA	NA	1	0	1	1	194
234	TRUE	116	17	0.146551724137931	0	0	0	0	86
235	FALSE	NA	NA	NA	0	0	0	0	0
236	FALSE	NA	NA	NA	0	0	0	0	286
237	FALSE	NA	NA	NA	1	0	1	1	281
238	FALSE	NA	NA	NA	2	0	2	2	425
239	FALSE	NA	NA	NA	0	0	0	0	0
240	TRUE	357	125	0.350140056022409	0	0	0	0	266
241	FALSE	NA	NA	NA	1	0	1	1	123
242	FALSE	NA	NA	NA	1	0	1	1	543
243	TRUE	186	155	0.833333333333333	0	0	0	1	169
244	FALSE	NA	NA	NA	2	0	2	2	739
245	FALSE	NA	NA	NA	1	0	1	1	256
246	FALSE	NA	NA	NA	3	1	3	3	289
247	TRUE	245	60	0.244897959183673	0	0	0	0	163
97	FALSE	NA	NA	NA	1	0	1	1	7129
98	FALSE	NA	NA	NA	1	0	1	1	4407
99	TRUE	2542	526	0.206923682140047	1	0	1	1	7855
A136P	TRUE	1003	348	0.346959122632104	0	0	0	0	890
CIRM87	FALSE	NA	NA	NA	0	0	0	0	850
NTC1	FALSE	NA	NA	NA	0	0	0	0	0
NTC2	FALSE	NA	NA	NA	0	0	0	0	1
NTC3	TRUE	94	10	0.106382978723404	0	0	0	0	85
</code></pre></div></div> <p>Here is a list of all the variants/edit appearing more than once and their count.</p> <div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>     62 chr2	161416811	AC	A
     61 chr2	161416756	GCGGCCACTGCTCCCAGTGCCATGTTCCCGTACCCCGGCCAGCA	G
     43 chr2	161416799	ACGGACCGGCGCAC	*,A
     17 chr2	161416582	G	T
     12 chr2	161416799	ACGGACCGGCGCAC	A
     11 chr2	161416796	AGCACGGACCGGC	A
     10 chr2	161416798	C	CGCCG
      9 chr2	161416800	CGG	*,C
      8 chr2	161416803	ACCGGCGCAC	*,A
      8 chr2	161416799	A	C
      7 chr2	161416994	C	CGGTCATAG
      6 chr2	161416796	AGCACGGACCGGCGCAC	A
      6 chr2	161416581	G	T
      5 chr2	161416799	ACGGACCGGCGCAC	*,AGGCGCAC
      5 chr2	161416782	CCCGTACCCCGGCCAGCA	C
      3 chr2	161416798	CACGG	C
      2 chr2	161416995	C	T
      2 chr2	161416994	C	CGG
      2 chr2	161416990	T	TC
      2 chr2	161416988	C	CTCCACGGTCATAG
      2 chr2	161416799	ACGGACCGGCGCAC	A,CCGGACCGGCGCAC
</code></pre></div></div> </div> </div> <div class="search-overlay"></div> </div> </body> </html>
