<!DOCTYPE html> <html lang="en-US"> <head> <meta charset="UTF-8"> <meta http-equiv="X-UA-Compatible" content="IE=Edge"> <title>Shotgun Sequencing of Amplicon Sequence, 220406 - ARSN Work Blog</title> <link rel="shortcut icon" href="/favicon.ico" type="image/x-icon"> <link rel="stylesheet" href="/assets/css/just-the-docs-default.css"> <script src="/assets/js/vendor/lunr.min.js"></script> <script src="/assets/js/just-the-docs.js"></script> <meta name="viewport" content="width=device-width, initial-scale=1"> <!-- Begin Jekyll SEO tag v2.8.0 --> <title>Shotgun Sequencing of Amplicon Sequence, 220406 | ARSN Work Blog</title> <meta name="generator" content="Jekyll v4.3.0" /> <meta property="og:title" content="Shotgun Sequencing of Amplicon Sequence, 220406" /> <meta property="og:locale" content="en_US" /> <meta name="description" content="This is a blog for work projects Andrew Nishida, nishidaa@ohsu.edu, in CEDAR at OHSU." /> <meta property="og:description" content="This is a blog for work projects Andrew Nishida, nishidaa@ohsu.edu, in CEDAR at OHSU." /> <link rel="canonical" href="http://localhost:4000/docs/archive_amplicon/2022-04-06-post-0078.html" /> <meta property="og:url" content="http://localhost:4000/docs/archive_amplicon/2022-04-06-post-0078.html" /> <meta property="og:site_name" content="ARSN Work Blog" /> <meta property="og:type" content="article" /> <meta property="article:published_time" content="2022-04-06T00:00:00-07:00" /> <meta name="twitter:card" content="summary" /> <meta property="twitter:title" content="Shotgun Sequencing of Amplicon Sequence, 220406" /> <script type="application/ld+json"> {"@context":"https://schema.org","@type":"BlogPosting","dateModified":"2022-04-06T00:00:00-07:00","datePublished":"2022-04-06T00:00:00-07:00","description":"This is a blog for work projects Andrew Nishida, nishidaa@ohsu.edu, in CEDAR at OHSU.","headline":"Shotgun Sequencing of Amplicon Sequence, 220406","mainEntityOfPage":{"@type":"WebPage","@id":"http://localhost:4000/docs/archive_amplicon/2022-04-06-post-0078.html"},"url":"http://localhost:4000/docs/archive_amplicon/2022-04-06-post-0078.html"}</script> <!-- End Jekyll SEO tag --> </head> <body> <a class="skip-to-main" href="#main-content">Skip to main content</a> <svg xmlns="http://www.w3.org/2000/svg" style="display: none;"> <symbol id="svg-link" viewBox="0 0 24 24"> <title>Link</title> <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-link"> <path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path> </svg> </symbol> <symbol id="svg-search" viewBox="0 0 24 24"> <title>Search</title> <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-search"> <circle cx="11" cy="11" r="8"></circle><line x1="21" y1="21" x2="16.65" y2="16.65"></line> </svg> </symbol> <symbol id="svg-menu" viewBox="0 0 24 24"> <title>Menu</title> <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-menu"> <line x1="3" y1="12" x2="21" y2="12"></line><line x1="3" y1="6" x2="21" y2="6"></line><line x1="3" y1="18" x2="21" y2="18"></line> </svg> </symbol> <symbol id="svg-arrow-right" viewBox="0 0 24 24"> <title>Expand</title> <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-chevron-right"> <polyline points="9 18 15 12 9 6"></polyline> </svg> </symbol> <symbol id="svg-doc" viewBox="0 0 24 24"> <title>Document</title> <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-file"> <path d="M13 2H6a2 2 0 0 0-2 2v16a2 2 0 0 0 2 2h12a2 2 0 0 0 2-2V9z"></path><polyline points="13 2 13 9 20 9"></polyline> </svg> </symbol> <!-- Feather. MIT License: https://github.com/feathericons/feather/blob/master/LICENSE --> <symbol id="svg-external-link" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-external-link"> <title id="svg-external-link-title">(external link)</title> <path d="M18 13v6a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V8a2 2 0 0 1 2-2h6"></path><polyline points="15 3 21 3 21 9"></polyline><line x1="10" y1="14" x2="21" y2="3"></line> </symbol> </svg> <div class="side-bar"> <div class="site-header"> <a href="/" class="site-title lh-tight"> ARSN Work Blog </a> <a href="#" id="menu-button" class="site-button"> <svg viewBox="0 0 24 24" class="icon"><use xlink:href="#svg-menu"></use></svg> </a> </div> <nav aria-label="Main" id="site-nav" class="site-nav"> <ul class="nav-list"><li class="nav-list-item"><a href="/" class="nav-list-link">Home</a></li><li class="nav-list-item"><a href="#" class="nav-list-expander" aria-label="toggle links in sc_breastmultiome category"> <svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg> </a><a href="/docs/sc_breastmultiome/" class="nav-list-link">sc_breastmultiome</a><ul class="nav-list "><li class="nav-list-item "><a href="/docs/sc_breastmultiome/2022-12-12-sc_breastmultiome-epiconfig_filterset.html" class="nav-list-link">22-12-12, Project Info</a></li><li class="nav-list-item "><a href="/docs/sc_breastmultiome/2022-12-12-sc_breastmultiome-epiconfig_original_fullset.html" class="nav-list-link">22-12-12, Project Info</a></li><li class="nav-list-item "><a href="/docs/sc_breastmultiome/2022-12-12-sc_breastmultiome-project_info.html" class="nav-list-link">22-12-12, Project Info</a></li></ul></li><li class="nav-list-item"><a href="#" class="nav-list-expander" aria-label="toggle links in ravnica category"> <svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg> </a><a href="/docs/ravnica/" class="nav-list-link">ravnica</a><ul class="nav-list "><li class="nav-list-item "><a href="/docs/ravnica/2022-01-25-post-0073.html" class="nav-list-link">Dropbox Upload and Download with Curl and API Key</a></li><li class="nav-list-item "><a href="/docs/ravnica/2022-05-18-post-0086.html" class="nav-list-link">Cron Info for Automatically Running bcl2fastq</a></li><li class="nav-list-item "><a href="/docs/ravnica/2022-12-05-ravnica-global_install_initial.html" class="nav-list-link">22-12-05, Global Installs - Initial</a></li><li class="nav-list-item "><a href="/docs/ravnica/2022-12-05-ravnica-local_install_initial.html" class="nav-list-link">22-12-05, Local Installs - Initial</a></li><li class="nav-list-item "><a href="/docs/ravnica/2022-12-05-ravnica-mamba_install_initial.html" class="nav-list-link">22-12-05, Mamba Install - Initial</a></li><li class="nav-list-item "><a href="/docs/ravnica/2022-12-05-ravnica-mamba_setup.html" class="nav-list-link">22-12-05, Mamba Setup</a></li><li class="nav-list-item "><a href="/docs/ravnica/2022-12-05-ravnica-r_4.2.2_library_info.html" class="nav-list-link">22-12-05, R 4.2.2 Library Info</a></li><li class="nav-list-item "><a href="/docs/ravnica/2022-12-06-ravnica-example_bashprofile.html" class="nav-list-link">22-12-06, Example Bash Profile</a></li><li class="nav-list-item "><a href="/docs/ravnica/2022-12-06-ravnica-venv_macs3.html" class="nav-list-link">22-12-06, VENV for Macs3</a></li></ul></li><li class="nav-list-item"><a href="#" class="nav-list-expander" aria-label="toggle links in scitools category"> <svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg> </a><a href="/docs/scitools/" class="nav-list-link">scitools</a><ul class="nav-list "><li class="nav-list-item "><a href="/docs/scitools/2020-12-17-post-0005.html" class="nav-list-link">Scitools scATAC Metrics</a></li><li class="nav-list-item "><a href="/docs/scitools/2021-02-18-post-0006.html" class="nav-list-link">Scitools Scrublet Function</a></li><li class="nav-list-item "><a href="/docs/scitools/2021-05-08-post-0009.html" class="nav-list-link">Single Cell Alignments with no Alt Chromosomes</a></li><li class="nav-list-item "><a href="/docs/scitools/2022-04-03-post-0077.html" class="nav-list-link">scRNA-seq First Pass Pipeline - 220324_HB_Coassay_RNA</a></li><li class="nav-list-item "><a href="/docs/scitools/2022-04-11-post-0080.html" class="nav-list-link">scRNA-seq Second Pass Pipeline - 220324_HB_Coassay_RNA</a></li></ul></li><li class="nav-list-item"><a href="#" class="nav-list-expander" aria-label="toggle links in sc_dm6tau category"> <svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg> </a><a href="/docs/sc_dm6tau/" class="nav-list-link">sc_dm6tau</a><ul class="nav-list "><li class="nav-list-item "><a href="/docs/sc_dm6tau/2021-10-07-post-0055.html" class="nav-list-link">Drosophilia Tau Preliminary DA</a></li><li class="nav-list-item "><a href="/docs/sc_dm6tau/2022-01-08-post-0069.html" class="nav-list-link">Drosophilia Tau Final DA</a></li><li class="nav-list-item "><a href="/docs/sc_dm6tau/2022-01-21-post-0072.html" class="nav-list-link">Drosophilia Tau DA Results Test Analysis</a></li></ul></li><li class="nav-list-item"><a href="#" class="nav-list-expander" aria-label="toggle links in archive_ssc category"> <svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg> </a><a href="/docs/archive_ssc/" class="nav-list-link">archive_ssc</a><ul class="nav-list "><li class="nav-list-item "><a href="/docs/archive_ssc/2020-10-09-post-0002.html" class="nav-list-link">SSC Final Validation Tables with Annotations</a></li></ul></li><li class="nav-list-item active"><a href="#" class="nav-list-expander" aria-label="toggle links in archive_amplicon category"> <svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg> </a><a href="/docs/archive_amplicon/" class="nav-list-link">archive_amplicon</a><ul class="nav-list "><li class="nav-list-item "><a href="/docs/archive_amplicon/2022-03-07-post-0075.html" class="nav-list-link">Shotgun Sequencing of Amplicon Sequence, 220305</a></li><li class="nav-list-item active"><a href="/docs/archive_amplicon/2022-04-06-post-0078.html" class="nav-list-link active">Shotgun Sequencing of Amplicon Sequence, 220406</a></li><li class="nav-list-item "><a href="/docs/archive_amplicon/2022-05-18-post-0085.html" class="nav-list-link">Example of HDR Processing via BWA Alignment</a></li></ul></li><li class="nav-list-item"><a href="#" class="nav-list-expander" aria-label="toggle links in archive_misc category"> <svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg> </a><a href="/docs/archive_misc/" class="nav-list-link">archive_misc</a><ul class="nav-list "><li class="nav-list-item "><a href="/docs/archive_misc/2020-10-09-post-0003.html" class="nav-list-link">Two Week Update: 2020-10-09</a></li><li class="nav-list-item "><a href="/docs/archive_misc/2020-11-09-post-0004.html" class="nav-list-link">Two Week Update: 2020-11-09</a></li><li class="nav-list-item "><a href="/docs/archive_misc/2021-05-05-post-0007.html" class="nav-list-link">NextSeq Fidelity Evaluation</a></li><li class="nav-list-item "><a href="/docs/archive_misc/2021-11-04-post-0064.html" class="nav-list-link">Contamination References</a></li><li class="nav-list-item "><a href="/docs/archive_misc/2022-03-01-post-0074.html" class="nav-list-link">OneDrive Upload and Download with Rclone</a></li><li class="nav-list-item "><a href="/docs/archive_misc/2022-04-21-post-0081.html" class="nav-list-link">scRNA-seq and scATAC-seq Data Analysis DREAM Challenge - Dataset 2 Data Generation</a></li><li class="nav-list-item "><a href="/docs/archive_misc/2022-05-18-post-0084.html" class="nav-list-link">Celsee and Docker Info</a></li><li class="nav-list-item "><a href="/docs/archive_misc/2022-05-20-post-0089.html" class="nav-list-link">Final O'Roak Lab Data Locations</a></li><li class="nav-list-item "><a href="/docs/archive_misc/2020-10-07-post-0001.html" class="nav-list-link">Example</a></li></ul></li><li class="nav-list-item"><a href="#" class="nav-list-expander" aria-label="toggle links in archive_sc_tbr1 category"> <svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg> </a><a href="/docs/archive_sc_tbr1/" class="nav-list-link">archive_sc_tbr1</a><ul class="nav-list "><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-05-07-post-0008.html" class="nav-list-link">Single Cell TBR1 Preliminary Evaluations</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-05-09-post-0010.html" class="nav-list-link">Single Cell TBR1 Coembedding Tests</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-05-11-post-0011.html" class="nav-list-link">Single Cell TBR1 ENCODE DHS Index Tests</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-05-27-post-0013.html" class="nav-list-link">Single Cell TBR1 ENCODE DHS Index Final UMAP and Clusters</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-05-28-post-0014.html" class="nav-list-link">Single Cell TBR1 Cluster DA</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-06-10-post-0022.html" class="nav-list-link">Single Cell TBR1 Cicero/Monocle3 with TSS > 2.7</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-07-09-post-0028.html" class="nav-list-link">Single Cell TBR1 ChromVAR with TSS > 2.7</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-07-12-post-0029.html" class="nav-list-link">Single Cell TBR1 ChromVAR with TSS > 1.5</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-07-18-post-0033.html" class="nav-list-link">Single Cell TBR1 BioCluster/Genotype/Line/Time DA with TSS > 2.7</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-07-19-post-0034.html" class="nav-list-link">Single Cell TBR1 Misc Figures - ChromVAR Compare, FDR Hists, 3D</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-07-26-post-0035.html" class="nav-list-link">Single Cell TBR1 BioCluster/Genotype/Line/Time DA with TSS > 2.7, Filtering Ambiguous Groups</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-07-29-post-0040.html" class="nav-list-link">Single Cell TBR1 Bigwig Tracks</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-08-12-post-0045.html" class="nav-list-link">Single Cell TBR1 Cistopic Boxplots and UMAPs</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-08-12-post-0046.html" class="nav-list-link">Single Cell TBR1 ChIP-Seq Peak DA</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-11-28-post-0065.html" class="nav-list-link">Single Cell TBR1 Pseudo-Bulk Depth Tests</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2022-05-18-post-0087.html" class="nav-list-link">SRA Download Info for 'TBR1 Regulates Autism Risk Genes in the Developing Neocortex'</a></li></ul></li><li class="nav-list-item"><a href="#" class="nav-list-expander" aria-label="toggle links in archive_spark category"> <svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg> </a><a href="/docs/archive_spark/" class="nav-list-link">archive_spark</a><ul class="nav-list "><li class="nav-list-item "><a href="/docs/archive_spark/2021-05-11-post-0012.html" class="nav-list-link">21-05-11, SPARK Mosaic List, Summary Counts of Annotations</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-05-28-post-0015.html" class="nav-list-link">21-05-28, SPARK RefSeq CDS and Splice Site Reference Generation</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-05-30-post-0016.html" class="nav-list-link">21-05-30, SPARK Joint Coverage Counter Tool Fidelity</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-05-30-post-0017.html" class="nav-list-link">21-05-30, SPARK Joint Coverage Plots</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-06-02-post-0018.html" class="nav-list-link">21-06-02, SPARK Joint Coverage Table Fix</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-06-03-post-0019.html" class="nav-list-link">21-06-03, SPARK Mosaic List FRACTION_MORE Annotations</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-06-03-post-0020.html" class="nav-list-link">21-06-03, SPARK Mosaic Index</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-06-10-post-0021.html" class="nav-list-link">21-06-10, SPARK Gene Set Joint Coverage</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-06-22-post-0023.html" class="nav-list-link">21-06-22, SPARK Preliminary Pass 01, Filtering and Classifications</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-06-22-post-0024.html" class="nav-list-link">21-06-22, SPARK Preliminary Pass 02, Loading for Coverage and SNV Count Plots</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-06-26-post-0025.html" class="nav-list-link">21-06-26, SPARK Preliminary Pass 02b, Coverage and SNV Count Plots as Hists</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-06-28-post-0026.html" class="nav-list-link">21-06-28, SPARK Preliminary Pass 03, Allele Fraction Histograms</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-07-06-post-0027.html" class="nav-list-link">21-07-06, SPARK Preliminary Pass 04, Coverages and SNV Counts with Joint Family Coverages</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-07-14-post-0030.html" class="nav-list-link">21-07-14, SPARK Preliminary Pass 5, Final Filtering</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-07-17-post-0031.html" class="nav-list-link">21-07-17, SPARK Preliminary Pass 6a and 6b, Simplex Trio Coverages, SNV Counts, and AFs</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-07-17-post-0032.html" class="nav-list-link">21-07-17, SPARK Preliminary Pass 7a and 7b, Multiplex Quad Coverages, SNV Counts, and AFs</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-07-26-post-0036.html" class="nav-list-link">21-07-26, SPARK First Pass, Synonymous SNV Splice Distances</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-07-26-post-0037.html" class="nav-list-link">21-07-26, SPARK Preliminary Outlier Detection, Secondary Pass</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-07-26-post-0038.html" class="nav-list-link">21-07-26, SPARK Final Table Creation</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-07-27-post-0039.html" class="nav-list-link">21-07-27, SPARK Mosaic Index, July 27</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-08-03-post-0041.html" class="nav-list-link">21-08-03, SPARK High Confidence Allele Frequencies 01</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-08-03-post-0042.html" class="nav-list-link">21-08-03, SPARK High Confidence Synonymous Splice Distances 01</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-08-03-post-0043.html" class="nav-list-link">21-08-03, SPARK Burden Tables Autosomal 01</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-08-09-post-0044.html" class="nav-list-link">21-08-09, SPARK Burden Tables Gene Sets 01</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-08-26-post-0047.html" class="nav-list-link">21-08-26, SPARK IGV Shots of High Confident Calls</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-09-08-post-0048.html" class="nav-list-link">21-09-08, SPARK Burden Tables Gene Sets 02</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-09-14-post-0049.html" class="nav-list-link">21-09-14, SPARK Burden Tables Gene Sets 03</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-09-15-post-0050.html" class="nav-list-link">21-09-15, SPARK Concatenated High Confidence Table Checks 01</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-01-post-0051.html" class="nav-list-link">21-10-1, SPARK Final Sample List</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-01-post-0052.html" class="nav-list-link">21-10-01, SPARK Final IGV and Cohort Count Filter</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-06-post-0053.html" class="nav-list-link">21-10-06, SPARK LGD and NS List</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-06-post-0054.html" class="nav-list-link">21-10-06, SPARK Burden Tables Gene Sets 04</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-10-post-0056.html" class="nav-list-link">21-10-10, SPARK Summary of Final Lists</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-10-post-0057.html" class="nav-list-link">21-10-10, SPARK Final List Allele Frequency Distributions</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-11-post-0058.html" class="nav-list-link">21-10-11, SPARK Monozygotic Twins in Quad Families</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-11-post-0059.html" class="nav-list-link">21-10-11, SPARK Burden Tables Gene Sets, Final 01</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-13-post-0060.html" class="nav-list-link">21-10-13, SPARK Burden w/ REVEL Scores Gene Sets, 01</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-13-post-0061.html" class="nav-list-link">21-10-13, SPARK Parental Trans Mosaics Breakdown</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-13-post-0062.html" class="nav-list-link">21-10-13, SPARK HC Tables Against SPARK 167</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-14-post-0063.html" class="nav-list-link">21-10-14, SPARK Concat Variant Lists to DNV List Comparison</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-11-30-post-0066.html" class="nav-list-link">21-11-30, SPARK SSC List Formatting to Current SPARK</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-11-30-post-0067.html" class="nav-list-link">21-11-30, SPARK Burden Tables Gene Sets, SPARK+SSC</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-12-03-post-0068.html" class="nav-list-link">21-12-03, SPARK VEP Annotations</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2022-01-18-post-0070.html" class="nav-list-link">22-01-18, SPARK SSC hg38 Transformed Burden, 01</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2022-01-19-post-0071.html" class="nav-list-link">22-01-19, SPARK Pre Best Practices Filter, Model Scores</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2022-03-19-post-0076.html" class="nav-list-link">22-03-19, SPARK WES2 and WES3 List Inconsistencies</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2022-04-08-post-0079.html" class="nav-list-link">22-04-08, SPARK WES2 and WES3 Outliers by Counts and Coverage</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2022-04-28-post-0082.html" class="nav-list-link">22-04-28, SPARK Mosaic Index</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2022-05-03-post-0083.html" class="nav-list-link">22-05-03, SPARK IGV HC List Screenshots, 2022</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2022-05-20-post-0088.html" class="nav-list-link">22-05-20, SPARK Post-Hoc Cohort, IGV, and iWES Filter Lists</a></li></ul></li></ul> </nav> <footer class="site-footer"> This site uses <a href="https://github.com/just-the-docs/just-the-docs">Just the Docs</a>, a documentation theme for Jekyll. </footer> </div> <div class="main" id="top"> <div id="main-header" class="main-header"> <div class="search"> <div class="search-input-wrap"> <input type="text" id="search-input" class="search-input" tabindex="0" placeholder="Search ARSN Work Blog" aria-label="Search ARSN Work Blog" autocomplete="off"> <label for="search-input" class="search-label"><svg viewBox="0 0 24 24" class="search-icon"><use xlink:href="#svg-search"></use></svg></label> </div> <div id="search-results" class="search-results"></div> </div> <nav aria-label="Auxiliary" class="aux-nav"> <ul class="aux-nav-list"> <li class="aux-nav-list-item"> <a href="//github.com/just-the-docs/just-the-docs" class="site-button" > Just the Docs on GitHub </a> </li> </ul> </nav> </div> <div id="main-content-wrap" class="main-content-wrap"> <nav aria-label="Breadcrumb" class="breadcrumb-nav"> <ol class="breadcrumb-nav-list"> <li class="breadcrumb-nav-list-item"><a href="/docs/archive_amplicon/">archive_amplicon</a></li> <li class="breadcrumb-nav-list-item"><span>Shotgun Sequencing of Amplicon Sequence, 220406</span></li> </ol> </nav> <div id="main-content" class="main-content" role="main"> <p>Dir: <code class="language-plaintext highlighter-rouge">/home/groups/oroaklab/nishida/amplicon_220406</code> <br />Run: <code class="language-plaintext highlighter-rouge">/home/groups/oroaklab/nishida/amplicon_220406/run.txt</code></p> <p>This describes how to find our desired edit from shotgun sequencing of amplicon sequence. In lieu of standard amplicon input, we are taking the reads, aligning to hg38, and using GATK’s Haplotype Caller to identify the changes. Reads are being treated as paired-end and as a side-note about half the reads can be merged by overlap.</p> <p>The description of the reference creation, interpretation, and format of the summary is described here: <a href="https://arsnishida.github.io/2022/03/07/post-0075.html">first amplicon sequencing results</a>.</p> <p>For each sample, the following output is provided:</p> <ul> <li>SAMPLE - Sample ID.</li> <li>EDIT_T_OR_F - True if the variant 161416811 AC to A is called. False if otherwise.</li> <li>DEPTH - The depth at 161416811 if EDIT_T_OR_F is True. NA if False.</li> <li>ALT_DEPTH - The depth of the edit at 161416811 if EDIT_T_OR_F is True. NA if False.</li> <li>EDIT_FREQ - ALT_DEPTH / DEPTH</li> <li>NUM_VARS_IN_NHEJ_REGION - The number of other variants called in the NHEJ region, 161416793-161416808.</li> <li>NUM_VARS_WITHIN_10_BP - The number of other variants called +/- 10bp from 161416811 (161416801-161416821).</li> <li>NUM_VARS_WITHIN_25_BP - The number of other variants called +/- 25bp from 161416811 (161416786-161416836).</li> <li>NUM_VARS_WITHIN_50_BP - The number of other variants called +/- 50bp from 161416811 (161416761-161416861).</li> <li>READ_DEPTH_AT_161416811 - Number of reads at the site via GATK DepthOfCoverage (not going to be the same as the depths counted via Haplotype Caller due to local alignment)</li> </ul> <p>This output is copy+pasted below here. The edit was found in the A136P controls and all the other non-control samples with at least 1000 reads of coverage. It was also found in the HDR7_NTC control but it and the HDR6_NTC control had nearly no reads.</p> <div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>#SAMPLE	EDIT_T_OR_F	DEPTH	ALT_DEPTH	EDIT_FREQ	NUM_VARS_IN_NHEJ_REGION	NUM_VARS_WITHIN_10_BP	NUM_VARS_WITHIN_25_BP	NUM_VARS_WITHIN_50_BP	READ_DEPTH_AT_161416811
HDR6_A136P_Control	TRUE	2648	1280	0.483383685800604	0	0	0	0	5652
HDR6_CIRM87	FALSE	NA	NA	NA	0	0	0	0	6117
HDR6_NTC	FALSE	NA	NA	NA	0	0	0	0	16
HDR6_sg4_SYN88_1	TRUE	2721	1004	0.368981991914737	1	0	1	1	6791
HDR6_sg4_SYN88_2	TRUE	2772	1038	0.374458874458874	1	0	1	1	6219
HDR6_sg4_SYN88_3	TRUE	2708	974	0.359675036927622	1	0	1	1	5567
HDR6_sg4_WT88_1	TRUE	2794	1128	0.403722261989979	0	0	0	0	7411
HDR6_sg4_WT88_2	TRUE	2841	1019	0.358676522351285	0	0	0	0	6783
HDR6_sg4_WT88_3	TRUE	2798	959	0.342744817726948	0	0	0	0	6720
HDR7_A136P_Control	TRUE	2768	1299	0.469291907514451	0	0	0	0	7322
HDR7_CIRM87	FALSE	NA	NA	NA	0	0	0	0	5979
HDR7_NTC	TRUE	49	27	0.551020408163265	0	0	0	0	30
HDR7_sg4M_SYN88_1	TRUE	2798	1245	0.444960686204432	0	0	0	0	7936
HDR7_sg4M_SYN88_2	TRUE	2795	1229	0.439713774597496	0	0	0	0	7049
HDR7_sg4M_SYN88_3	TRUE	2797	1256	0.449052556310332	0	0	0	0	7418
HDR7_sg4M_SYN88_4	TRUE	2814	1187	0.42181947405828	0	0	0	0	8328
HDR7_sg4M_WT88_1	TRUE	2760	1315	0.476449275362319	0	0	0	0	6757
HDR7_sg4M_WT88_2	TRUE	2706	1189	0.439393939393939	0	0	0	0	6162
HDR7_sg4M_WT88_3	TRUE	2708	1206	0.445347119645495	0	0	0	0	6155
HDR7_sg4M_WT88_4	TRUE	2758	1292	0.468455402465555	0	0	0	0	6581
HDR7_sg4_SYN88_1	TRUE	2813	1230	0.437255599004621	0	0	0	0	7298
HDR7_sg4_SYN88_2	TRUE	2735	1294	0.473126142595978	0	0	0	0	6037
HDR7_sg4_SYN88_3	TRUE	2682	1159	0.43214019388516	0	0	0	0	6421
HDR7_sg4_SYN88_4	TRUE	2790	1198	0.429390681003584	0	0	0	0	6627
HDR7_sg4_WT88_1	TRUE	2758	1237	0.448513415518492	0	0	0	0	8276
HDR7_sg4_WT88_2	TRUE	2853	1251	0.438485804416404	0	0	0	0	7712
HDR7_sg4_WT88_3	TRUE	1572	637	0.405216284987277	0	0	0	0	1715
HDR7_sg4_WT88_4	TRUE	2767	1310	0.473436935308999	0	0	0	0	7281
</code></pre></div></div> <p>Here is a list of all the variants/edit appearing more than once and their count.</p> <div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>     25 chr2    161416811       AC      A
      5 chr2    161416993       ACCCAGC A
      5 chr2    161416990       TC      T
      5 chr2    161416582       G       T
      3 chr2    161416794       C       A
      3 chr2    161416581       G       T
      2 chr2    161417380       G       A
      2 chr2    161416993       A       AGGT
</code></pre></div></div> </div> </div> <div class="search-overlay"></div> </div> </body> </html>
