<!DOCTYPE html> <html lang="en-US"> <head> <meta charset="UTF-8"> <meta http-equiv="X-UA-Compatible" content="IE=Edge"> <title>21-06-03, SPARK Mosaic Index - ARSN Work Blog</title> <link rel="shortcut icon" href="/favicon.ico" type="image/x-icon"> <link rel="stylesheet" href="/assets/css/just-the-docs-default.css"> <script src="/assets/js/vendor/lunr.min.js"></script> <script src="/assets/js/just-the-docs.js"></script> <meta name="viewport" content="width=device-width, initial-scale=1"> <!-- Begin Jekyll SEO tag v2.8.0 --> <title>21-06-03, SPARK Mosaic Index | ARSN Work Blog</title> <meta name="generator" content="Jekyll v4.3.0" /> <meta property="og:title" content="21-06-03, SPARK Mosaic Index" /> <meta property="og:locale" content="en_US" /> <meta name="description" content="This is a blog for work projects Andrew Nishida, nishidaa@ohsu.edu, in CEDAR at OHSU." /> <meta property="og:description" content="This is a blog for work projects Andrew Nishida, nishidaa@ohsu.edu, in CEDAR at OHSU." /> <link rel="canonical" href="http://localhost:4000/docs/archive_spark/2021-06-03-post-0020.html" /> <meta property="og:url" content="http://localhost:4000/docs/archive_spark/2021-06-03-post-0020.html" /> <meta property="og:site_name" content="ARSN Work Blog" /> <meta property="og:type" content="article" /> <meta property="article:published_time" content="2021-06-03T00:00:00-07:00" /> <meta name="twitter:card" content="summary" /> <meta property="twitter:title" content="21-06-03, SPARK Mosaic Index" /> <script type="application/ld+json"> {"@context":"https://schema.org","@type":"BlogPosting","dateModified":"2021-06-03T00:00:00-07:00","datePublished":"2021-06-03T00:00:00-07:00","description":"This is a blog for work projects Andrew Nishida, nishidaa@ohsu.edu, in CEDAR at OHSU.","headline":"21-06-03, SPARK Mosaic Index","mainEntityOfPage":{"@type":"WebPage","@id":"http://localhost:4000/docs/archive_spark/2021-06-03-post-0020.html"},"url":"http://localhost:4000/docs/archive_spark/2021-06-03-post-0020.html"}</script> <!-- End Jekyll SEO tag --> </head> <body> <a class="skip-to-main" href="#main-content">Skip to main content</a> <svg xmlns="http://www.w3.org/2000/svg" style="display: none;"> <symbol id="svg-link" viewBox="0 0 24 24"> <title>Link</title> <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-link"> <path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path> </svg> </symbol> <symbol id="svg-search" viewBox="0 0 24 24"> <title>Search</title> <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-search"> <circle cx="11" cy="11" r="8"></circle><line x1="21" y1="21" x2="16.65" y2="16.65"></line> </svg> </symbol> <symbol id="svg-menu" viewBox="0 0 24 24"> <title>Menu</title> <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-menu"> <line x1="3" y1="12" x2="21" y2="12"></line><line x1="3" y1="6" x2="21" y2="6"></line><line x1="3" y1="18" x2="21" y2="18"></line> </svg> </symbol> <symbol id="svg-arrow-right" viewBox="0 0 24 24"> <title>Expand</title> <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-chevron-right"> <polyline points="9 18 15 12 9 6"></polyline> </svg> </symbol> <symbol id="svg-doc" viewBox="0 0 24 24"> <title>Document</title> <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-file"> <path d="M13 2H6a2 2 0 0 0-2 2v16a2 2 0 0 0 2 2h12a2 2 0 0 0 2-2V9z"></path><polyline points="13 2 13 9 20 9"></polyline> </svg> </symbol> <!-- Feather. MIT License: https://github.com/feathericons/feather/blob/master/LICENSE --> <symbol id="svg-external-link" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-external-link"> <title id="svg-external-link-title">(external link)</title> <path d="M18 13v6a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V8a2 2 0 0 1 2-2h6"></path><polyline points="15 3 21 3 21 9"></polyline><line x1="10" y1="14" x2="21" y2="3"></line> </symbol> </svg> <div class="side-bar"> <div class="site-header"> <a href="/" class="site-title lh-tight"> ARSN Work Blog </a> <a href="#" id="menu-button" class="site-button"> <svg viewBox="0 0 24 24" class="icon"><use xlink:href="#svg-menu"></use></svg> </a> </div> <nav aria-label="Main" id="site-nav" class="site-nav"> <ul class="nav-list"><li class="nav-list-item"><a href="/" class="nav-list-link">Home</a></li><li class="nav-list-item"><a href="#" class="nav-list-expander" aria-label="toggle links in sc_breastmultiome category"> <svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg> </a><a href="/docs/sc_breastmultiome/" class="nav-list-link">sc_breastmultiome</a><ul class="nav-list "><li class="nav-list-item "><a href="/docs/sc_breastmultiome/2022-12-12-sc_breastmultiome-epiconfig.html" class="nav-list-link">22-12-12, Epiconfig Runs on Full and Subset Data</a></li><li class="nav-list-item "><a href="/docs/sc_breastmultiome/2022-12-12-sc_breastmultiome-project_info.html" class="nav-list-link">22-12-12, Project Info</a></li><li class="nav-list-item "><a href="/docs/sc_breastmultiome/2023-01-12-sc_breastmultiome-epiconfig-downsample.html" class="nav-list-link">23-01-12, Epiconfig Run on Downampled Epithelial</a></li><li class="nav-list-item "><a href="/docs/sc_breastmultiome/2023-01-17-sc_breastmultiome-epiconfig-pam50umaps.html" class="nav-list-link">23-01-17, PAM50 UMAPs</a></li></ul></li><li class="nav-list-item"><a href="#" class="nav-list-expander" aria-label="toggle links in sc_dm6tau category"> <svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg> </a><a href="/docs/sc_dm6tau/" class="nav-list-link">sc_dm6tau</a><ul class="nav-list "><li class="nav-list-item "><a href="/docs/sc_dm6tau/2021-10-07-post-0055.html" class="nav-list-link">Drosophilia Tau Preliminary DA</a></li><li class="nav-list-item "><a href="/docs/sc_dm6tau/2022-01-08-post-0069.html" class="nav-list-link">Drosophilia Tau Final DA</a></li><li class="nav-list-item "><a href="/docs/sc_dm6tau/2022-01-21-post-0072.html" class="nav-list-link">Drosophilia Tau DA Results Test Analysis</a></li></ul></li><li class="nav-list-item"><a href="#" class="nav-list-expander" aria-label="toggle links in ravnica category"> <svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg> </a><a href="/docs/ravnica/" class="nav-list-link">ravnica</a><ul class="nav-list "><li class="nav-list-item "><a href="/docs/ravnica/2022-01-25-post-0073.html" class="nav-list-link">Dropbox Upload and Download with Curl and API Key</a></li><li class="nav-list-item "><a href="/docs/ravnica/2022-05-18-post-0086.html" class="nav-list-link">Cron Info for Automatically Running bcl2fastq</a></li><li class="nav-list-item "><a href="/docs/ravnica/2022-12-05-ravnica-global_install_initial.html" class="nav-list-link">22-12-05, Global Installs - Initial</a></li><li class="nav-list-item "><a href="/docs/ravnica/2022-12-05-ravnica-local_install_initial.html" class="nav-list-link">22-12-05, Local Installs - Initial</a></li><li class="nav-list-item "><a href="/docs/ravnica/2022-12-05-ravnica-mamba_install_initial.html" class="nav-list-link">22-12-05, Mamba Install - Initial</a></li><li class="nav-list-item "><a href="/docs/ravnica/2022-12-05-ravnica-mamba_setup.html" class="nav-list-link">22-12-05, Mamba Setup</a></li><li class="nav-list-item "><a href="/docs/ravnica/2022-12-05-ravnica-r_4.2.2_library_info.html" class="nav-list-link">22-12-05, R 4.2.2 Library Info</a></li><li class="nav-list-item "><a href="/docs/ravnica/2022-12-06-ravnica-example_bashprofile.html" class="nav-list-link">22-12-06, Example Bash Profile</a></li><li class="nav-list-item "><a href="/docs/ravnica/2022-12-06-ravnica-venv_macs3.html" class="nav-list-link">22-12-06, VENV for Macs3</a></li></ul></li><li class="nav-list-item"><a href="#" class="nav-list-expander" aria-label="toggle links in scitools category"> <svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg> </a><a href="/docs/scitools/" class="nav-list-link">scitools</a><ul class="nav-list "><li class="nav-list-item "><a href="/docs/scitools/2020-12-17-post-0005.html" class="nav-list-link">Scitools scATAC Metrics</a></li><li class="nav-list-item "><a href="/docs/scitools/2021-02-18-post-0006.html" class="nav-list-link">Scitools Scrublet Function</a></li><li class="nav-list-item "><a href="/docs/scitools/2021-05-08-post-0009.html" class="nav-list-link">Single Cell Alignments with no Alt Chromosomes</a></li><li class="nav-list-item "><a href="/docs/scitools/2022-04-03-post-0077.html" class="nav-list-link">scRNA-seq First Pass Pipeline - 220324_HB_Coassay_RNA</a></li><li class="nav-list-item "><a href="/docs/scitools/2022-04-11-post-0080.html" class="nav-list-link">scRNA-seq Second Pass Pipeline - 220324_HB_Coassay_RNA</a></li></ul></li><li class="nav-list-item"><a href="#" class="nav-list-expander" aria-label="toggle links in sc_klf4bc category"> <svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg> </a><a href="/docs/sc_klf4bc/" class="nav-list-link">sc_klf4bc</a><ul class="nav-list "><li class="nav-list-item "><a href="/docs/sc_klf4bc/2022-12-12-sc_breastmultiome-epiconfig_filterset.html" class="nav-list-link">22-12-13, KLF4 BC Initial Processing</a></li></ul></li><li class="nav-list-item"><a href="#" class="nav-list-expander" aria-label="toggle links in archive_misc category"> <svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg> </a><a href="/docs/archive_misc/" class="nav-list-link">archive_misc</a><ul class="nav-list "><li class="nav-list-item "><a href="/docs/archive_misc/2020-10-09-post-0003.html" class="nav-list-link">Two Week Update: 2020-10-09</a></li><li class="nav-list-item "><a href="/docs/archive_misc/2020-11-09-post-0004.html" class="nav-list-link">Two Week Update: 2020-11-09</a></li><li class="nav-list-item "><a href="/docs/archive_misc/2021-05-05-post-0007.html" class="nav-list-link">NextSeq Fidelity Evaluation</a></li><li class="nav-list-item "><a href="/docs/archive_misc/2021-11-04-post-0064.html" class="nav-list-link">Contamination References</a></li><li class="nav-list-item "><a href="/docs/archive_misc/2022-03-01-post-0074.html" class="nav-list-link">OneDrive Upload and Download with Rclone</a></li><li class="nav-list-item "><a href="/docs/archive_misc/2022-04-21-post-0081.html" class="nav-list-link">scRNA-seq and scATAC-seq Data Analysis DREAM Challenge - Dataset 2 Data Generation</a></li><li class="nav-list-item "><a href="/docs/archive_misc/2020-10-07-post-0001.html" class="nav-list-link">Example</a></li><li class="nav-list-item "><a href="/docs/archive_misc/2022-05-20-post-0089.html" class="nav-list-link">Final O'Roak Lab Data Locations</a></li><li class="nav-list-item "><a href="/docs/archive_misc/2022-05-18-post-0084.html" class="nav-list-link">Celsee and Docker Info</a></li></ul></li><li class="nav-list-item"><a href="#" class="nav-list-expander" aria-label="toggle links in archive_sc_tbr1 category"> <svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg> </a><a href="/docs/archive_sc_tbr1/" class="nav-list-link">archive_sc_tbr1</a><ul class="nav-list "><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-05-07-post-0008.html" class="nav-list-link">Single Cell TBR1 Preliminary Evaluations</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-05-09-post-0010.html" class="nav-list-link">Single Cell TBR1 Coembedding Tests</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-05-11-post-0011.html" class="nav-list-link">Single Cell TBR1 ENCODE DHS Index Tests</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-05-27-post-0013.html" class="nav-list-link">Single Cell TBR1 ENCODE DHS Index Final UMAP and Clusters</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-05-28-post-0014.html" class="nav-list-link">Single Cell TBR1 Cluster DA</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-06-10-post-0022.html" class="nav-list-link">Single Cell TBR1 Cicero/Monocle3 with TSS > 2.7</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-07-09-post-0028.html" class="nav-list-link">Single Cell TBR1 ChromVAR with TSS > 2.7</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-07-12-post-0029.html" class="nav-list-link">Single Cell TBR1 ChromVAR with TSS > 1.5</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-07-18-post-0033.html" class="nav-list-link">Single Cell TBR1 BioCluster/Genotype/Line/Time DA with TSS > 2.7</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-07-19-post-0034.html" class="nav-list-link">Single Cell TBR1 Misc Figures - ChromVAR Compare, FDR Hists, 3D</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-07-26-post-0035.html" class="nav-list-link">Single Cell TBR1 BioCluster/Genotype/Line/Time DA with TSS > 2.7, Filtering Ambiguous Groups</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-07-29-post-0040.html" class="nav-list-link">Single Cell TBR1 Bigwig Tracks</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-08-12-post-0045.html" class="nav-list-link">Single Cell TBR1 Cistopic Boxplots and UMAPs</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-08-12-post-0046.html" class="nav-list-link">Single Cell TBR1 ChIP-Seq Peak DA</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-11-28-post-0065.html" class="nav-list-link">Single Cell TBR1 Pseudo-Bulk Depth Tests</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2022-05-18-post-0087.html" class="nav-list-link">SRA Download Info for 'TBR1 Regulates Autism Risk Genes in the Developing Neocortex'</a></li></ul></li><li class="nav-list-item"><a href="#" class="nav-list-expander" aria-label="toggle links in archive_ssc category"> <svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg> </a><a href="/docs/archive_ssc/" class="nav-list-link">archive_ssc</a><ul class="nav-list "><li class="nav-list-item "><a href="/docs/archive_ssc/2020-10-09-post-0002.html" class="nav-list-link">SSC Final Validation Tables with Annotations</a></li></ul></li><li class="nav-list-item active"><a href="#" class="nav-list-expander" aria-label="toggle links in archive_spark category"> <svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg> </a><a href="/docs/archive_spark/" class="nav-list-link">archive_spark</a><ul class="nav-list "><li class="nav-list-item "><a href="/docs/archive_spark/2021-05-11-post-0012.html" class="nav-list-link">21-05-11, SPARK Mosaic List, Summary Counts of Annotations</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-05-28-post-0015.html" class="nav-list-link">21-05-28, SPARK RefSeq CDS and Splice Site Reference Generation</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-05-30-post-0016.html" class="nav-list-link">21-05-30, SPARK Joint Coverage Counter Tool Fidelity</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-05-30-post-0017.html" class="nav-list-link">21-05-30, SPARK Joint Coverage Plots</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-06-02-post-0018.html" class="nav-list-link">21-06-02, SPARK Joint Coverage Table Fix</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-06-03-post-0019.html" class="nav-list-link">21-06-03, SPARK Mosaic List FRACTION_MORE Annotations</a></li><li class="nav-list-item active"><a href="/docs/archive_spark/2021-06-03-post-0020.html" class="nav-list-link active">21-06-03, SPARK Mosaic Index</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-06-10-post-0021.html" class="nav-list-link">21-06-10, SPARK Gene Set Joint Coverage</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-06-22-post-0023.html" class="nav-list-link">21-06-22, SPARK Preliminary Pass 01, Filtering and Classifications</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-06-22-post-0024.html" class="nav-list-link">21-06-22, SPARK Preliminary Pass 02, Loading for Coverage and SNV Count Plots</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-06-26-post-0025.html" class="nav-list-link">21-06-26, SPARK Preliminary Pass 02b, Coverage and SNV Count Plots as Hists</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-06-28-post-0026.html" class="nav-list-link">21-06-28, SPARK Preliminary Pass 03, Allele Fraction Histograms</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-07-06-post-0027.html" class="nav-list-link">21-07-06, SPARK Preliminary Pass 04, Coverages and SNV Counts with Joint Family Coverages</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-07-14-post-0030.html" class="nav-list-link">21-07-14, SPARK Preliminary Pass 5, Final Filtering</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-07-17-post-0031.html" class="nav-list-link">21-07-17, SPARK Preliminary Pass 6a and 6b, Simplex Trio Coverages, SNV Counts, and AFs</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-07-17-post-0032.html" class="nav-list-link">21-07-17, SPARK Preliminary Pass 7a and 7b, Multiplex Quad Coverages, SNV Counts, and AFs</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-07-26-post-0036.html" class="nav-list-link">21-07-26, SPARK First Pass, Synonymous SNV Splice Distances</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-07-26-post-0037.html" class="nav-list-link">21-07-26, SPARK Preliminary Outlier Detection, Secondary Pass</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-07-26-post-0038.html" class="nav-list-link">21-07-26, SPARK Final Table Creation</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-07-27-post-0039.html" class="nav-list-link">21-07-27, SPARK Mosaic Index, July 27</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-08-03-post-0041.html" class="nav-list-link">21-08-03, SPARK High Confidence Allele Frequencies 01</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-08-03-post-0042.html" class="nav-list-link">21-08-03, SPARK High Confidence Synonymous Splice Distances 01</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-08-03-post-0043.html" class="nav-list-link">21-08-03, SPARK Burden Tables Autosomal 01</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-08-09-post-0044.html" class="nav-list-link">21-08-09, SPARK Burden Tables Gene Sets 01</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-08-26-post-0047.html" class="nav-list-link">21-08-26, SPARK IGV Shots of High Confident Calls</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-09-08-post-0048.html" class="nav-list-link">21-09-08, SPARK Burden Tables Gene Sets 02</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-09-14-post-0049.html" class="nav-list-link">21-09-14, SPARK Burden Tables Gene Sets 03</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-09-15-post-0050.html" class="nav-list-link">21-09-15, SPARK Concatenated High Confidence Table Checks 01</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-01-post-0051.html" class="nav-list-link">21-10-1, SPARK Final Sample List</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-01-post-0052.html" class="nav-list-link">21-10-01, SPARK Final IGV and Cohort Count Filter</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-06-post-0053.html" class="nav-list-link">21-10-06, SPARK LGD and NS List</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-06-post-0054.html" class="nav-list-link">21-10-06, SPARK Burden Tables Gene Sets 04</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-10-post-0056.html" class="nav-list-link">21-10-10, SPARK Summary of Final Lists</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-10-post-0057.html" class="nav-list-link">21-10-10, SPARK Final List Allele Frequency Distributions</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-11-post-0058.html" class="nav-list-link">21-10-11, SPARK Monozygotic Twins in Quad Families</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-11-post-0059.html" class="nav-list-link">21-10-11, SPARK Burden Tables Gene Sets, Final 01</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-13-post-0060.html" class="nav-list-link">21-10-13, SPARK Burden w/ REVEL Scores Gene Sets, 01</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-13-post-0061.html" class="nav-list-link">21-10-13, SPARK Parental Trans Mosaics Breakdown</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-13-post-0062.html" class="nav-list-link">21-10-13, SPARK HC Tables Against SPARK 167</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-14-post-0063.html" class="nav-list-link">21-10-14, SPARK Concat Variant Lists to DNV List Comparison</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-11-30-post-0066.html" class="nav-list-link">21-11-30, SPARK SSC List Formatting to Current SPARK</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-11-30-post-0067.html" class="nav-list-link">21-11-30, SPARK Burden Tables Gene Sets, SPARK+SSC</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-12-03-post-0068.html" class="nav-list-link">21-12-03, SPARK VEP Annotations</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2022-01-18-post-0070.html" class="nav-list-link">22-01-18, SPARK SSC hg38 Transformed Burden, 01</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2022-01-19-post-0071.html" class="nav-list-link">22-01-19, SPARK Pre Best Practices Filter, Model Scores</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2022-03-19-post-0076.html" class="nav-list-link">22-03-19, SPARK WES2 and WES3 List Inconsistencies</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2022-04-08-post-0079.html" class="nav-list-link">22-04-08, SPARK WES2 and WES3 Outliers by Counts and Coverage</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2022-04-28-post-0082.html" class="nav-list-link">22-04-28, SPARK Mosaic Index</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2022-05-03-post-0083.html" class="nav-list-link">22-05-03, SPARK IGV HC List Screenshots, 2022</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2022-05-20-post-0088.html" class="nav-list-link">22-05-20, SPARK Post-Hoc Cohort, IGV, and iWES Filter Lists</a></li></ul></li><li class="nav-list-item"><a href="#" class="nav-list-expander" aria-label="toggle links in archive_amplicon category"> <svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg> </a><a href="/docs/archive_amplicon/" class="nav-list-link">archive_amplicon</a><ul class="nav-list "><li class="nav-list-item "><a href="/docs/archive_amplicon/2022-03-07-post-0075.html" class="nav-list-link">Shotgun Sequencing of Amplicon Sequence, 220305</a></li><li class="nav-list-item "><a href="/docs/archive_amplicon/2022-04-06-post-0078.html" class="nav-list-link">Shotgun Sequencing of Amplicon Sequence, 220406</a></li><li class="nav-list-item "><a href="/docs/archive_amplicon/2022-05-18-post-0085.html" class="nav-list-link">Example of HDR Processing via BWA Alignment</a></li></ul></li></ul> </nav> <footer class="site-footer"> This site uses <a href="https://github.com/just-the-docs/just-the-docs">Just the Docs</a>, a documentation theme for Jekyll. </footer> </div> <div class="main" id="top"> <div id="main-header" class="main-header"> <div class="search"> <div class="search-input-wrap"> <input type="text" id="search-input" class="search-input" tabindex="0" placeholder="Search ARSN Work Blog" aria-label="Search ARSN Work Blog" autocomplete="off"> <label for="search-input" class="search-label"><svg viewBox="0 0 24 24" class="search-icon"><use xlink:href="#svg-search"></use></svg></label> </div> <div id="search-results" class="search-results"></div> </div> <nav aria-label="Auxiliary" class="aux-nav"> <ul class="aux-nav-list"> <li class="aux-nav-list-item"> <a href="//github.com/just-the-docs/just-the-docs" class="site-button" > Just the Docs on GitHub </a> </li> </ul> </nav> </div> <div id="main-content-wrap" class="main-content-wrap"> <nav aria-label="Breadcrumb" class="breadcrumb-nav"> <ol class="breadcrumb-nav-list"> <li class="breadcrumb-nav-list-item"><a href="/docs/archive_spark/">archive_spark</a></li> <li class="breadcrumb-nav-list-item"><span>21-06-03, SPARK Mosaic Index</span></li> </ol> </nav> <div id="main-content" class="main-content" role="main"> <p><em>This serves as a separate accessible copy of the README.txt for the SPARK Mosaic project.</em></p> <p>Final Mosaic Lists (July 17th): <br /><a href="https://www.dropbox.com/s/iou03kyskzstkyh/quadsimplex.finaltable.txt?dl=0">Simplex Quads w/ Loose Filter DISCONTINUED BUT HERE FOR POSTERITY</a></p> <p>Mosaic Lists (June): <br /><a href="https://www.dropbox.com/s/tyczipwxkch4a1f/quadsimplex_mosaics.8reads_0.001popAF_4cohort_3alt_exonic.jun3.txt?dl=0">Simplex Quads w/ Loose Filter</a> <br /><a href="https://www.dropbox.com/s/407gqs049ukcdcf/quadmultiplex_mosaics.8reads_0.001popAF_4cohort_3alt_exonic.jun3.txt?dl=0">Multiplex Quads w/ Loose Filter</a> <br /><a href="https://www.dropbox.com/s/2rs72k69tkepcrz/triosimplex_mosaics.8reads_0.001popAF_4cohort_3alt_exonic.jun3.txt?dl=0">Simplex Trios w/ Loose Filter</a> <br /><a href="https://www.dropbox.com/s/okrg9g1ug10788i/quadsimplex_mosaics.8reads_0.001popAF_4cohort_3alt_exonic.testfilter.jun3.txt?dl=0">Simplex Quads w/ Strict Test Filter</a> <br /><a href="https://www.dropbox.com/s/xabrs2f030xlz49/quadmultiplex_mosaics.8reads_0.001popAF_4cohort_3alt_exonic.testfilter.jun3.txt?dl=0">Multiplex Quads w/ Strict Test Filter</a> <br /><a href="https://www.dropbox.com/s/dv4ajax8bqbwfcu/triosimplex_mosaics.8reads_0.001popAF_4cohort_3alt_exonic.testfilter.jun3.txt?dl=0">Simplex Trios w/ Strict Test Filter</a> <br /><a href="https://www.dropbox.com/s/fjuuw3q7cekci7f/quadsimplex_mosaics.8reads_0.001popAF_4cohort_3alt_exonic.testfilter_noGermlineParental.jun3.txt?dl=0">Simplex Quads w/ Strict Test Filter, no Germline Parental</a> <br /><a href="https://www.dropbox.com/s/vev0qhne7n0gmx4/quadmultiplex_mosaics.8reads_0.001popAF_4cohort_3alt_exonic.testfilter_noGermlineParental.jun3.txt?dl=0">Multiplex Quads w/ Strict Test Filter, no Germline Parental</a> <br /><a href="https://www.dropbox.com/s/vmhwb1a307p92gs/triosimplex_mosaics.8reads_0.001popAF_4cohort_3alt_exonic.testfilter_noGermlineParental.jun3.txt?dl=0">Simplex Trios w/ Strict Test Filter, no Germline Parental</a></p> <p>SPARK Annotations Used, 20190820: <br /><a href="https://www.dropbox.com/s/gg5ewe0bbqhjimw/SPARK.27K.mastertable.20190820.txt?dl=0">SPARK Annotations</a></p> <p>Links to List of Gene Names with Alternate Names: <br /><a href="https://www.dropbox.com/s/4jcjuaz8c3dhbtf/Essential.alt.genes?dl=0">Essential.alt.genes</a> <br /><a href="https://www.dropbox.com/s/jbrpq5fb8qz16ri/Exac.Intol.alt.genes?dl=0">Exac.Intol.alt.genes</a> <br /><a href="https://www.dropbox.com/s/yjw7r0fwh3lxqil/Exac.Mid.alt.genes?dl=0">Exac.Mid.alt.genes</a> <br /><a href="https://www.dropbox.com/s/443h1423awxmxza/Exac.Tol.alt.genes?dl=0">Exac.Tol.alt.genes</a> <br /><a href="https://www.dropbox.com/s/qbptliwg3irwnod/FUSIL.alt.genes?dl=0">FUSIL.alt.genes</a></p> <p>Joint Coverage: <br /><a href="https://www.dropbox.com/s/o3qeqthlx4xjykn/jointcoverage.simplexquads.txt?dl=0">Genomic Individal/Avg Coverage, Simplex Quads</a> <br /><a href="https://www.dropbox.com/s/xp96tughn8vhjzu/jointcoverage.2.simplexquads.txt?dl=0">Genomic Joint Coverage, Simplex Quads</a> <br /><a href="https://www.dropbox.com/s/o5zx9ljr7q2bn92/jointcoverage.multiplexquads.txt?dl=0">Genomic Individal/Avg Coverage, Multiplex Quads</a> <br /><a href="https://www.dropbox.com/s/dm5ppo3fn2q0vrv/jointcoverage.2.multiplexquads.txt?dl=0">Genomic Joint Coverage, Multiplex Quads</a> <br /><a href="https://www.dropbox.com/s/qu5nps1knpeuf34/jointcoverage.simplextrios.txt?dl=0">Genomic Individal/Avg Coverage, Simplex Trios</a> <br /><a href="https://www.dropbox.com/s/pfkw1v3z7rs4ugy/jointcoverage.2.simplextrios.txt?dl=0">Genomic Joint Coverage, Simplex Trios</a></p> <p>Joint Coverage by Gene-set for Quad Simplex: <br /><a href="https://www.dropbox.com/s/y25s6feb140f9zb/jointcoverage.simplexquads_essential.txt?dl=0">jointcoverage.Essential.txt</a> <br /><a href="https://www.dropbox.com/s/da0va0l5ss0vak5/jointcoverage.simplexquads_exintol.txt?dl=0">jointcoverage.ExacInt.txt</a> <br /><a href="https://www.dropbox.com/s/zbmbc5ueysvzm7j/jointcoverage.simplexquads_exmid.txt?dl=0">jointcoverage.ExacMid.txt</a> <br /><a href="https://www.dropbox.com/s/0sik284bf7yrk2l/jointcoverage.simplexquads_extol.txt?dl=0">jointcoverage.ExacTol.txt</a> <br /><a href="https://www.dropbox.com/s/p430qmv0jsmhxkk/jointcoverage.simplexquads_fusil.txt?dl=0">jointcoverage.FUSIL.txt</a></p> <p>Joint Coverage by Gene-set for Quad Multiplex: <br /><a href="https://www.dropbox.com/s/t3rojxivrev0r9y/jointcoverage.multiplexquads_essential.txt?dl=0">jointcoverage.Essential.txt</a> <br /><a href="https://www.dropbox.com/s/8bjecr0y5nawjca/jointcoverage.multiplexquads_exintol.txt?dl=0">jointcoverage.ExacInt.txt</a> <br /><a href="https://www.dropbox.com/s/ramv1qr8a1hoty0/jointcoverage.multiplexquads_exmid.txt?dl=0">jointcoverage.ExacMid.txt</a> <br /><a href="https://www.dropbox.com/s/yqd7pzbxpk5c2ds/jointcoverage.multiplexquads_extol.txt?dl=0">jointcoverage.ExacTol.txt</a> <br /><a href="https://www.dropbox.com/s/6fpa9ocjp0x4nwi/jointcoverage.multiplexquads_fusil.txt?dl=0">jointcoverage.FUSIL.txt</a></p> <p>Joint Coverage by Gene-set for Trio Simplex: <br /><a href="https://www.dropbox.com/s/1gw1w6w4fwrn4o4/jointcoverage.simplextrios_essential.txt?dl=0">jointcoverage.Essential.txt</a> <br /><a href="https://www.dropbox.com/s/a30p1fe5r99vkf0/jointcoverage.simplextrios_exintol.txt?dl=0">jointcoverage.ExacInt.txt</a> <br /><a href="https://www.dropbox.com/s/xn1t88bl1r9028d/jointcoverage.simplextrios_exmid.txt?dl=0">jointcoverage.ExacMid.txt</a> <br /><a href="https://www.dropbox.com/s/7gzbgk0pryo7eoi/jointcoverage.simplextrios_extol.txt?dl=0">jointcoverage.ExacTol.txt</a> <br /><a href="https://www.dropbox.com/s/se7qcaaulzihpec/jointcoverage.simplextrios_fusil.txt?dl=0">jointcoverage.FUSIL.txt</a></p> <p>Markdowns: <br /><a href="https://www.dropbox.com/s/q758yakdp5bm6de/prelim_burden_quadsimplex_01.html?dl=0">Preliminary, Loading, Filtering, and Subsetting, Simplex Quad</a> <br /><a href="https://www.dropbox.com/s/nco21ui46vgv0xt/prelim_burden_quadsimplex_02.html?dl=0">Preliminary, Plotting Genomic Coverages and SNV Counts, Simplex Quad</a> <br /><a href="https://www.dropbox.com/s/w09g46khn7xazng/prelim_burden_quadsimplex_02b.html?dl=0">Preliminary, Plotting SNV Counts as Hists, Simplex Quad</a> <br /><a href="https://www.dropbox.com/s/29z48y39rflny96/prelim_burden_quadsimplex_03.html?dl=0">Preliminary, Plotting Allele Frequencies, Simplex Quad</a> <br /><a href="https://www.dropbox.com/s/3ne6r98f4o1n3hi/prelim_burden_quadsimplex_04a.html?dl=0">Preliminary, Updated Loading, Filtering, and Subsetting, Simplex Quad</a> <br /><a href="https://www.dropbox.com/s/nytxd9qmoq882se/prelim_burden_quadsimplex_04b.html?dl=0">Preliminary, Updated Genomic Coverages and SNV Counts, Simplex Quad</a> <br /><a href="https://www.dropbox.com/s/se5uhqkcxmqocmx/prelim_burden_quadsimplex_05.html?dl=0">Preliminary, Final Table Creation, Simplex Quad</a> <br /><a href="https://www.dropbox.com/s/n73w7pyxewrzl2l/prelim_burden_triosimplex_06a.html?dl=0">Preliminary, Loading, Filtering, and Subsetting, Simplex Trio</a> <br /><a href="https://www.dropbox.com/s/jls73uleobac7o5/prelim_burden_triosimplex_06b.html?dl=0">Preliminary, Genomic Coverages, SNV Counts, and Allele Frequencies, Simplex Trio</a> <br /><a href="https://www.dropbox.com/s/1wvlhs0bzrw2pwn/prelim_burden_quadmultiplex_07a.html?dl=0">Preliminary, Loading, Filtering, and Subsetting, Multiplex Quad</a> <br /><a href="https://www.dropbox.com/s/fvf07wuyqisbiyw/prelim_burden_quadmultiplex_07b.html?dl=0">Preliminary, Genomic Coverages, SNV Counts, and Allele Frequencies, Multiplex Quad</a> <br /><a href="https://www.dropbox.com/s/wu9t8w7lnv6uw7c/compare_burden_counter.html?dl=0">Testing, Comparison of Burden Counters</a> <br /><a href="https://www.dropbox.com/s/bon4m7zfee6yagp/summary_fractions.html?dl=0">Testing, Summary of FRACTION Annotations</a> <br /><a href="https://www.dropbox.com/s/xevia29rbrw8uab/burden_coverage_plots.html?dl=0">Testing, Genomic Burden Coverage Plots</a> <br /><a href="https://www.dropbox.com/s/8vwd6t3ap2g8t0n/firstpass_simplexquads_splicedist_08.html?dl=0">Simplex Quad Synonymous SNV Splice Distances</a> <br /><a href="https://www.dropbox.com/s/p2rghqq58an52jf/prelim_burden_quadsimplex_09a.html?dl=0">Outlier Detection Prep, Simplex Quads</a> <br /><a href="https://www.dropbox.com/s/bz3ovlqfk1l2t5j/prelim_burden_quadsimplex_09b.html?dl=0">Outlier Detection Figures, Simplex Quads</a> <br /><a href="https://www.dropbox.com/s/tkptvw9v740dnn1/prelim_burden_triosimplex_10a.html?dl=0">Outlier Detection Prep, Simplex Trios</a> <br /><a href="https://www.dropbox.com/s/hscrmzueyqk0h5p/prelim_burden_triosimplex_10b.html?dl=0">Outlier Detection Figures, Simplex Trios</a> <br /><a href="https://www.dropbox.com/s/pvke3s22tkxojzq/prelim_burden_quadmultiplex_11a.html?dl=0">Outlier Detection Prep, Multiplex Quads</a> <br /><a href="https://www.dropbox.com/s/x1tcpf9beovlk69/prelim_burden_quadmultiplex_11b.html?dl=0">Outlier Detection Figures, Multiplex Quads</a></p> <p>The data is first filtered with loose criteria that will not be changed: there must be 8 reads at the location for the samples, the population allele frequencies must be less than 0.001, the variant is called less than four times across the families in the list, it must have at least 3 variant reads, and it must have an exonic refseq annotation. The data is then filtered again with stricter criteria (testfilter). These criteria were developed for the older lists, might not be as appropriate for these lists, and may change eventually. These stricter criteria are: five variant reads, greater than 0.03 AF, called by at least two callers (mpup, lofreq, or varscan), has an upper 90% confidence intervals via Agresti-Coull method above or equal to 5%, MISMATCH_XM_ALT_MEDIAN &lt;= 3, and SCORE_400_RB3 &gt; 0.518. I then filtered it down to toss out the numerous germline parental calls that won’t be of use to us for looking at mosaics. The data here is unlikely to change but will certainly get filtered down further in the future.</p> <p>Currently, the three lists were made and looked at completely separate from each other. So columns like “NUM_FAMILY_FOUND” will only count up families where the variant is found within the one list and not across all three lists.</p> <p>All columns are the same across the three lists. The trio list has “NAs” for the columns referencing the ‘sib’ family member. For the multiplex list, know that Spark labels usually use the convention of ‘father’, ‘mother’, ‘proband’, and ‘sibling’ but a few families are labeled differently – either with multiple probands, multiple siblings, or “half_sibling” or other labels. In these cases, I continue to use the column names for PRO and SIB in my list even if a family has two samples labeled “SIB”. The exact sample in question will be in the Sample field.</p> <p>The categories (FRACTION column) we group into are: GERMLINE_CHILD, MOSAIC_CHILD, MOSAIC_PARENTAL_NONTRANS, MOSAIC_PARENTAL_TRANS, GERMLINE_CONFLICT, MOSAIC_CHILD_WITH_PARENTAL_CONFLICT, MOSAIC_PARENTAL_WITH_TRANS_CONFLICT, and MULTI_MOSAIC. The “MULTI_MOSAIC” and “CONFLICT” groups will probably be entirely ignored going forward. They indicate that a putative mosaic was found but the variant’s presence or absence in the other members of the family is not resolved or conflating.</p> <p>“SCORE_400_RB3” is probably our best column for the time being to use for sorting mosaics by which have the best support.</p> <p>Here are descriptions of all the columns in the table: <br />1/CHR, 2/START, 3/STOP, 4/REF, 5/ALT - general variant information <br />6/FAMILY, 7/SAMPLE, 8/FRACTION - SF family ID, SP sample ID, putative mosaic annotation <br />9/DP_MAIN, 10/DPALT_MAIN, 11/AF_MAIN - the depth of reads, depth of alt reads, and AF for the main person of the family <br />12/LOFREQ_STATS_NAMES, 13/LOFREQ_STATS, 14/MPUP_STATS_NAMES, 15/MPUP_STATS, 16/VARSCAN_STATS_NAMES, 17/VARSCAN_STATS - stats from the three variant callers with their descriptor fields <br />18/MISMATCH_REF_READS, 19/MISMATCH_ALT_READS, 20/MISMATCH_OTHER_READS, 21/MISMATCH_SPAN_DELS, 22/MISMATCH_MAPQ_REF_MEAN, 23/MISMATCH_MAPQ_ALT_MEAN, 24/MISMATCH_MAPQ_OTHER_MEAN, 25/MISMATCH_MAPQ_REF_MEDIAN, 26/MISMATCH_MAPQ_ALT_MEDIAN, 27/MISMATCH_MAPQ_OTHER_MEDIAN, 28/MISMATCH_MAPQ_REF_MODE, 29/MISMATCH_MAPQ_ALT_MODE, 30/MISMATCH_MAPQ_OTHER_MODE, 31/MISMATCH_XM_REF_MEAN, 32/MISMATCH_XM_ALT_MEAN, 33/MISMATCH_XM_OTHER_MEAN, 34/MISMATCH_XM_REF_MEDIAN, 35/MISMATCH_XM_ALT_MEDIAN, 36/MISMATCH_XM_OTHER_MEDIAN, 37/MISMATCH_XM_REF_MODE, 38/MISMATCH_XM_ALT_MODE, 39/MISMATCH_XM_OTHER_MODE - large collection of stats concerning mismatches across the region, only MISMATCH_XM_ALT_MEDIAN gets used downstream for another derived column <br />40/DP_FA, 41/DP_MO, 42/DP_PRO, 43/DP_SIB, 44/DPALT_FA, 45/DPALT_MO, 46/DPALT_PRO, 47/DPALT_SIB - depths across all family members <br />48/AF_EXAC, 49/AF_gnomAD - AFs from ExAC v0.3 and gnomAD 2.1 <br />50/ERROR_VAR, 51/ERROR_ALL, 52/ERROR_VARMAX, 53/ERROR_RATE - error in the region calculated over a random subset of 1000 samples, only rate gets used downstream for another derived column <br />54/NUM_FAMILY_FOUND - number of families the variant is found before filtering <br />55/PHET_FA, 56/PHET_MO, 57/PHET_PRO, 58/PHET_SIB - pvalue of exact test given the depth and the depth of the alt read in each family member testing if it is less than 0.5 (heterogeneous) <br />59/PERSON - identity of the person <br />60/DPALT_BIN - binary column derived from DPALT_MAIN used for SCORE_400_RB3 <br />61/ERROR_BIN_3 - categorical column (low, medium, high) derived from ERROR_RATE used for SCORE_400_RB3 <br />62/AF_BIN_3 - categorical column (low, medium, high) derived from AF_MAIN used for SCORE_400_RB3 <br />63/REF_CG - categorical column (C or G) derived from REF used for SCORE_400_RB3 <br />64/XM_ALT_1 - binary column derived from MISMATCH_XM_ALT_MEDIAN used for SCORE_400_RB3 <br />65/SCORE_400_RB3 - score from logistic regression model that takes into account many of the other columns <br />66/genomicSuperDupes, 67/rmsk, 68/simpleRepeat, 69/mutationType, 70/exonicAnnotation, 71/variantFunctionType, 72/variantFunctionGene - annotations with annovar-2018-06-15 <br />73/filteredCohortCount - number of families the variant is found after preliminary filtering <br />74/AF_L90CI_AC_MAIN, 75/AF_U90CI_AC_MAIN - upper and lower 90% confidence intervals via Agresti-Coull method <br />76/FRACTION_ORIGINAL - old, erroneous annotations of the site for posterity, see May 9 update <br />77/FRACTION_MORE - a subset of column 8, FRACTION, with further granularity, see June 3 update <br />78/PSAME_FAxPRO, 79/PSAME_FAxSIB, 80/PSAME_MOxPRO, 81/PSAME_MOxSIB - Fisher’s exact test comparing depths and alt reads from parents versus children for the purposes of testing a transmission event. Final tables also contain: <br />82/MIN_DP - The minimum depth of all family members. <br />83/ACC_DIST, 84/DON_DIST, 85/ACC_TRANSCRIPT, 86/DON_TRANSCRIPT - The distances in basepairs to an acceptor or donor site and which transcript from which this distance is derived. <br />87/COMPLEX_EVENT - All FALSE in final tables representing collapsed complex events.</p> <p>###</p> <p>*Update for July 17, 2021</p> <p>Tables are modified for July 17. Final tables have been added with final filters applied and collapsing of duplicate entries and complex events. Five new columns have been added. One for minimum depth, four for the donor and acceptor distances and transcripts, and one for whether or not the row represents a complex event.</p> <p>###</p> <p>*Update for June 3, 2021</p> <p>Tables are modified for June 3. I added five new columns. The PSAME columns and the FRACTION_MORE column. FRACTION_MORE is derived <a href="https://arsnishida.github.io/2021/06/03/post-0019.html">here</a>.</p> <p>###</p> <p>*Update for May 9, 2021</p> <p>Tables are modified for May 9. I updated the “FRACTION” annotation column adjusting an error in the later files. There is one new column at the end called, “FRACTION_ORIGINAL”. This contains the first call of the fraction that should be ignored but I’m keeping it in the file as a record.</p> </div> </div> <div class="search-overlay"></div> </div> </body> </html>
