<!DOCTYPE html> <html lang="en-US"> <head> <meta charset="UTF-8"> <meta http-equiv="X-UA-Compatible" content="IE=Edge"> <title>21-07-27, SPARK Mosaic Index, July 27 - ARSN Work Blog</title> <link rel="shortcut icon" href="/favicon.ico" type="image/x-icon"> <link rel="stylesheet" href="/assets/css/just-the-docs-default.css"> <script src="/assets/js/vendor/lunr.min.js"></script> <script src="/assets/js/just-the-docs.js"></script> <meta name="viewport" content="width=device-width, initial-scale=1"> <!-- Begin Jekyll SEO tag v2.8.0 --> <title>21-07-27, SPARK Mosaic Index, July 27 | ARSN Work Blog</title> <meta name="generator" content="Jekyll v4.3.0" /> <meta property="og:title" content="21-07-27, SPARK Mosaic Index, July 27" /> <meta property="og:locale" content="en_US" /> <meta name="description" content="This is a blog for work projects Andrew Nishida, nishidaa@ohsu.edu, in CEDAR at OHSU." /> <meta property="og:description" content="This is a blog for work projects Andrew Nishida, nishidaa@ohsu.edu, in CEDAR at OHSU." /> <link rel="canonical" href="http://localhost:4000/docs/archive_spark/2021-07-27-post-0039.html" /> <meta property="og:url" content="http://localhost:4000/docs/archive_spark/2021-07-27-post-0039.html" /> <meta property="og:site_name" content="ARSN Work Blog" /> <meta property="og:type" content="article" /> <meta property="article:published_time" content="2021-07-27T00:00:00-07:00" /> <meta name="twitter:card" content="summary" /> <meta property="twitter:title" content="21-07-27, SPARK Mosaic Index, July 27" /> <script type="application/ld+json"> {"@context":"https://schema.org","@type":"BlogPosting","dateModified":"2021-07-27T00:00:00-07:00","datePublished":"2021-07-27T00:00:00-07:00","description":"This is a blog for work projects Andrew Nishida, nishidaa@ohsu.edu, in CEDAR at OHSU.","headline":"21-07-27, SPARK Mosaic Index, July 27","mainEntityOfPage":{"@type":"WebPage","@id":"http://localhost:4000/docs/archive_spark/2021-07-27-post-0039.html"},"url":"http://localhost:4000/docs/archive_spark/2021-07-27-post-0039.html"}</script> <!-- End Jekyll SEO tag --> </head> <body> <a class="skip-to-main" href="#main-content">Skip to main content</a> <svg xmlns="http://www.w3.org/2000/svg" style="display: none;"> <symbol id="svg-link" viewBox="0 0 24 24"> <title>Link</title> <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-link"> <path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path> </svg> </symbol> <symbol id="svg-search" viewBox="0 0 24 24"> <title>Search</title> <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-search"> <circle cx="11" cy="11" r="8"></circle><line x1="21" y1="21" x2="16.65" y2="16.65"></line> </svg> </symbol> <symbol id="svg-menu" viewBox="0 0 24 24"> <title>Menu</title> <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-menu"> <line x1="3" y1="12" x2="21" y2="12"></line><line x1="3" y1="6" x2="21" y2="6"></line><line x1="3" y1="18" x2="21" y2="18"></line> </svg> </symbol> <symbol id="svg-arrow-right" viewBox="0 0 24 24"> <title>Expand</title> <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-chevron-right"> <polyline points="9 18 15 12 9 6"></polyline> </svg> </symbol> <symbol id="svg-doc" viewBox="0 0 24 24"> <title>Document</title> <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-file"> <path d="M13 2H6a2 2 0 0 0-2 2v16a2 2 0 0 0 2 2h12a2 2 0 0 0 2-2V9z"></path><polyline points="13 2 13 9 20 9"></polyline> </svg> </symbol> <!-- Feather. MIT License: https://github.com/feathericons/feather/blob/master/LICENSE --> <symbol id="svg-external-link" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-external-link"> <title id="svg-external-link-title">(external link)</title> <path d="M18 13v6a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V8a2 2 0 0 1 2-2h6"></path><polyline points="15 3 21 3 21 9"></polyline><line x1="10" y1="14" x2="21" y2="3"></line> </symbol> </svg> <div class="side-bar"> <div class="site-header"> <a href="/" class="site-title lh-tight"> ARSN Work Blog </a> <a href="#" id="menu-button" class="site-button"> <svg viewBox="0 0 24 24" class="icon"><use xlink:href="#svg-menu"></use></svg> </a> </div> <nav aria-label="Main" id="site-nav" class="site-nav"> <ul class="nav-list"><li class="nav-list-item"><a href="/" class="nav-list-link">Home</a></li><li class="nav-list-item"><a href="#" class="nav-list-expander" aria-label="toggle links in sc_breastmultiome category"> <svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg> </a><a href="/docs/sc_breastmultiome/" class="nav-list-link">sc_breastmultiome</a><ul class="nav-list "><li class="nav-list-item "><a href="/docs/sc_breastmultiome/2022-12-12-sc_breastmultiome-epiconfig.html" class="nav-list-link">22-12-12, Epiconfig Runs on Full and Subset Data</a></li><li class="nav-list-item "><a href="/docs/sc_breastmultiome/2022-12-12-sc_breastmultiome-project_info.html" class="nav-list-link">22-12-12, Project Info</a></li><li class="nav-list-item "><a href="/docs/sc_breastmultiome/2023-01-12-sc_breastmultiome-epiconfig-downsample.html" class="nav-list-link">23-01-12, Epiconfig Run on Downampled Epithelial</a></li><li class="nav-list-item "><a href="/docs/sc_breastmultiome/2023-01-17-sc_breastmultiome-epiconfig-pam50umaps.html" class="nav-list-link">23-01-17, PAM50 UMAPs</a></li><li class="nav-list-item "><a href="/docs/sc_breastmultiome/2023-01-24-sc_breastmultiome-epiconfig-norm_luma.html" class="nav-list-link">23-01-24, Normal and LumA Subsets</a></li><li class="nav-list-item "><a href="/docs/sc_breastmultiome/2023-01-30-sc_breastmultiome-epiconfig-cnvnormal.html" class="nav-list-link">23-01-30, CNV Normal Calls</a></li></ul></li><li class="nav-list-item"><a href="#" class="nav-list-expander" aria-label="toggle links in ravnica category"> <svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg> </a><a href="/docs/ravnica/" class="nav-list-link">ravnica</a><ul class="nav-list "><li class="nav-list-item "><a href="/docs/ravnica/2022-01-25-post-0073.html" class="nav-list-link">Dropbox Upload and Download with Curl and API Key</a></li><li class="nav-list-item "><a href="/docs/ravnica/2022-12-05-ravnica-global_install_initial.html" class="nav-list-link">22-12-05, Global Installs - Initial</a></li><li class="nav-list-item "><a href="/docs/ravnica/2022-12-05-ravnica-local_install_initial.html" class="nav-list-link">22-12-05, Local Installs - Initial</a></li><li class="nav-list-item "><a href="/docs/ravnica/2022-12-05-ravnica-mamba_install_initial.html" class="nav-list-link">22-12-05, Mamba Install - Initial</a></li><li class="nav-list-item "><a href="/docs/ravnica/2022-12-05-ravnica-mamba_setup.html" class="nav-list-link">22-12-05, Mamba Setup</a></li><li class="nav-list-item "><a href="/docs/ravnica/2022-12-05-ravnica-r_4.2.2_library_info.html" class="nav-list-link">22-12-05, R 4.2.2 Library Info</a></li><li class="nav-list-item "><a href="/docs/ravnica/2022-12-06-ravnica-example_bashprofile.html" class="nav-list-link">22-12-06, Example Bash Profile</a></li><li class="nav-list-item "><a href="/docs/ravnica/2022-12-06-ravnica-venv_macs3.html" class="nav-list-link">22-12-06, VENV for Macs3</a></li><li class="nav-list-item "><a href="/docs/ravnica/2022-05-18-post-0086.html" class="nav-list-link">Cron Info for Automatically Running bcl2fastq</a></li></ul></li><li class="nav-list-item"><a href="#" class="nav-list-expander" aria-label="toggle links in scitools category"> <svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg> </a><a href="/docs/scitools/" class="nav-list-link">scitools</a><ul class="nav-list "><li class="nav-list-item "><a href="/docs/scitools/2020-12-17-post-0005.html" class="nav-list-link">Scitools scATAC Metrics</a></li><li class="nav-list-item "><a href="/docs/scitools/2021-02-18-post-0006.html" class="nav-list-link">Scitools Scrublet Function</a></li><li class="nav-list-item "><a href="/docs/scitools/2021-05-08-post-0009.html" class="nav-list-link">Single Cell Alignments with no Alt Chromosomes</a></li><li class="nav-list-item "><a href="/docs/scitools/2022-04-03-post-0077.html" class="nav-list-link">scRNA-seq First Pass Pipeline - 220324_HB_Coassay_RNA</a></li><li class="nav-list-item "><a href="/docs/scitools/2022-04-11-post-0080.html" class="nav-list-link">scRNA-seq Second Pass Pipeline - 220324_HB_Coassay_RNA</a></li></ul></li><li class="nav-list-item"><a href="#" class="nav-list-expander" aria-label="toggle links in sc_dm6tau category"> <svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg> </a><a href="/docs/sc_dm6tau/" class="nav-list-link">sc_dm6tau</a><ul class="nav-list "><li class="nav-list-item "><a href="/docs/sc_dm6tau/2021-10-07-post-0055.html" class="nav-list-link">Drosophilia Tau Preliminary DA</a></li><li class="nav-list-item "><a href="/docs/sc_dm6tau/2022-01-08-post-0069.html" class="nav-list-link">Drosophilia Tau Final DA</a></li><li class="nav-list-item "><a href="/docs/sc_dm6tau/2022-01-21-post-0072.html" class="nav-list-link">Drosophilia Tau DA Results Test Analysis</a></li></ul></li><li class="nav-list-item"><a href="#" class="nav-list-expander" aria-label="toggle links in sc_klf4bc category"> <svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg> </a><a href="/docs/sc_klf4bc/" class="nav-list-link">sc_klf4bc</a><ul class="nav-list "><li class="nav-list-item "><a href="/docs/sc_klf4bc/2022-12-12-sc_breastmultiome-epiconfig_filterset.html" class="nav-list-link">22-12-13, KLF4 BC Initial Processing</a></li></ul></li><li class="nav-list-item active"><a href="#" class="nav-list-expander" aria-label="toggle links in archive_spark category"> <svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg> </a><a href="/docs/archive_spark/" class="nav-list-link">archive_spark</a><ul class="nav-list "><li class="nav-list-item "><a href="/docs/archive_spark/2021-05-11-post-0012.html" class="nav-list-link">21-05-11, SPARK Mosaic List, Summary Counts of Annotations</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-05-28-post-0015.html" class="nav-list-link">21-05-28, SPARK RefSeq CDS and Splice Site Reference Generation</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-05-30-post-0016.html" class="nav-list-link">21-05-30, SPARK Joint Coverage Counter Tool Fidelity</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-05-30-post-0017.html" class="nav-list-link">21-05-30, SPARK Joint Coverage Plots</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-06-02-post-0018.html" class="nav-list-link">21-06-02, SPARK Joint Coverage Table Fix</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-06-03-post-0019.html" class="nav-list-link">21-06-03, SPARK Mosaic List FRACTION_MORE Annotations</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-06-03-post-0020.html" class="nav-list-link">21-06-03, SPARK Mosaic Index</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-06-10-post-0021.html" class="nav-list-link">21-06-10, SPARK Gene Set Joint Coverage</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-06-22-post-0023.html" class="nav-list-link">21-06-22, SPARK Preliminary Pass 01, Filtering and Classifications</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-06-22-post-0024.html" class="nav-list-link">21-06-22, SPARK Preliminary Pass 02, Loading for Coverage and SNV Count Plots</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-06-26-post-0025.html" class="nav-list-link">21-06-26, SPARK Preliminary Pass 02b, Coverage and SNV Count Plots as Hists</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-06-28-post-0026.html" class="nav-list-link">21-06-28, SPARK Preliminary Pass 03, Allele Fraction Histograms</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-07-06-post-0027.html" class="nav-list-link">21-07-06, SPARK Preliminary Pass 04, Coverages and SNV Counts with Joint Family Coverages</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-07-14-post-0030.html" class="nav-list-link">21-07-14, SPARK Preliminary Pass 5, Final Filtering</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-07-17-post-0031.html" class="nav-list-link">21-07-17, SPARK Preliminary Pass 6a and 6b, Simplex Trio Coverages, SNV Counts, and AFs</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-07-17-post-0032.html" class="nav-list-link">21-07-17, SPARK Preliminary Pass 7a and 7b, Multiplex Quad Coverages, SNV Counts, and AFs</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-07-26-post-0036.html" class="nav-list-link">21-07-26, SPARK First Pass, Synonymous SNV Splice Distances</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-07-26-post-0037.html" class="nav-list-link">21-07-26, SPARK Preliminary Outlier Detection, Secondary Pass</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-07-26-post-0038.html" class="nav-list-link">21-07-26, SPARK Final Table Creation</a></li><li class="nav-list-item active"><a href="/docs/archive_spark/2021-07-27-post-0039.html" class="nav-list-link active">21-07-27, SPARK Mosaic Index, July 27</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-08-03-post-0041.html" class="nav-list-link">21-08-03, SPARK High Confidence Allele Frequencies 01</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-08-03-post-0042.html" class="nav-list-link">21-08-03, SPARK High Confidence Synonymous Splice Distances 01</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-08-03-post-0043.html" class="nav-list-link">21-08-03, SPARK Burden Tables Autosomal 01</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-08-09-post-0044.html" class="nav-list-link">21-08-09, SPARK Burden Tables Gene Sets 01</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-08-26-post-0047.html" class="nav-list-link">21-08-26, SPARK IGV Shots of High Confident Calls</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-09-08-post-0048.html" class="nav-list-link">21-09-08, SPARK Burden Tables Gene Sets 02</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-09-14-post-0049.html" class="nav-list-link">21-09-14, SPARK Burden Tables Gene Sets 03</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-09-15-post-0050.html" class="nav-list-link">21-09-15, SPARK Concatenated High Confidence Table Checks 01</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-01-post-0051.html" class="nav-list-link">21-10-1, SPARK Final Sample List</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-01-post-0052.html" class="nav-list-link">21-10-01, SPARK Final IGV and Cohort Count Filter</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-06-post-0053.html" class="nav-list-link">21-10-06, SPARK LGD and NS List</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-06-post-0054.html" class="nav-list-link">21-10-06, SPARK Burden Tables Gene Sets 04</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-10-post-0056.html" class="nav-list-link">21-10-10, SPARK Summary of Final Lists</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-10-post-0057.html" class="nav-list-link">21-10-10, SPARK Final List Allele Frequency Distributions</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-11-post-0058.html" class="nav-list-link">21-10-11, SPARK Monozygotic Twins in Quad Families</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-11-post-0059.html" class="nav-list-link">21-10-11, SPARK Burden Tables Gene Sets, Final 01</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-13-post-0060.html" class="nav-list-link">21-10-13, SPARK Burden w/ REVEL Scores Gene Sets, 01</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-13-post-0061.html" class="nav-list-link">21-10-13, SPARK Parental Trans Mosaics Breakdown</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-13-post-0062.html" class="nav-list-link">21-10-13, SPARK HC Tables Against SPARK 167</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-14-post-0063.html" class="nav-list-link">21-10-14, SPARK Concat Variant Lists to DNV List Comparison</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-11-30-post-0066.html" class="nav-list-link">21-11-30, SPARK SSC List Formatting to Current SPARK</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-11-30-post-0067.html" class="nav-list-link">21-11-30, SPARK Burden Tables Gene Sets, SPARK+SSC</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-12-03-post-0068.html" class="nav-list-link">21-12-03, SPARK VEP Annotations</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2022-01-18-post-0070.html" class="nav-list-link">22-01-18, SPARK SSC hg38 Transformed Burden, 01</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2022-01-19-post-0071.html" class="nav-list-link">22-01-19, SPARK Pre Best Practices Filter, Model Scores</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2022-03-19-post-0076.html" class="nav-list-link">22-03-19, SPARK WES2 and WES3 List Inconsistencies</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2022-04-08-post-0079.html" class="nav-list-link">22-04-08, SPARK WES2 and WES3 Outliers by Counts and Coverage</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2022-04-28-post-0082.html" class="nav-list-link">22-04-28, SPARK Mosaic Index</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2022-05-03-post-0083.html" class="nav-list-link">22-05-03, SPARK IGV HC List Screenshots, 2022</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2022-05-20-post-0088.html" class="nav-list-link">22-05-20, SPARK Post-Hoc Cohort, IGV, and iWES Filter Lists</a></li></ul></li><li class="nav-list-item"><a href="#" class="nav-list-expander" aria-label="toggle links in archive_amplicon category"> <svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg> </a><a href="/docs/archive_amplicon/" class="nav-list-link">archive_amplicon</a><ul class="nav-list "><li class="nav-list-item "><a href="/docs/archive_amplicon/2022-03-07-post-0075.html" class="nav-list-link">Shotgun Sequencing of Amplicon Sequence, 220305</a></li><li class="nav-list-item "><a href="/docs/archive_amplicon/2022-04-06-post-0078.html" class="nav-list-link">Shotgun Sequencing of Amplicon Sequence, 220406</a></li><li class="nav-list-item "><a href="/docs/archive_amplicon/2022-05-18-post-0085.html" class="nav-list-link">Example of HDR Processing via BWA Alignment</a></li></ul></li><li class="nav-list-item"><a href="#" class="nav-list-expander" aria-label="toggle links in archive_sc_tbr1 category"> <svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg> </a><a href="/docs/archive_sc_tbr1/" class="nav-list-link">archive_sc_tbr1</a><ul class="nav-list "><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-05-07-post-0008.html" class="nav-list-link">Single Cell TBR1 Preliminary Evaluations</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-05-09-post-0010.html" class="nav-list-link">Single Cell TBR1 Coembedding Tests</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-05-11-post-0011.html" class="nav-list-link">Single Cell TBR1 ENCODE DHS Index Tests</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-05-27-post-0013.html" class="nav-list-link">Single Cell TBR1 ENCODE DHS Index Final UMAP and Clusters</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-05-28-post-0014.html" class="nav-list-link">Single Cell TBR1 Cluster DA</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-06-10-post-0022.html" class="nav-list-link">Single Cell TBR1 Cicero/Monocle3 with TSS > 2.7</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-07-09-post-0028.html" class="nav-list-link">Single Cell TBR1 ChromVAR with TSS > 2.7</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-07-12-post-0029.html" class="nav-list-link">Single Cell TBR1 ChromVAR with TSS > 1.5</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-07-18-post-0033.html" class="nav-list-link">Single Cell TBR1 BioCluster/Genotype/Line/Time DA with TSS > 2.7</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-07-19-post-0034.html" class="nav-list-link">Single Cell TBR1 Misc Figures - ChromVAR Compare, FDR Hists, 3D</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-07-26-post-0035.html" class="nav-list-link">Single Cell TBR1 BioCluster/Genotype/Line/Time DA with TSS > 2.7, Filtering Ambiguous Groups</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-07-29-post-0040.html" class="nav-list-link">Single Cell TBR1 Bigwig Tracks</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-08-12-post-0045.html" class="nav-list-link">Single Cell TBR1 Cistopic Boxplots and UMAPs</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-08-12-post-0046.html" class="nav-list-link">Single Cell TBR1 ChIP-Seq Peak DA</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-11-28-post-0065.html" class="nav-list-link">Single Cell TBR1 Pseudo-Bulk Depth Tests</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2022-05-18-post-0087.html" class="nav-list-link">SRA Download Info for 'TBR1 Regulates Autism Risk Genes in the Developing Neocortex'</a></li></ul></li><li class="nav-list-item"><a href="#" class="nav-list-expander" aria-label="toggle links in archive_misc category"> <svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg> </a><a href="/docs/archive_misc/" class="nav-list-link">archive_misc</a><ul class="nav-list "><li class="nav-list-item "><a href="/docs/archive_misc/2020-10-09-post-0003.html" class="nav-list-link">Two Week Update: 2020-10-09</a></li><li class="nav-list-item "><a href="/docs/archive_misc/2020-11-09-post-0004.html" class="nav-list-link">Two Week Update: 2020-11-09</a></li><li class="nav-list-item "><a href="/docs/archive_misc/2021-05-05-post-0007.html" class="nav-list-link">NextSeq Fidelity Evaluation</a></li><li class="nav-list-item "><a href="/docs/archive_misc/2021-11-04-post-0064.html" class="nav-list-link">Contamination References</a></li><li class="nav-list-item "><a href="/docs/archive_misc/2022-03-01-post-0074.html" class="nav-list-link">OneDrive Upload and Download with Rclone</a></li><li class="nav-list-item "><a href="/docs/archive_misc/2022-04-21-post-0081.html" class="nav-list-link">scRNA-seq and scATAC-seq Data Analysis DREAM Challenge - Dataset 2 Data Generation</a></li><li class="nav-list-item "><a href="/docs/archive_misc/2022-05-18-post-0084.html" class="nav-list-link">Celsee and Docker Info</a></li><li class="nav-list-item "><a href="/docs/archive_misc/2020-10-07-post-0001.html" class="nav-list-link">Example</a></li><li class="nav-list-item "><a href="/docs/archive_misc/2022-05-20-post-0089.html" class="nav-list-link">Final O'Roak Lab Data Locations</a></li></ul></li><li class="nav-list-item"><a href="#" class="nav-list-expander" aria-label="toggle links in archive_ssc category"> <svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg> </a><a href="/docs/archive_ssc/" class="nav-list-link">archive_ssc</a><ul class="nav-list "><li class="nav-list-item "><a href="/docs/archive_ssc/2020-10-09-post-0002.html" class="nav-list-link">SSC Final Validation Tables with Annotations</a></li></ul></li></ul> </nav> <footer class="site-footer"> This site uses <a href="https://github.com/just-the-docs/just-the-docs">Just the Docs</a>, a documentation theme for Jekyll. </footer> </div> <div class="main" id="top"> <div id="main-header" class="main-header"> <div class="search"> <div class="search-input-wrap"> <input type="text" id="search-input" class="search-input" tabindex="0" placeholder="Search ARSN Work Blog" aria-label="Search ARSN Work Blog" autocomplete="off"> <label for="search-input" class="search-label"><svg viewBox="0 0 24 24" class="search-icon"><use xlink:href="#svg-search"></use></svg></label> </div> <div id="search-results" class="search-results"></div> </div> <nav aria-label="Auxiliary" class="aux-nav"> <ul class="aux-nav-list"> <li class="aux-nav-list-item"> <a href="//github.com/just-the-docs/just-the-docs" class="site-button" > Just the Docs on GitHub </a> </li> </ul> </nav> </div> <div id="main-content-wrap" class="main-content-wrap"> <nav aria-label="Breadcrumb" class="breadcrumb-nav"> <ol class="breadcrumb-nav-list"> <li class="breadcrumb-nav-list-item"><a href="/docs/archive_spark/">archive_spark</a></li> <li class="breadcrumb-nav-list-item"><span>21-07-27, SPARK Mosaic Index, July 27</span></li> </ol> </nav> <div id="main-content" class="main-content" role="main"> <p>Compiled Writing and Figures: <br /><a href="https://docs.google.com/document/d/1q82t5ll2yVP9Fyrwq7jdnZU2bgDnZEgXCu-Tl0VnDDc/edit?usp=sharing">Google Doc</a> <br /><a href="https://www.dropbox.com/s/0pdmukd8jp5pq7p/figures_01_filter.zip?dl=0">Figures 1, Filtering</a> <br /><a href="https://www.dropbox.com/s/reg9z0jluwkvx1g/figures_02_metrics.zip?dl=0">Figures 2, HC List Metrics</a> <br /><a href="https://www.dropbox.com/s/9lpiv5djf2stod3/figures_03_burden.zip?dl=0">Figures 3, Burden</a></p> <p>Final Mosaic Lists (July 27th): <br /><a href="https://www.dropbox.com/s/2v0su9gdz25yv5w/quadsimplex.burden.txt?dl=0">Simplex Quads, for Burden Analysis</a> <br /><a href="https://www.dropbox.com/s/dphpofhrh7xa84z/quadsimplex.highconfidence.txt?dl=0">Simplex Quads, High Confidence Calls</a> <br /><a href="https://www.dropbox.com/s/8rlzsphugatls37/triosimplex.burden.txt?dl=0">Simplex Trios, for Burden Analysis</a> <br /><a href="https://www.dropbox.com/s/lrdb5trc75ohgx8/triosimplex.highconfidence.txt?dl=0">Simplex Trios, High Confidence Calls</a> <br /><a href="https://www.dropbox.com/s/luyljm77ci1hu1k/quadmultiplex.burden.txt?dl=0">Multiplex Quads, for Burden Analysis</a> <br /><a href="https://www.dropbox.com/s/cn3x5sfm30eq04a/quadmultiplex.highconfidence.txt?dl=0">Multiplex Quads, High Confidence Calls</a> <br /><a href="https://www.dropbox.com/s/hkjkeqibfk2nkyw/ssc.transformed.txt?dl=0">SSC List in SPARK List Format</a></p> <p>Markdowns: <br /><a href="https://www.dropbox.com/s/o2qqos8qembl586/01_qs.organize_prefilter.html?dl=0">01_QS, Organizing and Prefiltering</a> <br /><a href="https://www.dropbox.com/s/og3913n31a8z27a/01_ts.organize_prefilter.html?dl=0">01_TS, Organizing and Prefiltering</a> <br /><a href="https://www.dropbox.com/s/jpmql5snxqr2uyv/01_qm.organize_prefilter.html?dl=0">01_QM, Organizing and Prefiltering</a> <br /><a href="https://www.dropbox.com/s/8y75fw081y8ddo5/02_qs.outlier_detection.html?dl=0">02_QS, Outlier Detection</a> <br /><a href="https://www.dropbox.com/s/62cr2hmihv0phas/02_ts.outlier_detection.html?dl=0">02_TS, Outlier Detection</a> <br /><a href="https://www.dropbox.com/s/bxvqbgd8adjlwj2/02_qm.outlier_detection.html?dl=0">02_QM, Outlier Detection</a> <br /><a href="https://www.dropbox.com/s/vqx366c95usmanb/03_qs.final_tables.html?dl=0">03_QS, Final Table</a> <br /><a href="https://www.dropbox.com/s/06esgmqu3fux0h7/03_ts.final_tables.html?dl=0">03_TS, Final Table</a> <br /><a href="https://www.dropbox.com/s/tn6njzth0v7v11u/03_qm.final_tables.html?dl=0">03_QM, Final Table</a> <br /><a href="https://www.dropbox.com/s/6s5dhposyznatnl/04_qs_highconf_allelefreqs_01.html?dl=0">04, High Confidence Simplex Quad Allele Frequencies</a> <br /><a href="https://www.dropbox.com/s/mbcztsj0j9th4yx/05_highconf_splicedist_01.html?dl=0">05 High Confidence Simplex Quad Splice Distances</a> <br /><a href="https://www.dropbox.com/s/zzd6ikgyh4g8bzp/05_highconf_splicedist_02.html?dl=0">05 High Confidence All Lists Splice Distances</a> <br /><a href="https://www.dropbox.com/s/5l34qb95m88tuvt/06_burden_autosomal_01.html?dl=0">06 Burden Tables Autosomal</a> <br /><a href="https://www.dropbox.com/s/879g61jul2vnryi/07_burden_genesets_01.html?dl=0">07_01 Burden Gene Sets</a> <br /><a href="https://www.dropbox.com/s/zwvp4yolro6gdvp/07_burden_genesets_02.html?dl=0">07_02 Burden Gene Sets</a> <br /><a href="https://www.dropbox.com/s/a2sr4qp1pfwip3q/07_burden_genesets_03.html?dl=0">07_03 Burden Gene Sets</a> <br /><a href="https://www.dropbox.com/s/pjrsx5sgete3fzr/07_burden_genesets_04.html?dl=0">07_04 Burden Gene Sets</a> <br /><a href="https://www.dropbox.com/s/obkmyne6iz28frt/08_concat_lists_01.html?dl=0">08 Concatenate High Confidence Lists</a> <br /><a href="https://www.dropbox.com/s/p70jk08tzvqgb10/09_lgd_ns_lists.html?dl=0">09 Generation of LGD and NS Lists</a> <br /><a href="https://www.dropbox.com/s/hbkj3lsf8bcrj4a/10_summarize_final_lists.html?dl=0">10 Summarize Final Lists</a> <br /><a href="https://www.dropbox.com/s/arsi93z6a5eoy6b/11_hc_AF_quadsimplex.html?dl=0">11 High Confidence AF Dists, QS</a> <br /><a href="https://www.dropbox.com/s/odjqybqtzlmqjgx/11_hc_AF_qs_ts.html?dl=0">11 High Confidence AF Dists, QS and TS</a> <br /><a href="https://www.dropbox.com/s/p1nkifu6mav4sw5/11_hc_AF_quads_quadm_trios.html?dl=0">11 High Confidence AF Dists, QS, TS, and QM</a> <br /><a href="https://www.dropbox.com/s/p7xp30rzsszzd5p/12_burden_genesets_qs_missense.html?dl=0">12 Burden Genesets, QS and missense</a> <br /><a href="https://www.dropbox.com/s/bww75i48qwxyl7f/12_burden_genesets_qs_synonymous.html?dl=0">12 Burden Genesets, QS and synonymous</a> <br /><a href="https://www.dropbox.com/s/9sglvog8nkacxi9/12_burden_genesets_qs_nonsensesplicing.html?dl=0">12 Burden Genesets, QS and nonsense-splicing</a> <br /><a href="https://www.dropbox.com/s/19obpuviidgkncb/12_burden_genesets_qsts_missense.html?dl=0">12 Burden Genesets, QS and TS and missense</a> <br /><a href="https://www.dropbox.com/s/cbg14hg27jurzjq/12_burden_genesets_qsts_synonymous.html?dl=0">12 Burden Genesets, QS and TS and synonymous</a> <br /><a href="https://www.dropbox.com/s/uxw9y1evp7qrar6/12_burden_genesets_qsts_nonsensesplicing.html?dl=0">12 Burden Genesets, QS and TS and nonsense-splicing</a> <br /><a href="https://www.dropbox.com/s/ag58qxihyjcc3pk/13_burden_revel_qs_missense.html?dl=0">13 Burden REVEL, QS and missense</a> <br /><a href="https://www.dropbox.com/s/ng86fbx2o4ap3sw/13_burden_revel_qsts_missense.html?dl=0">13 Burden REVEL, QS+TS and missense</a> <br /><a href="https://www.dropbox.com/s/t7suz2lrk612wvt/14_parental_transmission_01.html?dl=0">14 Parental Transmission Breakdown</a> <br /><a href="https://www.dropbox.com/s/5v9xiutv72g4tj1/15_spark_167_list_01.html?dl=0">15 SPARK 167 Intersection</a> <br /><a href="https://www.dropbox.com/s/v0mzdm4jevtaq4x/16_concat_lists_02.html?dl=0">16 Concat Variant Lists to DNV List</a> <br /><a href="https://www.dropbox.com/s/j1flcekwsrp6s03/17_ssc_list.html?dl=0">17 SSC List Formatting to Current SPARK</a> <br /><a href="https://www.dropbox.com/s/0pv9gk7cxk24v9j/19_burden_genesets_ssc_missense.html?dl=0">19 SSC Solo Burden Markdown</a> <br /><a href="https://www.dropbox.com/s/thbr9vgqibvbobj/01_WES2_SQ.html?dl=0">01 WES2_SQ Prep</a> <br /><a href="https://www.dropbox.com/s/we6ixqoi5iedtfn/01_WES2_MQ.html?dl=0">01 WES2_MQ Prep</a> <br /><a href="https://www.dropbox.com/s/2lt3iohpemotvew/01_WES2_ST.html?dl=0">01 WES2_ST Prep</a> <br /><a href="https://www.dropbox.com/s/cpwikcnv1hrjx34/01_WES3_SQ.html?dl=0">01 WES3_SQ Prep</a> <br /><a href="https://www.dropbox.com/s/q7eahh0vj743377/01_WES3_MQ.html?dl=0">01 WES3_MQ Prep</a> <br /><a href="https://www.dropbox.com/s/j0aqujobwvemu0a/01_WES3_ST.html?dl=0">01 WES3_ST Prep</a> <br /><a href="https://www.dropbox.com/s/5ytjk71msu8xe21/02_WES2_SQ.html?dl=0">02 WES2_SQ Outlier Plots</a> <br /><a href="https://www.dropbox.com/s/ldlqrday2v6cpq7/02_WES2_MQ.html?dl=0">02 WES2_MQ Outlier Plots</a> <br /><a href="https://www.dropbox.com/s/8meafdj3kj3ec10/02_WES2_ST.html?dl=0">02 WES2_ST Outlier Plots</a> <br /><a href="https://www.dropbox.com/s/gw650phhs3db3xx/02_WES3_SQ.html?dl=0">02 WES3_SQ Outlier Plots</a> <br /><a href="https://www.dropbox.com/s/6mfpzw4pl8996e3/02_WES3_MQ.html?dl=0">02 WES3_MQ Outlier Plots</a> <br /><a href="https://www.dropbox.com/s/jeeqfj7bcdrhfxn/02_WES3_ST.html?dl=0">02 WES3_ST Outlier Plots</a> <br /><a href="https://www.dropbox.com/s/7gcqbxsjhtdy7nd/02_WES2_WES3_combined.html?dl=0">WES2 and WES3 Combined Coverage Plots</a></p> <p>Family Lists: <br /><a href="https://www.dropbox.com/s/awrb72bkxcxgmix/final_sample_list.txt?dl=0">Final Sample List</a> <br /><a href="https://www.dropbox.com/s/lscs9dl6yudj0qn/final_filter_igv_cohort.txt?dl=0">Final filter list by IGV shots and concatenated cohort count</a> <br /><a href="https://www.dropbox.com/s/y061oxm1pw4eykx/lgd_family_list.txt?dl=0">LGD List</a> <br /><a href="https://www.dropbox.com/s/a8bbg0konbpuppw/ns_family_list.txt?dl=0">NS List</a> <br /><a href="https://www.dropbox.com/s/naugav6a3labpab/SSC_families_lgd_ns.txt?dl=0">SSC Family and LGD/NS List</a></p> <p>Mosaic Lists, Full Non-Analysis Sets (March 30th, 2022): <br /><a href="https://www.dropbox.com/s/jbimhfc6pk55797/quadsimplex_mosaics.8reads_0.001popAF_4cohort_3alt_exonic.220216.txt?dl=0">Simplex Quads w/ Loose Filter</a> <br /><a href="https://www.dropbox.com/s/8h0054xbllpk266/quadmultiplex_mosaics.8reads_0.001popAF_4cohort_3alt_exonic.220216.txt?dl=0">Multiplex Quads w/ Loose Filter</a> <br /><a href="https://www.dropbox.com/s/xjhnyvq1kjaapzf/triosimplex_mosaics.8reads_0.001popAF_4cohort_3alt_exonic.220216.txt?dl=0">Simplex Trios w/ Loose Filter</a> <br /><a href="https://www.dropbox.com/s/0pbv4g2a5k3ky5j/quadsimplex_mosaics.8reads_0.001popAF_4cohort_3alt_exonic.220216.testfilter.txt?dl=0">Simplex Quads w/ Strict Test Filter</a> <br /><a href="https://www.dropbox.com/s/ev8muf74dq025ce/quadmultiplex_mosaics.8reads_0.001popAF_4cohort_3alt_exonic.220216.testfilter.txt?dl=0">Multiplex Quads w/ Strict Test Filter</a> <br /><a href="https://www.dropbox.com/s/w4kekne3v7acuhx/triosimplex_mosaics.8reads_0.001popAF_4cohort_3alt_exonic.220216.testfilter.txt?dl=0">Simplex Trios w/ Strict Test Filter</a> <br /><a href="https://www.dropbox.com/s/grld1vbtpki4ccq/quadsimplex_mosaics.8reads_0.001popAF_4cohort_3alt_exonic.220216.testfilter_noGermlineParental.txt?dl=0">Simplex Quads w/ Strict Test Filter, no Germline Parental</a> <br /><a href="https://www.dropbox.com/s/6x11s3lne639m0a/quadmultiplex_mosaics.8reads_0.001popAF_4cohort_3alt_exonic.220216.testfilter_noGermlineParental.txt?dl=0">Multiplex Quads w/ Strict Test Filter, no Germline Parental</a> <br /><a href="https://www.dropbox.com/s/qc8ugca2am346xf/triosimplex_mosaics.8reads_0.001popAF_4cohort_3alt_exonic.220216.testfilter_noGermlineParental.txt?dl=0">Simplex Trios w/ Strict Test Filter, no Germline Parental</a> <br /><a href="https://www.dropbox.com/s/ih2gauu9cx0z8z4/WES2.SQ_mosaics.8reads_0.001popAF_4cohort_3alt_exonic.txt?dl=0">WES2, Simplex Quads w/ Loose Filter</a> <br /><a href="https://www.dropbox.com/s/io19j9j3b7xlzgt/WES2.MQ_mosaics.8reads_0.001popAF_4cohort_3alt_exonic.txt?dl=0">WES2, Multiplex Quads w/ Loose Filter</a> <br /><a href="https://www.dropbox.com/s/fvncosmegwswo36/WES2.ST_mosaics.8reads_0.001popAF_4cohort_3alt_exonic.txt?dl=0">WES2, Simplex Trios w/ Loose Filter</a> <br /><a href="https://www.dropbox.com/s/35fwzog99dy5s91/WES2.SQ_mosaics.8reads_0.001popAF_4cohort_3alt_exonic.testfilter.txt?dl=0">WES2, Simplex Quads w/ Strict Test Filter</a> <br /><a href="https://www.dropbox.com/s/8ol2h29sx887w79/WES2.MQ_mosaics.8reads_0.001popAF_4cohort_3alt_exonic.testfilter.txt?dl=0">WES2, Multiplex Quads w/ Strict Test Filter</a> <br /><a href="https://www.dropbox.com/s/1hfc09k0s0qm5pb/WES2.ST_mosaics.8reads_0.001popAF_4cohort_3alt_exonic.testfilter.txt?dl=0">WES2, Simplex Trios w/ Strict Test Filter</a> <br /><a href="https://www.dropbox.com/s/j3xow8h6dquigon/WES2.SQ_mosaics.8reads_0.001popAF_4cohort_3alt_exonic.testfilter_noGermlineParental.txt?dl=0">WES2, Simplex Quads w/ Strict Test Filter, no Germline Parental</a> <br /><a href="https://www.dropbox.com/s/52v329kf70e6iq5/WES2.MQ_mosaics.8reads_0.001popAF_4cohort_3alt_exonic.testfilter_noGermlineParental.txt?dl=0">WES2, Multiplex Quads w/ Strict Test Filter, no Germline Parental</a> <br /><a href="https://www.dropbox.com/s/xxjvlvcc26iphoh/WES2.ST_mosaics.8reads_0.001popAF_4cohort_3alt_exonic.testfilter_noGermlineParental.txt?dl=0">WES2, Simplex Trios w/ Strict Test Filter, no Germline Parental</a> <br /><a href="https://www.dropbox.com/s/03yen5se00ecwx0/WES3.SQ_mosaics.8reads_0.001popAF_4cohort_3alt_exonic.txt?dl=0">WES3, Simplex Quads w/ Loose Filter</a> <br /><a href="https://www.dropbox.com/s/hmibhizpl9126jm/WES3.MQ_mosaics.8reads_0.001popAF_4cohort_3alt_exonic.txt?dl=0">WES3, Multiplex Quads w/ Loose Filter</a> <br /><a href="https://www.dropbox.com/s/tsngioi7azyhyo1/WES3.ST_mosaics.8reads_0.001popAF_4cohort_3alt_exonic.txt?dl=0">WES3, Simplex Trios w/ Loose Filter</a> <br /><a href="https://www.dropbox.com/s/kc30k9ibhsvrvu2/WES3.SQ_mosaics.8reads_0.001popAF_4cohort_3alt_exonic.testfilter.txt?dl=0">WES3, Simplex Quads w/ Strict Test Filter</a> <br /><a href="https://www.dropbox.com/s/6c6328ghlebby49/WES3.MQ_mosaics.8reads_0.001popAF_4cohort_3alt_exonic.testfilter.txt?dl=0">WES3, Multiplex Quads w/ Strict Test Filter</a> <br /><a href="https://www.dropbox.com/s/dyaccssd0qx2p75/WES3.ST_mosaics.8reads_0.001popAF_4cohort_3alt_exonic.testfilter.txt?dl=0">WES3, Simplex Trios w/ Strict Test Filter</a> <br /><a href="https://www.dropbox.com/s/xq5rwlg3e9cxf7q/WES3.SQ_mosaics.8reads_0.001popAF_4cohort_3alt_exonic.testfilter_noGermlineParental.txt?dl=0">WES3, Simplex Quads w/ Strict Test Filter, no Germline Parental</a> <br /><a href="https://www.dropbox.com/s/kjcp2vopnm7mjuv/WES3.MQ_mosaics.8reads_0.001popAF_4cohort_3alt_exonic.testfilter_noGermlineParental.txt?dl=0">WES3, Multiplex Quads w/ Strict Test Filter, no Germline Parental</a> <br /><a href="https://www.dropbox.com/s/9hctuypk36x4nqy/WES3.ST_mosaics.8reads_0.001popAF_4cohort_3alt_exonic.testfilter_noGermlineParental.txt?dl=0">WES3, Simplex Trios w/ Strict Test Filter, no Germline Parental</a></p> <p>SPARK Annotations Used, 20190820: <br /><a href="https://www.dropbox.com/s/bjwj52ibth2rytw/quadsimplex.highconfidence.add_annot.txt?dl=0">Simplex Quad, Added Annotations</a> <br /><a href="https://www.dropbox.com/s/fgr8kt3mt98juq5/quadmultiplex.highconfidence.add_annot.txt?dl=0">Multiplex Quad, Added Annotations</a> <br /><a href="https://www.dropbox.com/s/ntn4cr1jmh44q70/triosimplex.highconfidence.add_annot.txt?dl=0">Simplex Trio, Added Annotations</a> <br /><a href="https://www.dropbox.com/s/gg5ewe0bbqhjimw/SPARK.27K.mastertable.20190820.txt?dl=0">SPARK Annotations</a></p> <p>Links to List of Gene Names with Alternate Names: <br /><a href="https://www.dropbox.com/s/g19mqe9ayqrqnf4/spark_gene_list_sept2021.txt?dl=0">SPARK list from https://sparkforautism.org/portal/page/spark-gene-list/</a> <br /><a href="https://www.dropbox.com/s/4jcjuaz8c3dhbtf/Essential.alt.genes?dl=0">Essential.alt.genes</a> <br /><a href="https://www.dropbox.com/s/jbrpq5fb8qz16ri/Exac.Intol.alt.genes?dl=0">Exac.Intol.alt.genes</a> <br /><a href="https://www.dropbox.com/s/yjw7r0fwh3lxqil/Exac.Mid.alt.genes?dl=0">Exac.Mid.alt.genes</a> <br /><a href="https://www.dropbox.com/s/443h1423awxmxza/Exac.Tol.alt.genes?dl=0">Exac.Tol.alt.genes</a> <br /><a href="https://www.dropbox.com/s/qbptliwg3irwnod/FUSIL.alt.genes?dl=0">FUSIL.alt.genes</a></p> <p>Supplemental Annotations: <br /><a href="https://www.dropbox.com/s/4aaak22tpx5r1kl/revel.triosimplex.txt?dl=0">Supplemental Annotations, REVEL, trio simplex</a> <br /><a href="https://www.dropbox.com/s/svr5kdpds11chqq/revel.quadsimplex.txt?dl=0">Supplemental Annotations, REVEL, quad simplex</a> <br /><a href="https://www.dropbox.com/s/n5ahznpustist5b/revel.quadmultiplex.txt?dl=0">Supplemental Annotations, REVEL, quad simplex</a> <br /><a href="https://www.dropbox.com/s/ivp1z6l7hyqr4my/spark167.mosaic_child.txt?dl=0">HC list Variants Intersecting w/ SPARK 167, Mosaic Child only</a> <br /><a href="https://www.dropbox.com/s/ftzjerfi7ebhw4q/spark167.txt?dl=0">HC list Variants Intersecting w/ SPARK 167</a></p> <p>Joint Coverage: <br /><a href="https://www.dropbox.com/s/72p5sfqhwmbiqsz/transformed.jointcov_ssc.all.txt?dl=0">Genomic Joint Coverage, SSC Joint Coverage</a> <br /><a href="https://www.dropbox.com/s/xp96tughn8vhjzu/jointcoverage.2.simplexquads.txt?dl=0">Genomic Joint Coverage, WES1 Simplex Quads</a> <br /><a href="https://www.dropbox.com/s/dm5ppo3fn2q0vrv/jointcoverage.2.multiplexquads.txt?dl=0">Genomic Joint Coverage, WES1 Multiplex Quads</a> <br /><a href="https://www.dropbox.com/s/pfkw1v3z7rs4ugy/jointcoverage.2.simplextrios.txt?dl=0">Genomic Joint Coverage, WES1 Simplex Trios</a> <br /><a href="https://www.dropbox.com/s/2231a1bsnxekwz8/jointcov.WES2.SQ.txt?dl=0">Genomic Joint Coverage, WES2 Simplex Quads</a> <br /><a href="https://www.dropbox.com/s/xr3d2cer91onuza/jointcov.WES2.MQ.txt?dl=0">Genomic Joint Coverage, WES2 Multiplex Quads</a> <br /><a href="https://www.dropbox.com/s/ngmo9db5rwtx24u/jointcov.WES2.ST.txt?dl=0">Genomic Joint Coverage, WES2 Simplex Trios</a> <br /><a href="https://www.dropbox.com/s/7zte7fdecuq6wy2/jointcov.WES3.SQ.txt?dl=0">Genomic Joint Coverage, WES3 Simplex Quads</a> <br /><a href="https://www.dropbox.com/s/g5dhkfkkoqi4lv5/jointcov.WES3.MQ.txt?dl=0">Genomic Joint Coverage, WES3 Multiplex Quads</a> <br /><a href="https://www.dropbox.com/s/2dbh83d0v2m1sot/jointcov.WES3.ST.txt?dl=0">Genomic Joint Coverage, WES3 Simplex Trios</a></p> <p>IGV Shots: <br /><a href="https://www.dropbox.com/sh/nj8wglswrxj7lik/AADvVXrVXapW3lukFMppeuLAa?dl=0">IGV Shots of High Confidence Sites</a></p> <p>Here are descriptions of all the columns in the table: <br />1/CHR, 2/START, 3/STOP, 4/REF, 5/ALT - general variant information <br />6/FAMILY, 7/SAMPLE, 8/FRACTION - SF family ID, SP sample ID, putative mosaic annotation <br />9/DP_MAIN, 10/DPALT_MAIN, 11/AF_MAIN - the depth of reads, depth of alt reads, and AF for the main person of the family <br />12/LOFREQ_STATS_NAMES, 13/LOFREQ_STATS, 14/MPUP_STATS_NAMES, 15/MPUP_STATS, 16/VARSCAN_STATS_NAMES, 17/VARSCAN_STATS - stats from the three variant callers with their descriptor fields <br />18/MISMATCH_REF_READS, 19/MISMATCH_ALT_READS, 20/MISMATCH_OTHER_READS, 21/MISMATCH_SPAN_DELS, 22/MISMATCH_MAPQ_REF_MEAN, 23/MISMATCH_MAPQ_ALT_MEAN, 24/MISMATCH_MAPQ_OTHER_MEAN, 25/MISMATCH_MAPQ_REF_MEDIAN, 26/MISMATCH_MAPQ_ALT_MEDIAN, 27/MISMATCH_MAPQ_OTHER_MEDIAN, 28/MISMATCH_MAPQ_REF_MODE, 29/MISMATCH_MAPQ_ALT_MODE, 30/MISMATCH_MAPQ_OTHER_MODE, 31/MISMATCH_XM_REF_MEAN, 32/MISMATCH_XM_ALT_MEAN, 33/MISMATCH_XM_OTHER_MEAN, 34/MISMATCH_XM_REF_MEDIAN, 35/MISMATCH_XM_ALT_MEDIAN, 36/MISMATCH_XM_OTHER_MEDIAN, 37/MISMATCH_XM_REF_MODE, 38/MISMATCH_XM_ALT_MODE, 39/MISMATCH_XM_OTHER_MODE - large collection of stats concerning mismatches across the region, only MISMATCH_XM_ALT_MEDIAN gets used downstream for another derived column <br />40/DP_FA, 41/DP_MO, 42/DP_PRO, 43/DP_SIB, 44/DPALT_FA, 45/DPALT_MO, 46/DPALT_PRO, 47/DPALT_SIB - depths across all family members <br />48/AF_EXAC, 49/AF_gnomAD - AFs from ExAC v0.3 and gnomAD 2.1 <br />50/ERROR_VAR, 51/ERROR_ALL, 52/ERROR_VARMAX, 53/ERROR_RATE - error in the region calculated over a random subset of 1000 samples, only rate gets used downstream for another derived column <br />54/NUM_FAMILY_FOUND - number of families the variant is found before filtering <br />55/PHET_FA, 56/PHET_MO, 57/PHET_PRO, 58/PHET_SIB - pvalue of exact test given the depth and the depth of the alt read in each family member testing if it is less than 0.5 (heterogeneous) <br />59/PERSON - identity of the person <br />60/DPALT_BIN - binary column derived from DPALT_MAIN used for SCORE_400_RB3 <br />61/ERROR_BIN_3 - categorical column (low, medium, high) derived from ERROR_RATE used for SCORE_400_RB3 <br />62/AF_BIN_3 - categorical column (low, medium, high) derived from AF_MAIN used for SCORE_400_RB3 <br />63/REF_CG - categorical column (C or G) derived from REF used for SCORE_400_RB3 <br />64/XM_ALT_1 - binary column derived from MISMATCH_XM_ALT_MEDIAN used for SCORE_400_RB3 <br />65/SCORE_400_RB3 - score from logistic regression model that takes into account many of the other columns <br />66/genomicSuperDupes, 67/rmsk, 68/simpleRepeat, 69/mutationType, 70/exonicAnnotation, 71/variantFunctionType, 72/variantFunctionGene - annotations with annovar-2018-06-15 <br />73/filteredCohortCount - number of families the variant is found after preliminary filtering <br />74/AF_L90CI_AC_MAIN, 75/AF_U90CI_AC_MAIN - upper and lower 90% confidence intervals via Agresti-Coull method <br />76/FRACTION_ORIGINAL - old, erroneous annotations of the site for posterity, see May 9 update <br />77/FRACTION_MORE - a subset of column 8, FRACTION, with further granularity, see June 3 update <br />78/PSAME_FAxPRO, 79/PSAME_FAxSIB, 80/PSAME_MOxPRO, 81/PSAME_MOxSIB - Fisher’s exact test comparing depths and alt reads from parents versus children for the purposes of testing a transmission event. Final tables also contain: <br />82/MIN_DP - The minimum depth of all family members. <br />83/ACC_DIST, 84/DON_DIST, 85/ACC_TRANSCRIPT, 86/DON_TRANSCRIPT - The distances in basepairs to an acceptor or donor site and which transcript from which this distance is derived. <br />87/COMPLEX_EVENT - All FALSE in final tables representing collapsed complex events. <br />88/HGNCsymbol_to_ENSID, 89/RefSeq_to_ENSID - ENSEMBL IDs derived from both HGNC symbosl and RefSeq</p> <p>dbNSFP Column Descriptions: <br /><a href="https://www.dropbox.com/s/j95lkv05al56kcp/dbNSFP_columns.txt?dl=0">Added dbNSFP Annotation Column Descriptions</a></p> </div> </div> <div class="search-overlay"></div> </div> </body> </html>
