<!DOCTYPE html> <html lang="en-US"> <head> <meta charset="UTF-8"> <meta http-equiv="X-UA-Compatible" content="IE=Edge"> <title>21-11-30, SPARK SSC List Formatting to Current SPARK - ARSN Work Blog</title> <link rel="shortcut icon" href="/favicon.ico" type="image/x-icon"> <link rel="stylesheet" href="/assets/css/just-the-docs-default.css"> <script src="/assets/js/vendor/lunr.min.js"></script> <script src="/assets/js/just-the-docs.js"></script> <meta name="viewport" content="width=device-width, initial-scale=1"> <!-- Begin Jekyll SEO tag v2.8.0 --> <title>21-11-30, SPARK SSC List Formatting to Current SPARK | ARSN Work Blog</title> <meta name="generator" content="Jekyll v4.3.0" /> <meta property="og:title" content="21-11-30, SPARK SSC List Formatting to Current SPARK" /> <meta property="og:locale" content="en_US" /> <meta name="description" content="This is a blog for work projects Andrew Nishida, nishidaa@ohsu.edu, in CEDAR at OHSU." /> <meta property="og:description" content="This is a blog for work projects Andrew Nishida, nishidaa@ohsu.edu, in CEDAR at OHSU." /> <link rel="canonical" href="http://localhost:4000/docs/archive_spark/2021-11-30-post-0066.html" /> <meta property="og:url" content="http://localhost:4000/docs/archive_spark/2021-11-30-post-0066.html" /> <meta property="og:site_name" content="ARSN Work Blog" /> <meta property="og:type" content="article" /> <meta property="article:published_time" content="2021-11-30T00:00:00-08:00" /> <meta name="twitter:card" content="summary" /> <meta property="twitter:title" content="21-11-30, SPARK SSC List Formatting to Current SPARK" /> <script type="application/ld+json"> {"@context":"https://schema.org","@type":"BlogPosting","dateModified":"2021-11-30T00:00:00-08:00","datePublished":"2021-11-30T00:00:00-08:00","description":"This is a blog for work projects Andrew Nishida, nishidaa@ohsu.edu, in CEDAR at OHSU.","headline":"21-11-30, SPARK SSC List Formatting to Current SPARK","mainEntityOfPage":{"@type":"WebPage","@id":"http://localhost:4000/docs/archive_spark/2021-11-30-post-0066.html"},"url":"http://localhost:4000/docs/archive_spark/2021-11-30-post-0066.html"}</script> <!-- End Jekyll SEO tag --> </head> <body> <a class="skip-to-main" href="#main-content">Skip to main content</a> <svg xmlns="http://www.w3.org/2000/svg" style="display: none;"> <symbol id="svg-link" viewBox="0 0 24 24"> <title>Link</title> <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-link"> <path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path> </svg> </symbol> <symbol id="svg-search" viewBox="0 0 24 24"> <title>Search</title> <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-search"> <circle cx="11" cy="11" r="8"></circle><line x1="21" y1="21" x2="16.65" y2="16.65"></line> </svg> </symbol> <symbol id="svg-menu" viewBox="0 0 24 24"> <title>Menu</title> <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-menu"> <line x1="3" y1="12" x2="21" y2="12"></line><line x1="3" y1="6" x2="21" y2="6"></line><line x1="3" y1="18" x2="21" y2="18"></line> </svg> </symbol> <symbol id="svg-arrow-right" viewBox="0 0 24 24"> <title>Expand</title> <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-chevron-right"> <polyline points="9 18 15 12 9 6"></polyline> </svg> </symbol> <symbol id="svg-doc" viewBox="0 0 24 24"> <title>Document</title> <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-file"> <path d="M13 2H6a2 2 0 0 0-2 2v16a2 2 0 0 0 2 2h12a2 2 0 0 0 2-2V9z"></path><polyline points="13 2 13 9 20 9"></polyline> </svg> </symbol> <!-- Feather. MIT License: https://github.com/feathericons/feather/blob/master/LICENSE --> <symbol id="svg-external-link" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-external-link"> <title id="svg-external-link-title">(external link)</title> <path d="M18 13v6a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V8a2 2 0 0 1 2-2h6"></path><polyline points="15 3 21 3 21 9"></polyline><line x1="10" y1="14" x2="21" y2="3"></line> </symbol> </svg> <div class="side-bar"> <div class="site-header"> <a href="/" class="site-title lh-tight"> ARSN Work Blog </a> <a href="#" id="menu-button" class="site-button"> <svg viewBox="0 0 24 24" class="icon"><use xlink:href="#svg-menu"></use></svg> </a> </div> <nav aria-label="Main" id="site-nav" class="site-nav"> <ul class="nav-list"><li class="nav-list-item"><a href="/" class="nav-list-link">Home</a></li><li class="nav-list-item"><a href="#" class="nav-list-expander" aria-label="toggle links in sc_breastmultiome category"> <svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg> </a><a href="/docs/sc_breastmultiome/" class="nav-list-link">sc_breastmultiome</a><ul class="nav-list "><li class="nav-list-item "><a href="/docs/sc_breastmultiome/2022-12-12-sc_breastmultiome-epiconfig.html" class="nav-list-link">22-12-12, Epiconfig Runs on Full and Subset Data</a></li><li class="nav-list-item "><a href="/docs/sc_breastmultiome/2022-12-12-sc_breastmultiome-project_info.html" class="nav-list-link">22-12-12, Project Info</a></li><li class="nav-list-item "><a href="/docs/sc_breastmultiome/2023-01-12-sc_breastmultiome-epiconfig-downsample.html" class="nav-list-link">23-01-12, Epiconfig Run on Downampled Epithelial</a></li><li class="nav-list-item "><a href="/docs/sc_breastmultiome/2023-01-17-sc_breastmultiome-epiconfig-pam50umaps.html" class="nav-list-link">23-01-17, PAM50 UMAPs</a></li></ul></li><li class="nav-list-item"><a href="#" class="nav-list-expander" aria-label="toggle links in sc_klf4bc category"> <svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg> </a><a href="/docs/sc_klf4bc/" class="nav-list-link">sc_klf4bc</a><ul class="nav-list "><li class="nav-list-item "><a href="/docs/sc_klf4bc/2022-12-12-sc_breastmultiome-epiconfig_filterset.html" class="nav-list-link">22-12-13, KLF4 BC Initial Processing</a></li><li class="nav-list-item "><a href="/docs/sc_klf4bc/2022-12-12-sc_breastmultiome-epiconfig_filterset2.html" class="nav-list-link">22-12-13, KLF4 BC Initial Processing</a></li></ul></li><li class="nav-list-item"><a href="#" class="nav-list-expander" aria-label="toggle links in sc_dm6tau category"> <svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg> </a><a href="/docs/sc_dm6tau/" class="nav-list-link">sc_dm6tau</a><ul class="nav-list "><li class="nav-list-item "><a href="/docs/sc_dm6tau/2021-10-07-post-0055.html" class="nav-list-link">Drosophilia Tau Preliminary DA</a></li><li class="nav-list-item "><a href="/docs/sc_dm6tau/2022-01-08-post-0069.html" class="nav-list-link">Drosophilia Tau Final DA</a></li><li class="nav-list-item "><a href="/docs/sc_dm6tau/2022-01-21-post-0072.html" class="nav-list-link">Drosophilia Tau DA Results Test Analysis</a></li></ul></li><li class="nav-list-item"><a href="#" class="nav-list-expander" aria-label="toggle links in ravnica category"> <svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg> </a><a href="/docs/ravnica/" class="nav-list-link">ravnica</a><ul class="nav-list "><li class="nav-list-item "><a href="/docs/ravnica/2022-01-25-post-0073.html" class="nav-list-link">Dropbox Upload and Download with Curl and API Key</a></li><li class="nav-list-item "><a href="/docs/ravnica/2022-05-18-post-0086.html" class="nav-list-link">Cron Info for Automatically Running bcl2fastq</a></li><li class="nav-list-item "><a href="/docs/ravnica/2022-12-05-ravnica-global_install_initial.html" class="nav-list-link">22-12-05, Global Installs - Initial</a></li><li class="nav-list-item "><a href="/docs/ravnica/2022-12-05-ravnica-local_install_initial.html" class="nav-list-link">22-12-05, Local Installs - Initial</a></li><li class="nav-list-item "><a href="/docs/ravnica/2022-12-05-ravnica-mamba_install_initial.html" class="nav-list-link">22-12-05, Mamba Install - Initial</a></li><li class="nav-list-item "><a href="/docs/ravnica/2022-12-05-ravnica-mamba_setup.html" class="nav-list-link">22-12-05, Mamba Setup</a></li><li class="nav-list-item "><a href="/docs/ravnica/2022-12-05-ravnica-r_4.2.2_library_info.html" class="nav-list-link">22-12-05, R 4.2.2 Library Info</a></li><li class="nav-list-item "><a href="/docs/ravnica/2022-12-06-ravnica-example_bashprofile.html" class="nav-list-link">22-12-06, Example Bash Profile</a></li><li class="nav-list-item "><a href="/docs/ravnica/2022-12-06-ravnica-venv_macs3.html" class="nav-list-link">22-12-06, VENV for Macs3</a></li></ul></li><li class="nav-list-item"><a href="#" class="nav-list-expander" aria-label="toggle links in scitools category"> <svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg> </a><a href="/docs/scitools/" class="nav-list-link">scitools</a><ul class="nav-list "><li class="nav-list-item "><a href="/docs/scitools/2020-12-17-post-0005.html" class="nav-list-link">Scitools scATAC Metrics</a></li><li class="nav-list-item "><a href="/docs/scitools/2021-02-18-post-0006.html" class="nav-list-link">Scitools Scrublet Function</a></li><li class="nav-list-item "><a href="/docs/scitools/2021-05-08-post-0009.html" class="nav-list-link">Single Cell Alignments with no Alt Chromosomes</a></li><li class="nav-list-item "><a href="/docs/scitools/2022-04-03-post-0077.html" class="nav-list-link">scRNA-seq First Pass Pipeline - 220324_HB_Coassay_RNA</a></li><li class="nav-list-item "><a href="/docs/scitools/2022-04-11-post-0080.html" class="nav-list-link">scRNA-seq Second Pass Pipeline - 220324_HB_Coassay_RNA</a></li></ul></li><li class="nav-list-item active"><a href="#" class="nav-list-expander" aria-label="toggle links in archive_spark category"> <svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg> </a><a href="/docs/archive_spark/" class="nav-list-link">archive_spark</a><ul class="nav-list "><li class="nav-list-item "><a href="/docs/archive_spark/2021-05-11-post-0012.html" class="nav-list-link">21-05-11, SPARK Mosaic List, Summary Counts of Annotations</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-05-28-post-0015.html" class="nav-list-link">21-05-28, SPARK RefSeq CDS and Splice Site Reference Generation</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-05-30-post-0016.html" class="nav-list-link">21-05-30, SPARK Joint Coverage Counter Tool Fidelity</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-05-30-post-0017.html" class="nav-list-link">21-05-30, SPARK Joint Coverage Plots</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-06-02-post-0018.html" class="nav-list-link">21-06-02, SPARK Joint Coverage Table Fix</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-06-03-post-0019.html" class="nav-list-link">21-06-03, SPARK Mosaic List FRACTION_MORE Annotations</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-06-03-post-0020.html" class="nav-list-link">21-06-03, SPARK Mosaic Index</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-06-10-post-0021.html" class="nav-list-link">21-06-10, SPARK Gene Set Joint Coverage</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-06-22-post-0023.html" class="nav-list-link">21-06-22, SPARK Preliminary Pass 01, Filtering and Classifications</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-06-22-post-0024.html" class="nav-list-link">21-06-22, SPARK Preliminary Pass 02, Loading for Coverage and SNV Count Plots</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-06-26-post-0025.html" class="nav-list-link">21-06-26, SPARK Preliminary Pass 02b, Coverage and SNV Count Plots as Hists</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-06-28-post-0026.html" class="nav-list-link">21-06-28, SPARK Preliminary Pass 03, Allele Fraction Histograms</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-07-06-post-0027.html" class="nav-list-link">21-07-06, SPARK Preliminary Pass 04, Coverages and SNV Counts with Joint Family Coverages</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-07-14-post-0030.html" class="nav-list-link">21-07-14, SPARK Preliminary Pass 5, Final Filtering</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-07-17-post-0031.html" class="nav-list-link">21-07-17, SPARK Preliminary Pass 6a and 6b, Simplex Trio Coverages, SNV Counts, and AFs</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-07-17-post-0032.html" class="nav-list-link">21-07-17, SPARK Preliminary Pass 7a and 7b, Multiplex Quad Coverages, SNV Counts, and AFs</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-07-26-post-0036.html" class="nav-list-link">21-07-26, SPARK First Pass, Synonymous SNV Splice Distances</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-07-26-post-0037.html" class="nav-list-link">21-07-26, SPARK Preliminary Outlier Detection, Secondary Pass</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-07-26-post-0038.html" class="nav-list-link">21-07-26, SPARK Final Table Creation</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-07-27-post-0039.html" class="nav-list-link">21-07-27, SPARK Mosaic Index, July 27</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-08-03-post-0041.html" class="nav-list-link">21-08-03, SPARK High Confidence Allele Frequencies 01</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-08-03-post-0042.html" class="nav-list-link">21-08-03, SPARK High Confidence Synonymous Splice Distances 01</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-08-03-post-0043.html" class="nav-list-link">21-08-03, SPARK Burden Tables Autosomal 01</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-08-09-post-0044.html" class="nav-list-link">21-08-09, SPARK Burden Tables Gene Sets 01</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-08-26-post-0047.html" class="nav-list-link">21-08-26, SPARK IGV Shots of High Confident Calls</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-09-08-post-0048.html" class="nav-list-link">21-09-08, SPARK Burden Tables Gene Sets 02</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-09-14-post-0049.html" class="nav-list-link">21-09-14, SPARK Burden Tables Gene Sets 03</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-09-15-post-0050.html" class="nav-list-link">21-09-15, SPARK Concatenated High Confidence Table Checks 01</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-01-post-0051.html" class="nav-list-link">21-10-1, SPARK Final Sample List</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-01-post-0052.html" class="nav-list-link">21-10-01, SPARK Final IGV and Cohort Count Filter</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-06-post-0053.html" class="nav-list-link">21-10-06, SPARK LGD and NS List</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-06-post-0054.html" class="nav-list-link">21-10-06, SPARK Burden Tables Gene Sets 04</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-10-post-0056.html" class="nav-list-link">21-10-10, SPARK Summary of Final Lists</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-10-post-0057.html" class="nav-list-link">21-10-10, SPARK Final List Allele Frequency Distributions</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-11-post-0058.html" class="nav-list-link">21-10-11, SPARK Monozygotic Twins in Quad Families</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-11-post-0059.html" class="nav-list-link">21-10-11, SPARK Burden Tables Gene Sets, Final 01</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-13-post-0060.html" class="nav-list-link">21-10-13, SPARK Burden w/ REVEL Scores Gene Sets, 01</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-13-post-0061.html" class="nav-list-link">21-10-13, SPARK Parental Trans Mosaics Breakdown</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-13-post-0062.html" class="nav-list-link">21-10-13, SPARK HC Tables Against SPARK 167</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-10-14-post-0063.html" class="nav-list-link">21-10-14, SPARK Concat Variant Lists to DNV List Comparison</a></li><li class="nav-list-item active"><a href="/docs/archive_spark/2021-11-30-post-0066.html" class="nav-list-link active">21-11-30, SPARK SSC List Formatting to Current SPARK</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-11-30-post-0067.html" class="nav-list-link">21-11-30, SPARK Burden Tables Gene Sets, SPARK+SSC</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2021-12-03-post-0068.html" class="nav-list-link">21-12-03, SPARK VEP Annotations</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2022-01-18-post-0070.html" class="nav-list-link">22-01-18, SPARK SSC hg38 Transformed Burden, 01</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2022-01-19-post-0071.html" class="nav-list-link">22-01-19, SPARK Pre Best Practices Filter, Model Scores</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2022-03-19-post-0076.html" class="nav-list-link">22-03-19, SPARK WES2 and WES3 List Inconsistencies</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2022-04-08-post-0079.html" class="nav-list-link">22-04-08, SPARK WES2 and WES3 Outliers by Counts and Coverage</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2022-04-28-post-0082.html" class="nav-list-link">22-04-28, SPARK Mosaic Index</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2022-05-03-post-0083.html" class="nav-list-link">22-05-03, SPARK IGV HC List Screenshots, 2022</a></li><li class="nav-list-item "><a href="/docs/archive_spark/2022-05-20-post-0088.html" class="nav-list-link">22-05-20, SPARK Post-Hoc Cohort, IGV, and iWES Filter Lists</a></li></ul></li><li class="nav-list-item"><a href="#" class="nav-list-expander" aria-label="toggle links in archive_amplicon category"> <svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg> </a><a href="/docs/archive_amplicon/" class="nav-list-link">archive_amplicon</a><ul class="nav-list "><li class="nav-list-item "><a href="/docs/archive_amplicon/2022-03-07-post-0075.html" class="nav-list-link">Shotgun Sequencing of Amplicon Sequence, 220305</a></li><li class="nav-list-item "><a href="/docs/archive_amplicon/2022-04-06-post-0078.html" class="nav-list-link">Shotgun Sequencing of Amplicon Sequence, 220406</a></li><li class="nav-list-item "><a href="/docs/archive_amplicon/2022-05-18-post-0085.html" class="nav-list-link">Example of HDR Processing via BWA Alignment</a></li></ul></li><li class="nav-list-item"><a href="#" class="nav-list-expander" aria-label="toggle links in archive_ssc category"> <svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg> </a><a href="/docs/archive_ssc/" class="nav-list-link">archive_ssc</a><ul class="nav-list "><li class="nav-list-item "><a href="/docs/archive_ssc/2020-10-09-post-0002.html" class="nav-list-link">SSC Final Validation Tables with Annotations</a></li></ul></li><li class="nav-list-item"><a href="#" class="nav-list-expander" aria-label="toggle links in archive_sc_tbr1 category"> <svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg> </a><a href="/docs/archive_sc_tbr1/" class="nav-list-link">archive_sc_tbr1</a><ul class="nav-list "><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-05-07-post-0008.html" class="nav-list-link">Single Cell TBR1 Preliminary Evaluations</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-05-09-post-0010.html" class="nav-list-link">Single Cell TBR1 Coembedding Tests</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-05-11-post-0011.html" class="nav-list-link">Single Cell TBR1 ENCODE DHS Index Tests</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-05-27-post-0013.html" class="nav-list-link">Single Cell TBR1 ENCODE DHS Index Final UMAP and Clusters</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-05-28-post-0014.html" class="nav-list-link">Single Cell TBR1 Cluster DA</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-06-10-post-0022.html" class="nav-list-link">Single Cell TBR1 Cicero/Monocle3 with TSS > 2.7</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-07-09-post-0028.html" class="nav-list-link">Single Cell TBR1 ChromVAR with TSS > 2.7</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-07-12-post-0029.html" class="nav-list-link">Single Cell TBR1 ChromVAR with TSS > 1.5</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-07-18-post-0033.html" class="nav-list-link">Single Cell TBR1 BioCluster/Genotype/Line/Time DA with TSS > 2.7</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-07-19-post-0034.html" class="nav-list-link">Single Cell TBR1 Misc Figures - ChromVAR Compare, FDR Hists, 3D</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-07-26-post-0035.html" class="nav-list-link">Single Cell TBR1 BioCluster/Genotype/Line/Time DA with TSS > 2.7, Filtering Ambiguous Groups</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-07-29-post-0040.html" class="nav-list-link">Single Cell TBR1 Bigwig Tracks</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-08-12-post-0045.html" class="nav-list-link">Single Cell TBR1 Cistopic Boxplots and UMAPs</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-08-12-post-0046.html" class="nav-list-link">Single Cell TBR1 ChIP-Seq Peak DA</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2021-11-28-post-0065.html" class="nav-list-link">Single Cell TBR1 Pseudo-Bulk Depth Tests</a></li><li class="nav-list-item "><a href="/docs/archive_sc_tbr1/2022-05-18-post-0087.html" class="nav-list-link">SRA Download Info for 'TBR1 Regulates Autism Risk Genes in the Developing Neocortex'</a></li></ul></li><li class="nav-list-item"><a href="#" class="nav-list-expander" aria-label="toggle links in archive_misc category"> <svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg> </a><a href="/docs/archive_misc/" class="nav-list-link">archive_misc</a><ul class="nav-list "><li class="nav-list-item "><a href="/docs/archive_misc/2020-10-09-post-0003.html" class="nav-list-link">Two Week Update: 2020-10-09</a></li><li class="nav-list-item "><a href="/docs/archive_misc/2020-11-09-post-0004.html" class="nav-list-link">Two Week Update: 2020-11-09</a></li><li class="nav-list-item "><a href="/docs/archive_misc/2021-05-05-post-0007.html" class="nav-list-link">NextSeq Fidelity Evaluation</a></li><li class="nav-list-item "><a href="/docs/archive_misc/2021-11-04-post-0064.html" class="nav-list-link">Contamination References</a></li><li class="nav-list-item "><a href="/docs/archive_misc/2022-03-01-post-0074.html" class="nav-list-link">OneDrive Upload and Download with Rclone</a></li><li class="nav-list-item "><a href="/docs/archive_misc/2022-04-21-post-0081.html" class="nav-list-link">scRNA-seq and scATAC-seq Data Analysis DREAM Challenge - Dataset 2 Data Generation</a></li><li class="nav-list-item "><a href="/docs/archive_misc/2022-05-18-post-0084.html" class="nav-list-link">Celsee and Docker Info</a></li><li class="nav-list-item "><a href="/docs/archive_misc/2020-10-07-post-0001.html" class="nav-list-link">Example</a></li><li class="nav-list-item "><a href="/docs/archive_misc/2022-05-20-post-0089.html" class="nav-list-link">Final O'Roak Lab Data Locations</a></li></ul></li></ul> </nav> <footer class="site-footer"> This site uses <a href="https://github.com/just-the-docs/just-the-docs">Just the Docs</a>, a documentation theme for Jekyll. </footer> </div> <div class="main" id="top"> <div id="main-header" class="main-header"> <div class="search"> <div class="search-input-wrap"> <input type="text" id="search-input" class="search-input" tabindex="0" placeholder="Search ARSN Work Blog" aria-label="Search ARSN Work Blog" autocomplete="off"> <label for="search-input" class="search-label"><svg viewBox="0 0 24 24" class="search-icon"><use xlink:href="#svg-search"></use></svg></label> </div> <div id="search-results" class="search-results"></div> </div> <nav aria-label="Auxiliary" class="aux-nav"> <ul class="aux-nav-list"> <li class="aux-nav-list-item"> <a href="//github.com/just-the-docs/just-the-docs" class="site-button" > Just the Docs on GitHub </a> </li> </ul> </nav> </div> <div id="main-content-wrap" class="main-content-wrap"> <nav aria-label="Breadcrumb" class="breadcrumb-nav"> <ol class="breadcrumb-nav-list"> <li class="breadcrumb-nav-list-item"><a href="/docs/archive_spark/">archive_spark</a></li> <li class="breadcrumb-nav-list-item"><span>21-11-30, SPARK SSC List Formatting to Current SPARK</span></li> </ol> </nav> <div id="main-content" class="main-content" role="main"> <p>The SSC variant calls are taken from this Box file, <code class="language-plaintext highlighter-rouge">https://ohsu.box.com/s/dqm3garunt0xobvgdz41onpexi87yqin</code>, on this path, <code class="language-plaintext highlighter-rouge">All Files/kruppd/Writing/SSC somatic pilot/data/Data Used for Mosaic Paper/High Confidence Variants/HighConfidence Variants Collapsed_2.csv</code>. This appears to be a similar version of Table S5 from the Krupp 2017 paper with the columns with the statistics which are excluded from the paper. Unlike the table in the paper, this also collapses sites in overlapping genes into a single entry which is more in line with the SPARK lists where genes are combined in one entry separated by commas. All SSC columns are transformed to match the columns for the SPARK list.</p> <p>Here is the transformed SSC list. It has the same columns as the SPARK lists and has extra columns representing the SSC specific entries. The two important extra columns to consider that are unique to the SSC list are column 90 and 91 which are the ‘FILTER’ and ‘OLD_TYPE’ respectively. ‘FILTER’ is TRUE or FALSE. TRUE entries should be filtered from the analysis and their designation is explained below. In general they are calls that do not meet our criteria after reannotation in hg38 versus hg19. ‘OLD_TYPE’ designates whether the call comes from a quad or trio family which may or may not be important for us.</p> <p>SSC list in SPARK list format: <br /><a href="https://www.dropbox.com/s/hkjkeqibfk2nkyw/ssc.transformed.txt?dl=0">SSC List in SPARK List Format</a></p> <p>Markdown showing how the list was modified: <br /><a href="https://www.dropbox.com/s/j1flcekwsrp6s03/17_ssc_list.html?dl=0">17 SSC List Formatting to Current SPARK</a></p> <p>The column ‘FILTER’ designates calls to remove from downstream analysis. ‘TRUE’ calls should be removed and are given if:</p> <div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>- there are no FRACTION calls made (not labeled GERMLINE or MOSAIC, labeled 'NA')
- the variant falls in a region that is annotated in either a non-exonic, super duplicate, or trf region
- the variant does not have a predicted functional mutation type
</code></pre></div></div> <p>The following columns were modified in the transformation:</p> <div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>Coordinates were changed via LiftOver with 100% of the sites found going from hg19 to hg38.
All genomic and functional annotations were redone with AnnoVAR for hg38.
FRACTION_MORE is derived from FRACTION with name changes to match the SPARK list ("GERMLINE_C" is changed to "GERMLINE_CHILD").
ACC_DIST and DON_DIST are the same value. The SSC list keeps only the closest wheras the SPARK list keeps both separate.
COMPLEX_EVENT is descriptive in the SSC list and binary in the SPARK list.
NUM_FAMILY_FOUND and filteredCohortCount represent cohort counts before and after filtering. In the SSC list, only the final count is kept so these are the same for the SSC list.
HGNCsymbol_to_ENSID and RefSeq_to_ENSID are the ENSEMBL ID's determined from the hg38 annotations.
</code></pre></div></div> <p>The following columns were dropped and all set to ‘NA’:</p> <div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>There are no numerical sample IDs in the SSC list.
SSC uses combined family stats from the variant callers whereas the SPARK list keeps them by individual so these are dropped.
Mismatch stats for raw mismatch counts, 'other' read counts, and the means are not present in the SSC list.
gnomAD allele fractions are not present in the earlier SSC table.
The only error columns in the SSC list are the calculated rates.
FRACTION_OLD is a legacy column in the SPARK list and does not have meaning in the SSC list.
The splice site transcript IDs are not present just the distances.
</code></pre></div></div> <p>The following extra columns specific to the SSC list were added at the end. These are potentially important for analysis:</p> <div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>FILTER - TRUE or FALSE and represent variants that should be filtered out after the transformation of hg19 to hg38
OLD_TYPE - TRIO or QUAD
OLD_SEGDUP, OLD_RMSK, OLD_TRF, OLD_FUNC_TYPE, OLD_TRANS_REF, OLD_CLASS_REF, OLD_GENE_REF - these are the original hg19 AnnoVAR annotations
</code></pre></div></div> <p>The following SSC specific columns were kept and added at the end. These can likely be ignored. I’ve tried to describe them as best as possible without knowing their exact provenance:</p> <div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>OLD_STUDY - COHORT or PILOT24 or PILOT400
OLD_PUBLISHED - IOSSIFOV_2014 or KRUMM_2015
OLD_AF_ESP, OLD_AF_1KG - allele fractions from other databases
OLD_PCTILE_CDS_AUTO_45X - 5% percentiles of CDS AUTO 45x
OLD_FORMAT_VARSCAN, OLD_FORMAT_LOFREQ, OLD_FORMAT_SMOC, OLD_VARSCAN_232_FA, OLD_LOFREQ_211_FA, OLD_SMOC_FA, OLD_VARSCAN_232_MO, OLD_LOFREQ_211_MO, OLD_SMOC_MO, OLD_VARSCAN_232_PRO, OLD_LOFREQ_211_PRO, OLD_SMOC_PRO, OLD_VARSCAN_232_SIB, OLD_LOFREQ_211_SIB, OLD_SMOC_SIB - family-based variant calling output
OLD_DIST_TO_PREV, OLD_DIST_TO_NEXT, OLD_PERSON_PREV, OLD_PERSON_NEXT - closest call and person	
OLD_QC_STATUS - at this stage they are all 'PASS', unknown criteria
OLD_COHORT_DP, OLD_COHORT_DPALT, OLD_P400_COUNT_MOS, OLD_P400_COUNT_GERM, OLD_COHORT_COUNT_MOS, OLD_COHORT_COUNT_GERM, OLD_COHORT_MAX_AD, OLD_P400_COUNT_ALL - various counts within the cohort subsets
OLD_AF_FA, OLD_AF_MO, OLD_AF_PRO, OLD_AF_SIB, OLD_CALLED_FA, OLD_CALLED_MO, OLD_CALLED_PRO, OLD_CALLED_SIB, OLD_DETECT_FA, OLD_DETECT_MO, OLD_DETECT_PRO, OLD_DETECT_SIB, OLD_PHET_MAIN, OLD_SITE_KEY, OLD_PARENTAL, OLD_GONADAL, OLD_CALLED_VARSCAN, OLD_CALLED_LOFREQ, OLD_CALLED_SMOC - these are all redundant derived columns
</code></pre></div></div> </div> </div> <div class="search-overlay"></div> </div> </body> </html>
