<!DOCTYPE html>
<html lang="en"><head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1"><!-- Begin Jekyll SEO tag v2.6.1 -->
<title>SPARK Mosaic Index, July 27 | ARSN Lab Notebook</title>
<meta name="generator" content="Jekyll v4.1.1" />
<meta property="og:title" content="SPARK Mosaic Index, July 27" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="Final Mosaic Lists (July 27th): Simplex Quads, for Burden Analysis Simplex Quads, High Confidence Calls Simplex Trios, for Burden Analysis Simplex Trios, High Confidence Calls Multiplex Quads, for Burden Analysis Multiplex Quads, High Confidence Calls" />
<meta property="og:description" content="Final Mosaic Lists (July 27th): Simplex Quads, for Burden Analysis Simplex Quads, High Confidence Calls Simplex Trios, for Burden Analysis Simplex Trios, High Confidence Calls Multiplex Quads, for Burden Analysis Multiplex Quads, High Confidence Calls" />
<link rel="canonical" href="http://localhost:4000/2021/07/27/post-0039.html" />
<meta property="og:url" content="http://localhost:4000/2021/07/27/post-0039.html" />
<meta property="og:site_name" content="ARSN Lab Notebook" />
<meta property="og:type" content="article" />
<meta property="article:published_time" content="2021-07-27T00:00:00-07:00" />
<script type="application/ld+json">
{"headline":"SPARK Mosaic Index, July 27","dateModified":"2021-07-27T00:00:00-07:00","datePublished":"2021-07-27T00:00:00-07:00","url":"http://localhost:4000/2021/07/27/post-0039.html","mainEntityOfPage":{"@type":"WebPage","@id":"http://localhost:4000/2021/07/27/post-0039.html"},"description":"Final Mosaic Lists (July 27th): Simplex Quads, for Burden Analysis Simplex Quads, High Confidence Calls Simplex Trios, for Burden Analysis Simplex Trios, High Confidence Calls Multiplex Quads, for Burden Analysis Multiplex Quads, High Confidence Calls","@type":"BlogPosting","@context":"https://schema.org"}</script>
<!-- End Jekyll SEO tag -->
<link rel="stylesheet" href="/assets/main.css"><link type="application/atom+xml" rel="alternate" href="http://localhost:4000/feed.xml" title="ARSN Lab Notebook" /></head>
<body><header class="site-header" role="banner">

  <div class="wrapper"><a class="site-title" rel="author" href="/">ARSN Lab Notebook</a><nav class="site-nav">
        <input type="checkbox" id="nav-trigger" class="nav-trigger" />
        <label for="nav-trigger">
          <span class="menu-icon">
            <svg viewBox="0 0 18 15" width="18px" height="15px">
              <path d="M18,1.484c0,0.82-0.665,1.484-1.484,1.484H1.484C0.665,2.969,0,2.304,0,1.484l0,0C0,0.665,0.665,0,1.484,0 h15.032C17.335,0,18,0.665,18,1.484L18,1.484z M18,7.516C18,8.335,17.335,9,16.516,9H1.484C0.665,9,0,8.335,0,7.516l0,0 c0-0.82,0.665-1.484,1.484-1.484h15.032C17.335,6.031,18,6.696,18,7.516L18,7.516z M18,13.516C18,14.335,17.335,15,16.516,15H1.484 C0.665,15,0,14.335,0,13.516l0,0c0-0.82,0.665-1.483,1.484-1.483h15.032C17.335,12.031,18,12.695,18,13.516L18,13.516z"/>
            </svg>
          </span>
        </label>

        <div class="trigger"><a class="page-link" href="/about/">About</a></div>
      </nav></div>
</header>
<main class="page-content" aria-label="Content">
      <div class="wrapper">
        <article class="post h-entry" itemscope itemtype="http://schema.org/BlogPosting">

  <header class="post-header">
    <h1 class="post-title p-name" itemprop="name headline">SPARK Mosaic Index, July 27</h1>
    <p class="post-meta">
      <time class="dt-published" datetime="2021-07-27T00:00:00-07:00" itemprop="datePublished">Jul 27, 2021
      </time></p>
  </header>

  <div class="post-content e-content" itemprop="articleBody">
    <p>Final Mosaic Lists (July 27th):
<br /><a href="https://www.dropbox.com/s/2v0su9gdz25yv5w/quadsimplex.burden.txt?dl=0">Simplex Quads, for Burden Analysis</a>
<br /><a href="https://www.dropbox.com/s/dphpofhrh7xa84z/quadsimplex.highconfidence.txt?dl=0">Simplex Quads, High Confidence Calls</a>
<br /><a href="https://www.dropbox.com/s/8rlzsphugatls37/triosimplex.burden.txt?dl=0">Simplex Trios, for Burden Analysis</a>
<br /><a href="https://www.dropbox.com/s/lrdb5trc75ohgx8/triosimplex.highconfidence.txt?dl=0">Simplex Trios, High Confidence Calls</a>
<br /><a href="https://www.dropbox.com/s/luyljm77ci1hu1k/quadmultiplex.burden.txt?dl=0">Multiplex Quads, for Burden Analysis</a>
<br /><a href="https://www.dropbox.com/s/cn3x5sfm30eq04a/quadmultiplex.highconfidence.txt?dl=0">Multiplex Quads, High Confidence Calls</a></p>

<p>Markdowns:
<br /><a href="https://www.dropbox.com/s/o2qqos8qembl586/01_qs.organize_prefilter.html?dl=0">01_QS, Organizing and Prefiltering</a>
<br /><a href="https://www.dropbox.com/s/og3913n31a8z27a/01_ts.organize_prefilter.html?dl=0">01_TS, Organizing and Prefiltering</a>
<br /><a href="https://www.dropbox.com/s/jpmql5snxqr2uyv/01_qm.organize_prefilter.html?dl=0">01_QM, Organizing and Prefiltering</a>
<br /><a href="https://www.dropbox.com/s/8y75fw081y8ddo5/02_qs.outlier_detection.html?dl=0">02_QS, Outlier Detection</a>
<br /><a href="https://www.dropbox.com/s/62cr2hmihv0phas/02_ts.outlier_detection.html?dl=0">02_TS, Outlier Detection</a>
<br /><a href="https://www.dropbox.com/s/bxvqbgd8adjlwj2/02_qm.outlier_detection.html?dl=0">02_QM, Outlier Detection</a>
<br /><a href="https://www.dropbox.com/s/vqx366c95usmanb/03_qs.final_tables.html?dl=0">03_QS, Final Table</a>
<br /><a href="https://www.dropbox.com/s/06esgmqu3fux0h7/03_ts.final_tables.html?dl=0">03_TS, Final Table</a>
<br /><a href="https://www.dropbox.com/s/tn6njzth0v7v11u/03_qm.final_tables.html?dl=0">03_QM, Final Table</a>
<br /><a href="https://www.dropbox.com/s/6s5dhposyznatnl/04_qs_highconf_allelefreqs_01.html?dl=0">04, High Confidence Simplex Quad Allele Frequencies</a>
<br /><a href="https://www.dropbox.com/s/mbcztsj0j9th4yx/05_highconf_splicedist_01.html?dl=0">05 High Confidence Simplex Quad Splice Distances</a>
<br /><a href="https://www.dropbox.com/s/zzd6ikgyh4g8bzp/05_highconf_splicedist_02.html?dl=0">05 High Confidence All Lists Splice Distances</a>
<br /><a href="https://www.dropbox.com/s/5l34qb95m88tuvt/06_burden_autosomal_01.html?dl=0">06 Burden Tables Autosomal</a>
<br /><a href="https://www.dropbox.com/s/879g61jul2vnryi/07_burden_genesets_01.html?dl=0">07_01 Burden Gene Sets</a>
<br /><a href="https://www.dropbox.com/s/zwvp4yolro6gdvp/07_burden_genesets_02.html?dl=0">07_02 Burden Gene Sets</a></p>

<p>Mosaic Lists, Full Non-Analysis Sets (June):
<br /><a href="https://www.dropbox.com/s/tyczipwxkch4a1f/quadsimplex_mosaics.8reads_0.001popAF_4cohort_3alt_exonic.jun3.txt?dl=0">Simplex Quads w/ Loose Filter</a>
<br /><a href="https://www.dropbox.com/s/407gqs049ukcdcf/quadmultiplex_mosaics.8reads_0.001popAF_4cohort_3alt_exonic.jun3.txt?dl=0">Multiplex Quads w/ Loose Filter</a>
<br /><a href="https://www.dropbox.com/s/2rs72k69tkepcrz/triosimplex_mosaics.8reads_0.001popAF_4cohort_3alt_exonic.jun3.txt?dl=0">Simplex Trios w/ Loose Filter</a>
<br /><a href="https://www.dropbox.com/s/okrg9g1ug10788i/quadsimplex_mosaics.8reads_0.001popAF_4cohort_3alt_exonic.testfilter.jun3.txt?dl=0">Simplex Quads w/ Strict Test Filter</a>
<br /><a href="https://www.dropbox.com/s/xabrs2f030xlz49/quadmultiplex_mosaics.8reads_0.001popAF_4cohort_3alt_exonic.testfilter.jun3.txt?dl=0">Multiplex Quads w/ Strict Test Filter</a>
<br /><a href="https://www.dropbox.com/s/dv4ajax8bqbwfcu/triosimplex_mosaics.8reads_0.001popAF_4cohort_3alt_exonic.testfilter.jun3.txt?dl=0">Simplex Trios w/ Strict Test Filter</a>
<br /><a href="https://www.dropbox.com/s/fjuuw3q7cekci7f/quadsimplex_mosaics.8reads_0.001popAF_4cohort_3alt_exonic.testfilter_noGermlineParental.jun3.txt?dl=0">Simplex Quads w/ Strict Test Filter, no Germline Parental</a>
<br /><a href="https://www.dropbox.com/s/vev0qhne7n0gmx4/quadmultiplex_mosaics.8reads_0.001popAF_4cohort_3alt_exonic.testfilter_noGermlineParental.jun3.txt?dl=0">Multiplex Quads w/ Strict Test Filter, no Germline Parental</a>
<br /><a href="https://www.dropbox.com/s/vmhwb1a307p92gs/triosimplex_mosaics.8reads_0.001popAF_4cohort_3alt_exonic.testfilter_noGermlineParental.jun3.txt?dl=0">Simplex Trios w/ Strict Test Filter, no Germline Parental</a></p>

<p>SPARK Annotations Used, 20190820:
<br /><a href="https://www.dropbox.com/s/gg5ewe0bbqhjimw/SPARK.27K.mastertable.20190820.txt?dl=0">SPARK Annotations</a></p>

<p>Links to List of Gene Names with Alternate Names:
<br /><a href="https://www.dropbox.com/s/4jcjuaz8c3dhbtf/Essential.alt.genes?dl=0">Essential.alt.genes</a>
<br /><a href="https://www.dropbox.com/s/jbrpq5fb8qz16ri/Exac.Intol.alt.genes?dl=0">Exac.Intol.alt.genes</a>
<br /><a href="https://www.dropbox.com/s/yjw7r0fwh3lxqil/Exac.Mid.alt.genes?dl=0">Exac.Mid.alt.genes</a>
<br /><a href="https://www.dropbox.com/s/443h1423awxmxza/Exac.Tol.alt.genes?dl=0">Exac.Tol.alt.genes</a>
<br /><a href="https://www.dropbox.com/s/qbptliwg3irwnod/FUSIL.alt.genes?dl=0">FUSIL.alt.genes</a></p>

<p>Joint Coverage:
<br /><a href="https://www.dropbox.com/s/xp96tughn8vhjzu/jointcoverage.2.simplexquads.txt?dl=0">Genomic Joint Coverage, Simplex Quads</a>
<br /><a href="https://www.dropbox.com/s/dm5ppo3fn2q0vrv/jointcoverage.2.multiplexquads.txt?dl=0">Genomic Joint Coverage, Multiplex Quads</a>
<br /><a href="https://www.dropbox.com/s/pfkw1v3z7rs4ugy/jointcoverage.2.simplextrios.txt?dl=0">Genomic Joint Coverage, Simplex Trios</a></p>

<p>IGV Shots:
<br /><a href="https://www.dropbox.com/sh/nj8wglswrxj7lik/AADvVXrVXapW3lukFMppeuLAa?dl=0">IGV Shots of High Confidence Sites</a></p>

<p>Here are descriptions of all the columns in the table:
<br />1/CHR, 2/START, 3/STOP, 4/REF, 5/ALT - general variant information
<br />6/FAMILY, 7/SAMPLE, 8/FRACTION - SF family ID, SP sample ID, putative mosaic annotation
<br />9/DP_MAIN, 10/DPALT_MAIN, 11/AF_MAIN - the depth of reads, depth of alt reads, and AF for the main person of the family
<br />12/LOFREQ_STATS_NAMES, 13/LOFREQ_STATS, 14/MPUP_STATS_NAMES, 15/MPUP_STATS, 16/VARSCAN_STATS_NAMES, 17/VARSCAN_STATS - stats from the three variant callers with their descriptor fields
<br />18/MISMATCH_REF_READS, 19/MISMATCH_ALT_READS, 20/MISMATCH_OTHER_READS, 21/MISMATCH_SPAN_DELS, 22/MISMATCH_MAPQ_REF_MEAN, 23/MISMATCH_MAPQ_ALT_MEAN, 24/MISMATCH_MAPQ_OTHER_MEAN, 25/MISMATCH_MAPQ_REF_MEDIAN, 26/MISMATCH_MAPQ_ALT_MEDIAN, 27/MISMATCH_MAPQ_OTHER_MEDIAN, 28/MISMATCH_MAPQ_REF_MODE, 29/MISMATCH_MAPQ_ALT_MODE, 30/MISMATCH_MAPQ_OTHER_MODE, 31/MISMATCH_XM_REF_MEAN, 32/MISMATCH_XM_ALT_MEAN, 33/MISMATCH_XM_OTHER_MEAN, 34/MISMATCH_XM_REF_MEDIAN, 35/MISMATCH_XM_ALT_MEDIAN, 36/MISMATCH_XM_OTHER_MEDIAN, 37/MISMATCH_XM_REF_MODE, 38/MISMATCH_XM_ALT_MODE, 39/MISMATCH_XM_OTHER_MODE - large collection of stats concerning mismatches across the region, only MISMATCH_XM_ALT_MEDIAN gets used downstream for another derived column
<br />40/DP_FA, 41/DP_MO, 42/DP_PRO, 43/DP_SIB, 44/DPALT_FA, 45/DPALT_MO, 46/DPALT_PRO, 47/DPALT_SIB - depths across all family members
<br />48/AF_EXAC, 49/AF_gnomAD - AFs from ExAC v0.3 and gnomAD 2.1
<br />50/ERROR_VAR, 51/ERROR_ALL, 52/ERROR_VARMAX, 53/ERROR_RATE - error in the region calculated over a random subset of 1000 samples, only rate gets used downstream for another derived column
<br />54/NUM_FAMILY_FOUND - number of families the variant is found before filtering
<br />55/PHET_FA, 56/PHET_MO, 57/PHET_PRO, 58/PHET_SIB - pvalue of exact test given the depth and the depth of the alt read in each family member testing if it is less than 0.5 (heterogeneous)
<br />59/PERSON - identity of the person
<br />60/DPALT_BIN - binary column derived from DPALT_MAIN used for SCORE_400_RB3
<br />61/ERROR_BIN_3 - categorical column (low, medium, high) derived from ERROR_RATE used for SCORE_400_RB3
<br />62/AF_BIN_3 - categorical column (low, medium, high) derived from AF_MAIN used for SCORE_400_RB3
<br />63/REF_CG - categorical column (C or G) derived from REF used for SCORE_400_RB3
<br />64/XM_ALT_1 - binary column derived from MISMATCH_XM_ALT_MEDIAN used for SCORE_400_RB3
<br />65/SCORE_400_RB3 - score from logistic regression model that takes into account many of the other columns
<br />66/genomicSuperDupes, 67/rmsk, 68/simpleRepeat, 69/mutationType, 70/exonicAnnotation, 71/variantFunctionType, 72/variantFunctionGene - annotations with annovar-2018-06-15
<br />73/filteredCohortCount - number of families the variant is found after preliminary filtering
<br />74/AF_L90CI_AC_MAIN, 75/AF_U90CI_AC_MAIN - upper and lower 90% confidence intervals via Agresti-Coull method
<br />76/FRACTION_ORIGINAL - old, erroneous annotations of the site for posterity, see May 9 update
<br />77/FRACTION_MORE - a subset of column 8, FRACTION, with further granularity, see June 3 update
<br />78/PSAME_FAxPRO, 79/PSAME_FAxSIB, 80/PSAME_MOxPRO, 81/PSAME_MOxSIB - Fisherâ€™s exact test comparing depths and alt reads from parents versus children for the purposes of testing a transmission event.
Final tables also contain:
<br />82/MIN_DP - The minimum depth of all family members.
<br />83/ACC_DIST, 84/DON_DIST, 85/ACC_TRANSCRIPT, 86/DON_TRANSCRIPT - The distances in basepairs to an acceptor or donor site and which transcript from which this distance is derived.
<br />87/COMPLEX_EVENT - All FALSE in final tables representing collapsed complex events.
<br />88/HGNCsymbol_to_ENSID, 89/RefSeq_to_ENSID - ENSEMBL IDs derived from both HGNC symbosl and RefSeq</p>

  </div><a class="u-url" href="/2021/07/27/post-0039.html" hidden></a>
</article>

      </div>
    </main><footer class="site-footer h-card">
  <data class="u-url" href="/"></data>

  <div class="wrapper">

    <h2 class="footer-heading">ARSN Lab Notebook</h2>

    <div class="footer-col-wrapper">
      <div class="footer-col footer-col-1">
        <ul class="contact-list">
          <li class="p-name">ARSN Lab Notebook</li><li><a class="u-email" href="mailto:nishidaa@ohsu.edu">nishidaa@ohsu.edu</a></li></ul>
      </div>

      <div class="footer-col footer-col-2"><ul class="social-media-list"><li><a href="https://github.com/arsnishida"><svg class="svg-icon"><use xlink:href="/assets/minima-social-icons.svg#github"></use></svg> <span class="username">arsnishida</span></a></li></ul>
</div>

      <div class="footer-col footer-col-3">
        <p>ARSN Lab notebook.</p>
      </div>
    </div>

  </div>

</footer>
</body>

</html>
