---
layout: post
title:  "SPARK Mosaic Index, July 27"
date:   2021-07-27
tags: SPARK INDEX
---

Final Mosaic Lists (July 27th):
<br>[Simplex Quads, for Burden Analysis](https://www.dropbox.com/s/2v0su9gdz25yv5w/quadsimplex.burden.txt?dl=0)
<br>[Simplex Quads, High Confidence Calls](https://www.dropbox.com/s/dphpofhrh7xa84z/quadsimplex.highconfidence.txt?dl=0)
<br>[Simplex Trios, for Burden Analysis](https://www.dropbox.com/s/8rlzsphugatls37/triosimplex.burden.txt?dl=0)
<br>[Simplex Trios, High Confidence Calls](https://www.dropbox.com/s/lrdb5trc75ohgx8/triosimplex.highconfidence.txt?dl=0)
<br>[Multiplex Quads, for Burden Analysis](https://www.dropbox.com/s/luyljm77ci1hu1k/quadmultiplex.burden.txt?dl=0)
<br>[Multiplex Quads, High Confidence Calls](https://www.dropbox.com/s/cn3x5sfm30eq04a/quadmultiplex.highconfidence.txt?dl=0)

Markdowns:
<br>[01_QS, Organizing and Prefiltering](https://www.dropbox.com/s/o2qqos8qembl586/01_qs.organize_prefilter.html?dl=0)
<br>[01_TS, Organizing and Prefiltering](https://www.dropbox.com/s/og3913n31a8z27a/01_ts.organize_prefilter.html?dl=0)
<br>[01_QM, Organizing and Prefiltering](https://www.dropbox.com/s/jpmql5snxqr2uyv/01_qm.organize_prefilter.html?dl=0)
<br>[02_QS, Outlier Detection](https://www.dropbox.com/s/8y75fw081y8ddo5/02_qs.outlier_detection.html?dl=0)
<br>[02_TS, Outlier Detection](https://www.dropbox.com/s/62cr2hmihv0phas/02_ts.outlier_detection.html?dl=0)
<br>[02_QM, Outlier Detection](https://www.dropbox.com/s/bxvqbgd8adjlwj2/02_qm.outlier_detection.html?dl=0)
<br>[03_QS, Final Table](https://www.dropbox.com/s/vqx366c95usmanb/03_qs.final_tables.html?dl=0)
<br>[03_TS, Final Table](https://www.dropbox.com/s/06esgmqu3fux0h7/03_ts.final_tables.html?dl=0)
<br>[03_QM, Final Table](https://www.dropbox.com/s/tn6njzth0v7v11u/03_qm.final_tables.html?dl=0)
<br>[04, High Confidence Simplex Quad Allele Frequencies](https://www.dropbox.com/s/6s5dhposyznatnl/04_qs_highconf_allelefreqs_01.html?dl=0)
<br>[05 High Confidence Simplex Quad Splice Distances](https://www.dropbox.com/s/mbcztsj0j9th4yx/05_highconf_splicedist_01.html?dl=0)
<br>[05 High Confidence All Lists Splice Distances](https://www.dropbox.com/s/zzd6ikgyh4g8bzp/05_highconf_splicedist_02.html?dl=0)
<br>[06 Burden Tables Autosomal](https://www.dropbox.com/s/5l34qb95m88tuvt/06_burden_autosomal_01.html?dl=0)

Mosaic Lists, Full Non-Analysis Sets (June):
<br>[Simplex Quads w/ Loose Filter](https://www.dropbox.com/s/tyczipwxkch4a1f/quadsimplex_mosaics.8reads_0.001popAF_4cohort_3alt_exonic.jun3.txt?dl=0)
<br>[Multiplex Quads w/ Loose Filter](https://www.dropbox.com/s/407gqs049ukcdcf/quadmultiplex_mosaics.8reads_0.001popAF_4cohort_3alt_exonic.jun3.txt?dl=0)
<br>[Simplex Trios w/ Loose Filter](https://www.dropbox.com/s/2rs72k69tkepcrz/triosimplex_mosaics.8reads_0.001popAF_4cohort_3alt_exonic.jun3.txt?dl=0)
<br>[Simplex Quads w/ Strict Test Filter](https://www.dropbox.com/s/okrg9g1ug10788i/quadsimplex_mosaics.8reads_0.001popAF_4cohort_3alt_exonic.testfilter.jun3.txt?dl=0)
<br>[Multiplex Quads w/ Strict Test Filter](https://www.dropbox.com/s/xabrs2f030xlz49/quadmultiplex_mosaics.8reads_0.001popAF_4cohort_3alt_exonic.testfilter.jun3.txt?dl=0)
<br>[Simplex Trios w/ Strict Test Filter](https://www.dropbox.com/s/dv4ajax8bqbwfcu/triosimplex_mosaics.8reads_0.001popAF_4cohort_3alt_exonic.testfilter.jun3.txt?dl=0)
<br>[Simplex Quads w/ Strict Test Filter, no Germline Parental](https://www.dropbox.com/s/fjuuw3q7cekci7f/quadsimplex_mosaics.8reads_0.001popAF_4cohort_3alt_exonic.testfilter_noGermlineParental.jun3.txt?dl=0)
<br>[Multiplex Quads w/ Strict Test Filter, no Germline Parental](https://www.dropbox.com/s/vev0qhne7n0gmx4/quadmultiplex_mosaics.8reads_0.001popAF_4cohort_3alt_exonic.testfilter_noGermlineParental.jun3.txt?dl=0)
<br>[Simplex Trios w/ Strict Test Filter, no Germline Parental](https://www.dropbox.com/s/vmhwb1a307p92gs/triosimplex_mosaics.8reads_0.001popAF_4cohort_3alt_exonic.testfilter_noGermlineParental.jun3.txt?dl=0)

SPARK Annotations Used, 20190820:
<br>[SPARK Annotations](https://www.dropbox.com/s/gg5ewe0bbqhjimw/SPARK.27K.mastertable.20190820.txt?dl=0)

Links to List of Gene Names with Alternate Names:
<br>[Essential.alt.genes](https://www.dropbox.com/s/4jcjuaz8c3dhbtf/Essential.alt.genes?dl=0)
<br>[Exac.Intol.alt.genes](https://www.dropbox.com/s/jbrpq5fb8qz16ri/Exac.Intol.alt.genes?dl=0)
<br>[Exac.Mid.alt.genes](https://www.dropbox.com/s/yjw7r0fwh3lxqil/Exac.Mid.alt.genes?dl=0)
<br>[Exac.Tol.alt.genes](https://www.dropbox.com/s/443h1423awxmxza/Exac.Tol.alt.genes?dl=0)
<br>[FUSIL.alt.genes](https://www.dropbox.com/s/qbptliwg3irwnod/FUSIL.alt.genes?dl=0)

Joint Coverage:
<br>[Genomic Joint Coverage, Simplex Quads](https://www.dropbox.com/s/xp96tughn8vhjzu/jointcoverage.2.simplexquads.txt?dl=0)
<br>[Genomic Joint Coverage, Multiplex Quads](https://www.dropbox.com/s/dm5ppo3fn2q0vrv/jointcoverage.2.multiplexquads.txt?dl=0)
<br>[Genomic Joint Coverage, Simplex Trios](https://www.dropbox.com/s/pfkw1v3z7rs4ugy/jointcoverage.2.simplextrios.txt?dl=0)

Here are descriptions of all the columns in the table:
<br>1/CHR, 2/START, 3/STOP, 4/REF, 5/ALT - general variant information
<br>6/FAMILY, 7/SAMPLE, 8/FRACTION - SF family ID, SP sample ID, putative mosaic annotation
<br>9/DP_MAIN, 10/DPALT_MAIN, 11/AF_MAIN - the depth of reads, depth of alt reads, and AF for the main person of the family
<br>12/LOFREQ_STATS_NAMES, 13/LOFREQ_STATS, 14/MPUP_STATS_NAMES, 15/MPUP_STATS, 16/VARSCAN_STATS_NAMES, 17/VARSCAN_STATS - stats from the three variant callers with their descriptor fields
<br>18/MISMATCH_REF_READS, 19/MISMATCH_ALT_READS, 20/MISMATCH_OTHER_READS, 21/MISMATCH_SPAN_DELS, 22/MISMATCH_MAPQ_REF_MEAN, 23/MISMATCH_MAPQ_ALT_MEAN, 24/MISMATCH_MAPQ_OTHER_MEAN, 25/MISMATCH_MAPQ_REF_MEDIAN, 26/MISMATCH_MAPQ_ALT_MEDIAN, 27/MISMATCH_MAPQ_OTHER_MEDIAN, 28/MISMATCH_MAPQ_REF_MODE, 29/MISMATCH_MAPQ_ALT_MODE, 30/MISMATCH_MAPQ_OTHER_MODE, 31/MISMATCH_XM_REF_MEAN, 32/MISMATCH_XM_ALT_MEAN, 33/MISMATCH_XM_OTHER_MEAN, 34/MISMATCH_XM_REF_MEDIAN, 35/MISMATCH_XM_ALT_MEDIAN, 36/MISMATCH_XM_OTHER_MEDIAN, 37/MISMATCH_XM_REF_MODE, 38/MISMATCH_XM_ALT_MODE, 39/MISMATCH_XM_OTHER_MODE - large collection of stats concerning mismatches across the region, only MISMATCH_XM_ALT_MEDIAN gets used downstream for another derived column
<br>40/DP_FA, 41/DP_MO, 42/DP_PRO, 43/DP_SIB, 44/DPALT_FA, 45/DPALT_MO, 46/DPALT_PRO, 47/DPALT_SIB - depths across all family members
<br>48/AF_EXAC, 49/AF_gnomAD - AFs from ExAC v0.3 and gnomAD 2.1
<br>50/ERROR_VAR, 51/ERROR_ALL, 52/ERROR_VARMAX, 53/ERROR_RATE - error in the region calculated over a random subset of 1000 samples, only rate gets used downstream for another derived column
<br>54/NUM_FAMILY_FOUND - number of families the variant is found before filtering
<br>55/PHET_FA, 56/PHET_MO, 57/PHET_PRO, 58/PHET_SIB - pvalue of exact test given the depth and the depth of the alt read in each family member testing if it is less than 0.5 (heterogeneous)
<br>59/PERSON - identity of the person
<br>60/DPALT_BIN - binary column derived from DPALT_MAIN used for SCORE_400_RB3
<br>61/ERROR_BIN_3 - categorical column (low, medium, high) derived from ERROR_RATE used for SCORE_400_RB3
<br>62/AF_BIN_3 - categorical column (low, medium, high) derived from AF_MAIN used for SCORE_400_RB3
<br>63/REF_CG - categorical column (C or G) derived from REF used for SCORE_400_RB3
<br>64/XM_ALT_1 - binary column derived from MISMATCH_XM_ALT_MEDIAN used for SCORE_400_RB3
<br>65/SCORE_400_RB3 - score from logistic regression model that takes into account many of the other columns
<br>66/genomicSuperDupes, 67/rmsk, 68/simpleRepeat, 69/mutationType, 70/exonicAnnotation, 71/variantFunctionType, 72/variantFunctionGene - annotations with annovar-2018-06-15
<br>73/filteredCohortCount - number of families the variant is found after preliminary filtering
<br>74/AF_L90CI_AC_MAIN, 75/AF_U90CI_AC_MAIN - upper and lower 90% confidence intervals via Agresti-Coull method
<br>76/FRACTION_ORIGINAL - old, erroneous annotations of the site for posterity, see May 9 update
<br>77/FRACTION_MORE - a subset of column 8, FRACTION, with further granularity, see June 3 update
<br>78/PSAME_FAxPRO, 79/PSAME_FAxSIB, 80/PSAME_MOxPRO, 81/PSAME_MOxSIB - Fisher's exact test comparing depths and alt reads from parents versus children for the purposes of testing a transmission event.
Final tables also contain:
<br>82/MIN_DP - The minimum depth of all family members.
<br>83/ACC_DIST, 84/DON_DIST, 85/ACC_TRANSCRIPT, 86/DON_TRANSCRIPT - The distances in basepairs to an acceptor or donor site and which transcript from which this distance is derived.
<br>87/COMPLEX_EVENT - All FALSE in final tables representing collapsed complex events.
<br>88/HGNCsymbol_to_ENSID, 89/RefSeq_to_ENSID - ENSEMBL IDs derived from both HGNC symbosl and RefSeq