<!DOCTYPE html>
<html lang="en"><head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1"><!-- Begin Jekyll SEO tag v2.6.1 -->
<title>Single Cell Alignments with no Alt Chromosomes | ARSN Lab Notebook</title>
<meta name="generator" content="Jekyll v4.1.1" />
<meta property="og:title" content="Single Cell Alignments with no Alt Chromosomes" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="Alt chromosomes in our hg38 and mm10 alignments (UCSC description) cause multi-mapping in those regions. BWA flags multi-mapping reads with a MAPQ score of 0 causing them to all be lost downstream in our analysis pipeline. For most single-cell analyses we do not need to consider the alternative chromosomes and they can fall in regions of genes used as markers." />
<meta property="og:description" content="Alt chromosomes in our hg38 and mm10 alignments (UCSC description) cause multi-mapping in those regions. BWA flags multi-mapping reads with a MAPQ score of 0 causing them to all be lost downstream in our analysis pipeline. For most single-cell analyses we do not need to consider the alternative chromosomes and they can fall in regions of genes used as markers." />
<link rel="canonical" href="http://localhost:4000/2021/05/08/post-0009.html" />
<meta property="og:url" content="http://localhost:4000/2021/05/08/post-0009.html" />
<meta property="og:site_name" content="ARSN Lab Notebook" />
<meta property="og:type" content="article" />
<meta property="article:published_time" content="2021-05-08T00:00:00-07:00" />
<script type="application/ld+json">
{"headline":"Single Cell Alignments with no Alt Chromosomes","dateModified":"2021-05-08T00:00:00-07:00","datePublished":"2021-05-08T00:00:00-07:00","url":"http://localhost:4000/2021/05/08/post-0009.html","mainEntityOfPage":{"@type":"WebPage","@id":"http://localhost:4000/2021/05/08/post-0009.html"},"description":"Alt chromosomes in our hg38 and mm10 alignments (UCSC description) cause multi-mapping in those regions. BWA flags multi-mapping reads with a MAPQ score of 0 causing them to all be lost downstream in our analysis pipeline. For most single-cell analyses we do not need to consider the alternative chromosomes and they can fall in regions of genes used as markers.","@type":"BlogPosting","@context":"https://schema.org"}</script>
<!-- End Jekyll SEO tag -->
<link rel="stylesheet" href="/assets/main.css"><link type="application/atom+xml" rel="alternate" href="http://localhost:4000/feed.xml" title="ARSN Lab Notebook" /></head>
<body><header class="site-header" role="banner">

  <div class="wrapper"><a class="site-title" rel="author" href="/">ARSN Lab Notebook</a><nav class="site-nav">
        <input type="checkbox" id="nav-trigger" class="nav-trigger" />
        <label for="nav-trigger">
          <span class="menu-icon">
            <svg viewBox="0 0 18 15" width="18px" height="15px">
              <path d="M18,1.484c0,0.82-0.665,1.484-1.484,1.484H1.484C0.665,2.969,0,2.304,0,1.484l0,0C0,0.665,0.665,0,1.484,0 h15.032C17.335,0,18,0.665,18,1.484L18,1.484z M18,7.516C18,8.335,17.335,9,16.516,9H1.484C0.665,9,0,8.335,0,7.516l0,0 c0-0.82,0.665-1.484,1.484-1.484h15.032C17.335,6.031,18,6.696,18,7.516L18,7.516z M18,13.516C18,14.335,17.335,15,16.516,15H1.484 C0.665,15,0,14.335,0,13.516l0,0c0-0.82,0.665-1.483,1.484-1.483h15.032C17.335,12.031,18,12.695,18,13.516L18,13.516z"/>
            </svg>
          </span>
        </label>

        <div class="trigger"><a class="page-link" href="/about/">About</a></div>
      </nav></div>
</header>
<main class="page-content" aria-label="Content">
      <div class="wrapper">
        <article class="post h-entry" itemscope itemtype="http://schema.org/BlogPosting">

  <header class="post-header">
    <h1 class="post-title p-name" itemprop="name headline">Single Cell Alignments with no Alt Chromosomes</h1>
    <p class="post-meta">
      <time class="dt-published" datetime="2021-05-08T00:00:00-07:00" itemprop="datePublished">May 8, 2021
      </time></p>
  </header>

  <div class="post-content e-content" itemprop="articleBody">
    <p>Alt chromosomes in our hg38 and mm10 alignments (<a href="https://genome.ucsc.edu/FAQ/FAQdownloads.html#downloadAlt">UCSC description</a>) cause multi-mapping in those regions. BWA flags multi-mapping reads with a MAPQ score of 0 causing them to all be lost downstream in our analysis pipeline. For most single-cell analyses we do not need to consider the alternative chromosomes and they can fall in regions of genes used as markers.</p>

<p>Here are replaced genome builds without alts. See <code class="language-plaintext highlighter-rouge">/home/groups/oroaklab/refs/hg38/GCA_000001405.15_GRCh38/retrieval.txt</code> and <code class="language-plaintext highlighter-rouge">/home/groups/oroaklab/refs/mm10/GCA_000001635.5_GRCm38.p3_no_alt_analysis_set/retrieval.txt</code> for origins of FASTA files. Indexes for BWA were recalled with v0.7.15-r1140.</p>

<p><strong>FASTQ-Align Call</strong></p>
<div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>ln -s /home/groups/oroaklab/demultiplex/201214_NS500556_0445_AHMF2CBGXF/201214_NS500556_0445_AHMF2CBGXF.1.fq.gz testR1.fq.gz
ln -s /home/groups/oroaklab/demultiplex/201214_NS500556_0445_AHMF2CBGXF/201214_NS500556_0445_AHMF2CBGXF.2.fq.gz testR2.fq.gz 
no_alt_align_hg38_="/home/groups/oroaklab/refs/hg38/hs38d1_noalt.fna.gz"
no_alt_align_mm10_="/home/groups/oroaklab/refs/mm10/GCA_000001635.5_GRCm38.p3_noalt.fna.gz"
scitools fastq-align -t 28 $no_alt_align_hg38_ testoutput testR1.fq.gz testR2.fq.gz
</code></pre></div></div>

<p><strong>MAPQ using standard genome with alts vs. hs38d1_noalt</strong></p>
<div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>$ samtools view /home/groups/oroaklab/adey_lab/projects/Panc_U54/201214_NS500556_0445_AHMF2CBGXF/201214_NS500556_0445_AHMF2CBGXF.bam chr1:198,636,928-198,659,067 | cut -f 5 | sort | uniq -c
   2500 0
$ samtools view testoutput.bam chr1:198,636,928-198,659,067 | cut -f 5 | sort | uniq -c
     19 0
      1 1
      1 6
      3 24
      1 36
      1 39
     22 40
      1 41
      1 44
      1 45
      1 49
      1 50
      1 51
      2 53
      4 54
      1 57
      2 59
   4874 60
</code></pre></div></div>

  </div><a class="u-url" href="/2021/05/08/post-0009.html" hidden></a>
</article>

      </div>
    </main><footer class="site-footer h-card">
  <data class="u-url" href="/"></data>

  <div class="wrapper">

    <h2 class="footer-heading">ARSN Lab Notebook</h2>

    <div class="footer-col-wrapper">
      <div class="footer-col footer-col-1">
        <ul class="contact-list">
          <li class="p-name">ARSN Lab Notebook</li><li><a class="u-email" href="mailto:nishidaa@ohsu.edu">nishidaa@ohsu.edu</a></li></ul>
      </div>

      <div class="footer-col footer-col-2"><ul class="social-media-list"><li><a href="https://github.com/arsnishida"><svg class="svg-icon"><use xlink:href="/assets/minima-social-icons.svg#github"></use></svg> <span class="username">arsnishida</span></a></li></ul>
</div>

      <div class="footer-col footer-col-3">
        <p>ARSN Lab notebook.</p>
      </div>
    </div>

  </div>

</footer>
</body>

</html>
